<!doctype html>
<!--[if IE 9]><html class="ie9" lang="en"><![endif]-->
<html lang="en" class="preview">
<head>
<meta name="citation_pii" content="S1053811914004686" />
<meta name="citation_issn" content="1053-8119" />
<meta name="citation_volume" content="100" />
<meta name="citation_lastpage" content="315" />
<meta name="citation_publisher" content="Academic Press" />
<meta name="citation_firstpage" content="301" />
<meta name="citation_journal_title" content="NeuroImage" />
<meta name="citation_type" content="JOUR" />
<meta name="citation_doi" content="10.1016/j.neuroimage.2014.05.083" />
<meta name="dc.identifier" content="10.1016/j.neuroimage.2014.05.083" />
<meta name="citation_article_type" content="Full-length article" />
<meta property=og:description content="Modeling of resting state functional magnetic resonance imaging (rs-fMRI) data using network models is of increasing interest. It is often desirable tâ€¦" />
<meta property=og:image content="https://ars.els-cdn.com/content/image/1-s2.0-S1053811914X00157-cov150h.gif" />
<meta name="citation_title" content="Non-parametric Bayesian graph models reveal community structure in resting state fMRI" />
<meta property=og:title content="Non-parametric Bayesian graph models reveal community structure in resting state fMRI" />
<meta name="citation_publication_date" content="2014/10/15" />
<meta name="citation_online_date" content="2014/06/07" />
<meta name="robots" content="INDEX,FOLLOW,NOARCHIVE,NOODP,NOYDIR" />
<title>Non-parametric Bayesian graph models reveal community structure in resting state fMRI - ScienceDirect</title>
<link rel="canonical" href="https://www.sciencedirect.com/science/article/abs/pii/S1053811914004686" />
<meta property="og:type" content="article" />
<meta name="viewport" content="initial-scale=1" />
<meta name="SDTech" content="Proudly brought to you by the SD Technology team in London, Dayton, and Amsterdam" />
<script type="3855cf66cf87175080abf308-text/javascript">(function newRelicBrowserProSPA() {
  ;
  window.NREUM || (NREUM = {});
  NREUM.init = {
    privacy: {
      cookies_enabled: true
    },
    ajax: {
      deny_list: ["bam.nr-data.net"]
    }
  };
  ;
  NREUM.loader_config = {
    accountID: "2128461",
    trustKey: "2038175",
    agentID: "1124704362",
    licenseKey: "7ac4127487",
    applicationID: "814814845"
  };
  ;
  NREUM.info = {
    beacon: "bam.nr-data.net",
    errorBeacon: "bam.nr-data.net",
    licenseKey: "7ac4127487",
    applicationID: "814814845",
    sa: 1
  };
  ;
  (() => {
    var e,
      t,
      r = {
        9071: (e, t, r) => {
          "use strict";

          r.d(t, {
            I: () => n
          });
          var n = 0,
            i = navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);
          i && (n = +i[1]);
        },
        6900: (e, t, r) => {
          "use strict";

          let n;
          if (r.d(t, {
            H: () => i
          }), r(2374).il) {
            const e = document.createElement("div");
            e.innerHTML = "\x3c!--[if lte IE 6]><div></div><![endif]--\x3e\x3c!--[if lte IE 7]><div></div><![endif]--\x3e\x3c!--[if lte IE 8]><div></div><![endif]--\x3e\x3c!--[if lte IE 9]><div></div><![endif]--\x3e", n = e.getElementsByTagName("div").length;
          }
          var i;
          i = 4 === n ? 6 : 3 === n ? 7 : 2 === n ? 8 : 1 === n ? 9 : 0;
        },
        6562: (e, t, r) => {
          "use strict";

          r.d(t, {
            P_: () => p,
            Mt: () => v,
            C5: () => d,
            DL: () => y,
            OP: () => L,
            lF: () => H,
            Yu: () => E,
            Dg: () => g,
            CX: () => f,
            GE: () => w,
            sU: () => M
          });
          var n = {};
          r.r(n), r.d(n, {
            agent: () => O,
            match: () => N,
            version: () => A
          });
          var i = r(6797),
            o = r(909),
            a = r(8610);
          class s {
            constructor(e, t) {
              try {
                if (!e || "object" != typeof e) return (0, a.Z)("New setting a Configurable requires an object as input");
                if (!t || "object" != typeof t) return (0, a.Z)("Setting a Configurable requires a model to set its initial properties");
                Object.assign(this, t), Object.entries(e).forEach(e => {
                  let [t, r] = e;
                  const n = (0, o.q)(t);
                  n.length && r && "object" == typeof r && n.forEach(e => {
                    e in r && ((0, a.Z)(`"${e}" is a protected attribute and can not be changed in feature ${t}.  It will have no effect.`), delete r[e]);
                  }), this[t] = r;
                });
              } catch (e) {
                (0, a.Z)("An error occured while setting a Configurable", e);
              }
            }
          }
          const c = {
              beacon: i.ce.beacon,
              errorBeacon: i.ce.errorBeacon,
              licenseKey: void 0,
              applicationID: void 0,
              sa: void 0,
              queueTime: void 0,
              applicationTime: void 0,
              ttGuid: void 0,
              user: void 0,
              account: void 0,
              product: void 0,
              extra: void 0,
              jsAttributes: {},
              userAttributes: void 0,
              atts: void 0,
              transactionName: void 0,
              tNamePlain: void 0
            },
            u = {};
          function d(e) {
            if (!e) throw new Error("All info objects require an agent identifier!");
            if (!u[e]) throw new Error(`Info for ${e} was never set`);
            return u[e];
          }
          function f(e, t) {
            if (!e) throw new Error("All info objects require an agent identifier!");
            u[e] = new s(t, c), (0, i.Qy)(e, u[e], "info");
          }
          const l = {
              allow_bfcache: !1,
              privacy: {
                cookies_enabled: !0
              },
              ajax: {
                deny_list: void 0,
                enabled: !0,
                harvestTimeSeconds: 10
              },
              distributed_tracing: {
                enabled: void 0,
                exclude_newrelic_header: void 0,
                cors_use_newrelic_header: void 0,
                cors_use_tracecontext_headers: void 0,
                allowed_origins: void 0
              },
              ssl: void 0,
              obfuscate: void 0,
              jserrors: {
                enabled: !0,
                harvestTimeSeconds: 10
              },
              metrics: {
                enabled: !0,
                harvestTimeSeconds: 10
              },
              page_action: {
                enabled: !0,
                harvestTimeSeconds: 30
              },
              page_view_event: {
                enabled: !0
              },
              page_view_timing: {
                enabled: !0,
                harvestTimeSeconds: 30
              },
              session_trace: {
                enabled: !0,
                harvestTimeSeconds: 10
              },
              spa: {
                enabled: !0,
                harvestTimeSeconds: 10
              }
            },
            h = {};
          function p(e) {
            if (!e) throw new Error("All configuration objects require an agent identifier!");
            if (!h[e]) throw new Error(`Configuration for ${e} was never set`);
            return h[e];
          }
          function g(e, t) {
            if (!e) throw new Error("All configuration objects require an agent identifier!");
            h[e] = new s(t, l), (0, i.Qy)(e, h[e], "config");
          }
          function v(e, t) {
            if (!e) throw new Error("All configuration objects require an agent identifier!");
            var r = p(e);
            if (r) {
              for (var n = t.split("."), i = 0; i < n.length - 1; i++) if ("object" != typeof (r = r[n[i]])) return;
              r = r[n[n.length - 1]];
            }
            return r;
          }
          const m = {
              accountID: void 0,
              trustKey: void 0,
              agentID: void 0,
              licenseKey: void 0,
              applicationID: void 0,
              xpid: void 0
            },
            b = {};
          function y(e) {
            if (!e) throw new Error("All loader-config objects require an agent identifier!");
            if (!b[e]) throw new Error(`LoaderConfig for ${e} was never set`);
            return b[e];
          }
          function w(e, t) {
            if (!e) throw new Error("All loader-config objects require an agent identifier!");
            b[e] = new s(t, m), (0, i.Qy)(e, b[e], "loader_config");
          }
          const E = (0, i.mF)().o;
          var T = r(6900),
            x = r(2053),
            O = null,
            A = null;
          if (navigator.userAgent) {
            var _ = navigator.userAgent,
              S = _.match(/Version\/(\S+)\s+Safari/);
            S && -1 === _.indexOf("Chrome") && -1 === _.indexOf("Chromium") && (O = "Safari", A = S[1]);
          }
          function N(e, t) {
            if (!O) return !1;
            if (e !== O) return !1;
            if (!t) return !0;
            if (!A) return !1;
            for (var r = A.split("."), n = t.split("."), i = 0; i < n.length; i++) if (n[i] !== r[i]) return !1;
            return !0;
          }
          var D = r(5526),
            P = r(2374);
          const I = "NRBA_SESSION_ID";
          function R() {
            if (!P.il) return null;
            try {
              let e;
              return null === (e = window.sessionStorage.getItem(I)) && (e = (0, D.ky)(16), window.sessionStorage.setItem(I, e)), e;
            } catch (e) {
              return null;
            }
          }
          var C = P._A?.XMLHttpRequest,
            j = C && C.prototype;
          const k = {};
          function L(e) {
            if (!e) throw new Error("All runtime objects require an agent identifier!");
            if (!k[e]) throw new Error(`Runtime for ${e} was never set`);
            return k[e];
          }
          function M(e, t) {
            if (!e) throw new Error("All runtime objects require an agent identifier!");
            var r;
            k[e] = new s(t, (r = e, {
              customTransaction: void 0,
              disabled: !1,
              features: {},
              isolatedBacklog: !1,
              loaderType: void 0,
              maxBytes: 6 === T.H ? 2e3 : 3e4,
              offset: (0, x.yf)(),
              onerror: void 0,
              origin: "" + P._A?.location,
              ptid: void 0,
              releaseIds: {},
              sessionId: 1 == v(r, "privacy.cookies_enabled") ? R() : null,
              xhrWrappable: C && j && j.addEventListener && !/CriOS/.test(navigator.userAgent),
              userAgent: n
            })), (0, i.Qy)(e, k[e], "runtime");
          }
          function H(e) {
            return function (e) {
              try {
                const t = d(e);
                return !!t.licenseKey && !!t.errorBeacon && !!t.applicationID;
              } catch (e) {
                return !1;
              }
            }(e);
          }
        },
        8226: (e, t, r) => {
          "use strict";

          r.d(t, {
            q: () => n
          });
          const n = "1223.PROD";
        },
        9557: (e, t, r) => {
          "use strict";

          r.d(t, {
            w: () => o
          });
          var n = r(8610);
          const i = {
            agentIdentifier: ""
          };
          class o {
            constructor(e) {
              try {
                if ("object" != typeof e) return (0, n.Z)("shared context requires an object as input");
                this.sharedContext = {}, Object.assign(this.sharedContext, i), Object.entries(e).forEach(e => {
                  let [t, r] = e;
                  Object.keys(i).includes(t) && (this.sharedContext[t] = r);
                });
              } catch (e) {
                (0, n.Z)("An error occured while setting SharedContext", e);
              }
            }
          }
        },
        4329: (e, t, r) => {
          "use strict";

          r.d(t, {
            L: () => d,
            R: () => c
          });
          var n = r(3752),
            i = r(7022),
            o = r(4045),
            a = r(2325);
          const s = {};
          function c(e, t) {
            const r = {
              staged: !1,
              priority: a.p[t] || 0
            };
            u(e), s[e].get(t) || s[e].set(t, r);
          }
          function u(e) {
            e && (s[e] || (s[e] = new Map()));
          }
          function d() {
            let e = arguments.length > 0 && void 0 !== arguments[0] ? arguments[0] : "",
              t = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : "feature";
            if (u(e), !e || !s[e].get(t)) return a(t);
            s[e].get(t).staged = !0;
            const r = Array.from(s[e]);
            function a(t) {
              const r = e ? n.ee.get(e) : n.ee,
                a = o.X.handlers;
              if (r.backlog && a) {
                var s = r.backlog[t],
                  c = a[t];
                if (c) {
                  for (var u = 0; s && u < s.length; ++u) f(s[u], c);
                  (0, i.D)(c, function (e, t) {
                    (0, i.D)(t, function (t, r) {
                      r[0].on(e, r[1]);
                    });
                  });
                }
                delete a[t], r.backlog[t] = null, r.emit("drain-" + t, []);
              }
            }
            r.every(e => {
              let [t, r] = e;
              return r.staged;
            }) && (r.sort((e, t) => e[1].priority - t[1].priority), r.forEach(e => {
              let [t] = e;
              a(t);
            }));
          }
          function f(e, t) {
            var r = e[1];
            (0, i.D)(t[r], function (t, r) {
              var n = e[0];
              if (r[0] === n) {
                var i = r[1],
                  o = e[3],
                  a = e[2];
                i.apply(o, a);
              }
            });
          }
        },
        3752: (e, t, r) => {
          "use strict";

          r.d(t, {
            c: () => f,
            ee: () => u
          });
          var n = r(6797),
            i = r(3916),
            o = r(7022),
            a = r(6562),
            s = "nr@context";
          let c = (0, n.fP)();
          var u;
          function d() {}
          function f(e) {
            return (0, i.X)(e, s, l);
          }
          function l() {
            return new d();
          }
          function h() {
            (u.backlog.api || u.backlog.feature) && (u.aborted = !0, u.backlog = {});
          }
          c.ee ? u = c.ee : (u = function e(t, r) {
            var n = {},
              c = {},
              f = {},
              p = !1;
            try {
              p = 16 === r.length && (0, a.OP)(r).isolatedBacklog;
            } catch (e) {}
            var g = {
              on: b,
              addEventListener: b,
              removeEventListener: y,
              emit: m,
              get: E,
              listeners: w,
              context: v,
              buffer: T,
              abort: h,
              aborted: !1,
              isBuffering: x,
              debugId: r,
              backlog: p ? {} : t && "object" == typeof t.backlog ? t.backlog : {}
            };
            return g;
            function v(e) {
              return e && e instanceof d ? e : e ? (0, i.X)(e, s, l) : l();
            }
            function m(e, r, n, i, o) {
              if (!1 !== o && (o = !0), !u.aborted || i) {
                t && o && t.emit(e, r, n);
                for (var a = v(n), s = w(e), d = s.length, f = 0; f < d; f++) s[f].apply(a, r);
                var l = O()[c[e]];
                return l && l.push([g, e, r, a]), a;
              }
            }
            function b(e, t) {
              n[e] = w(e).concat(t);
            }
            function y(e, t) {
              var r = n[e];
              if (r) for (var i = 0; i < r.length; i++) r[i] === t && r.splice(i, 1);
            }
            function w(e) {
              return n[e] || [];
            }
            function E(t) {
              return f[t] = f[t] || e(g, t);
            }
            function T(e, t) {
              var r = O();
              g.aborted || (0, o.D)(e, function (e, n) {
                t = t || "feature", c[n] = t, t in r || (r[t] = []);
              });
            }
            function x(e) {
              return !!O()[c[e]];
            }
            function O() {
              return g.backlog;
            }
          }(void 0, "globalEE"), c.ee = u);
        },
        9252: (e, t, r) => {
          "use strict";

          r.d(t, {
            E: () => n,
            p: () => i
          });
          var n = r(3752).ee.get("handle");
          function i(e, t, r, i, o) {
            o ? (o.buffer([e], i), o.emit(e, t, r)) : (n.buffer([e], i), n.emit(e, t, r));
          }
        },
        4045: (e, t, r) => {
          "use strict";

          r.d(t, {
            X: () => o
          });
          var n = r(9252);
          o.on = a;
          var i = o.handlers = {};
          function o(e, t, r, o) {
            a(o || n.E, i, e, t, r);
          }
          function a(e, t, r, i, o) {
            o || (o = "feature"), e || (e = n.E);
            var a = t[o] = t[o] || {};
            (a[r] = a[r] || []).push([e, i]);
          }
        },
        8544: (e, t, r) => {
          "use strict";

          r.d(t, {
            bP: () => s,
            iz: () => c,
            m$: () => a
          });
          var n = r(2374),
            i = !1;
          try {
            var o = Object.defineProperty({}, "passive", {
              get: function () {
                i = !0;
              }
            });
            n._A?.addEventListener("testPassive", null, o), n._A?.removeEventListener("testPassive", null, o);
          } catch (e) {}
          function a(e) {
            return i ? {
              passive: !0,
              capture: !!e
            } : !!e;
          }
          function s(e, t) {
            let r = arguments.length > 2 && void 0 !== arguments[2] && arguments[2];
            window.addEventListener(e, t, a(r));
          }
          function c(e, t) {
            let r = arguments.length > 2 && void 0 !== arguments[2] && arguments[2];
            document.addEventListener(e, t, a(r));
          }
        },
        5526: (e, t, r) => {
          "use strict";

          r.d(t, {
            Ht: () => a,
            M: () => o,
            Rl: () => i,
            ky: () => s
          });
          var n = r(2374);
          function i() {
            var e = null,
              t = 0,
              r = n._A?.crypto || n._A?.msCrypto;
            function i() {
              return e ? 15 & e[t++] : 16 * Math.random() | 0;
            }
            r && r.getRandomValues && (e = r.getRandomValues(new Uint8Array(31)));
            for (var o, a = "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx", s = "", c = 0; c < a.length; c++) s += "x" === (o = a[c]) ? i().toString(16) : "y" === o ? (o = 3 & i() | 8).toString(16) : o;
            return s;
          }
          function o() {
            return s(16);
          }
          function a() {
            return s(32);
          }
          function s(e) {
            var t = null,
              r = 0,
              n = self.crypto || self.msCrypto;
            n && n.getRandomValues && Uint8Array && (t = n.getRandomValues(new Uint8Array(31)));
            for (var i = [], o = 0; o < e; o++) i.push(a().toString(16));
            return i.join("");
            function a() {
              return t ? 15 & t[r++] : 16 * Math.random() | 0;
            }
          }
        },
        2053: (e, t, r) => {
          "use strict";

          r.d(t, {
            nb: () => c,
            os: () => u,
            yf: () => s,
            zO: () => a
          });
          var n = r(7145),
            i = new Date().getTime(),
            o = i;
          function a() {
            return n.G && performance.now ? Math.round(performance.now()) : (i = Math.max(new Date().getTime(), i)) - o;
          }
          function s() {
            return i;
          }
          function c(e) {
            o = e;
          }
          function u() {
            return o;
          }
        },
        7145: (e, t, r) => {
          "use strict";

          r.d(t, {
            G: () => n
          });
          const n = void 0 !== r(2374)._A?.performance?.timing?.navigationStart;
        },
        6625: (e, t, r) => {
          "use strict";

          r.d(t, {
            s: () => c,
            v: () => u
          });
          var n = r(8283),
            i = r(9071),
            o = r(2053),
            a = r(7145),
            s = r(2374);
          let c = !0;
          function u(e) {
            var t = function () {
              if (i.I && i.I < 9) return;
              if (a.G) return c = !1, s._A?.performance?.timing?.navigationStart;
            }();
            t && ((0, n.B)(e, "starttime", t), (0, o.nb)(t));
          }
        },
        8283: (e, t, r) => {
          "use strict";

          r.d(t, {
            B: () => o,
            L: () => a
          });
          var n = r(2053),
            i = {};
          function o(e, t, r) {
            void 0 === r && (r = (0, n.zO)() + (0, n.os)()), i[e] = i[e] || {}, i[e][t] = r;
          }
          function a(e, t, r, n) {
            const o = e.sharedContext.agentIdentifier;
            var a = i[o]?.[r],
              s = i[o]?.[n];
            void 0 !== a && void 0 !== s && e.store("measures", t, {
              value: s - a
            });
          }
        },
        6368: (e, t, r) => {
          "use strict";

          r.d(t, {
            e: () => o
          });
          var n = r(2374),
            i = {};
          function o(e) {
            if (e in i) return i[e];
            if (0 === (e || "").indexOf("data:")) return {
              protocol: "data"
            };
            let t;
            var r = n._A?.location,
              o = {};
            if (n.il) t = document.createElement("a"), t.href = e;else try {
              t = new URL(e, r.href);
            } catch (e) {
              return o;
            }
            o.port = t.port;
            var a = t.href.split("://");
            !o.port && a[1] && (o.port = a[1].split("/")[0].split("@").pop().split(":")[1]), o.port && "0" !== o.port || (o.port = "https" === a[0] ? "443" : "80"), o.hostname = t.hostname || r.hostname, o.pathname = t.pathname, o.protocol = a[0], "/" !== o.pathname.charAt(0) && (o.pathname = "/" + o.pathname);
            var s = !t.protocol || ":" === t.protocol || t.protocol === r.protocol,
              c = t.hostname === r.hostname && t.port === r.port;
            return o.sameOrigin = s && (!t.hostname || c), "/" === o.pathname && (i[e] = o), o;
          }
        },
        9548: (e, t, r) => {
          "use strict";

          r.d(t, {
            T: () => i
          });
          var n = r(2374);
          const i = {
            isFileProtocol: function () {
              let e = Boolean("file:" === (0, n.lW)()?.location?.protocol);
              e && (i.supportabilityMetricSent = !0);
              return e;
            },
            supportabilityMetricSent: !1
          };
        },
        8610: (e, t, r) => {
          "use strict";

          function n(e, t) {
            console && console.warn && "function" == typeof console.warn && (console.warn(`New Relic: ${e}`), t && console.warn(t));
          }
          r.d(t, {
            Z: () => n
          });
        },
        3916: (e, t, r) => {
          "use strict";

          r.d(t, {
            X: () => i
          });
          var n = Object.prototype.hasOwnProperty;
          function i(e, t, r) {
            if (n.call(e, t)) return e[t];
            var i = r();
            if (Object.defineProperty && Object.keys) try {
              return Object.defineProperty(e, t, {
                value: i,
                writable: !0,
                enumerable: !1
              }), i;
            } catch (e) {}
            return e[t] = i, i;
          }
        },
        2374: (e, t, r) => {
          "use strict";

          r.d(t, {
            _A: () => o,
            il: () => n,
            lW: () => a,
            v6: () => i
          });
          const n = Boolean("undefined" != typeof window && window.document),
            i = Boolean("undefined" != typeof WorkerGlobalScope && self.navigator instanceof WorkerNavigator);
          let o = (() => {
            if (n) return window;
            if (i) {
              if ("undefined" != typeof globalThis && globalThis instanceof WorkerGlobalScope) return globalThis;
              if (self instanceof WorkerGlobalScope) return self;
            }
            throw new Error("New Relic browser agent shutting down due to error: Unable to locate global scope. This is possibly due to code redefining browser global variables like `self` and `window`.");
          })();
          function a() {
            return o;
          }
        },
        7022: (e, t, r) => {
          "use strict";

          r.d(t, {
            D: () => i
          });
          var n = Object.prototype.hasOwnProperty;
          function i(e, t) {
            var r = [],
              i = "",
              o = 0;
            for (i in e) n.call(e, i) && (r[o] = t(i, e[i]), o += 1);
            return r;
          }
        },
        9226: (e, t, r) => {
          "use strict";

          r.d(t, {
            $c: () => u,
            Ng: () => d,
            RR: () => c
          });
          var n = r(6562),
            i = r(9557),
            o = r(9548),
            a = r(8610),
            s = {
              regex: /^file:\/\/(.*)/,
              replacement: "file://OBFUSCATED"
            };
          class c extends i.w {
            constructor(e) {
              super(e);
            }
            shouldObfuscate() {
              return u(this.sharedContext.agentIdentifier).length > 0;
            }
            obfuscateString(e) {
              if (!e || "string" != typeof e) return e;
              for (var t = u(this.sharedContext.agentIdentifier), r = e, n = 0; n < t.length; n++) {
                var i = t[n].regex,
                  o = t[n].replacement || "*";
                r = r.replace(i, o);
              }
              return r;
            }
          }
          function u(e) {
            var t = [],
              r = (0, n.Mt)(e, "obfuscate") || [];
            return t = t.concat(r), o.T.isFileProtocol() && t.push(s), t;
          }
          function d(e) {
            for (var t = !1, r = !1, n = 0; n < e.length; n++) {
              "regex" in e[n] ? "string" != typeof e[n].regex && e[n].regex.constructor !== RegExp && ((0, a.Z)('An obfuscation replacement rule contains a "regex" value with an invalid type (must be a string or RegExp)'), r = !0) : ((0, a.Z)('An obfuscation replacement rule was detected missing a "regex" value.'), r = !0);
              var i = e[n].replacement;
              i && "string" != typeof i && ((0, a.Z)('An obfuscation replacement rule contains a "replacement" value with an invalid type (must be a string)'), t = !0);
            }
            return !t && !r;
          }
        },
        2650: (e, t, r) => {
          "use strict";

          r.d(t, {
            K: () => a,
            b: () => o
          });
          var n = r(8544);
          function i() {
            return "undefined" == typeof document || "complete" === document.readyState;
          }
          function o(e, t) {
            if (i()) return e();
            (0, n.bP)("load", e, t);
          }
          function a(e) {
            if (i()) return e();
            (0, n.iz)("DOMContentLoaded", e);
          }
        },
        6797: (e, t, r) => {
          "use strict";

          r.d(t, {
            EZ: () => u,
            Qy: () => c,
            ce: () => o,
            fP: () => a,
            gG: () => d,
            mF: () => s
          });
          var n = r(2053),
            i = r(2374);
          const o = {
            beacon: "bam.nr-data.net",
            errorBeacon: "bam.nr-data.net"
          };
          function a() {
            return i._A.NREUM || (i._A.NREUM = {}), void 0 === i._A.newrelic && (i._A.newrelic = i._A.NREUM), i._A.NREUM;
          }
          function s() {
            let e = a();
            return e.o || (e.o = {
              ST: i._A.setTimeout,
              SI: i._A.setImmediate,
              CT: i._A.clearTimeout,
              XHR: i._A.XMLHttpRequest,
              REQ: i._A.Request,
              EV: i._A.Event,
              PR: i._A.Promise,
              MO: i._A.MutationObserver,
              FETCH: i._A.fetch
            }), e;
          }
          function c(e, t, r) {
            let i = a();
            const o = i.initializedAgents || {},
              s = o[e] || {};
            return Object.keys(s).length || (s.initializedAt = {
              ms: (0, n.zO)(),
              date: new Date()
            }), i.initializedAgents = {
              ...o,
              [e]: {
                ...s,
                [r]: t
              }
            }, i;
          }
          function u(e, t) {
            a()[e] = t;
          }
          function d() {
            return function () {
              let e = a();
              const t = e.info || {};
              e.info = {
                beacon: o.beacon,
                errorBeacon: o.errorBeacon,
                ...t
              };
            }(), function () {
              let e = a();
              const t = e.init || {};
              e.init = {
                ...t
              };
            }(), s(), function () {
              let e = a();
              const t = e.loader_config || {};
              e.loader_config = {
                ...t
              };
            }(), a();
          }
        },
        6998: (e, t, r) => {
          "use strict";

          r.d(t, {
            N: () => i,
            e: () => o
          });
          var n = r(8544);
          function i(e) {
            let t = arguments.length > 1 && void 0 !== arguments[1] && arguments[1];
            return void (0, n.iz)("visibilitychange", r);
            function r() {
              if (t) {
                if ("hidden" != document.visibilityState) return;
                e();
              }
              e(document.visibilityState);
            }
          }
          function o() {
            return "hidden" === document.visibilityState ? -1 : 1 / 0;
          }
        },
        6408: (e, t, r) => {
          "use strict";

          r.d(t, {
            W: () => i
          });
          var n = r(2374);
          function i() {
            return "function" == typeof n._A?.PerformanceObserver;
          }
        },
        8675: (e, t, r) => {
          "use strict";

          r.d(t, {
            t: () => n
          });
          const n = r(2325).D.ajax;
        },
        8322: (e, t, r) => {
          "use strict";

          r.d(t, {
            A: () => i,
            t: () => n
          });
          const n = r(2325).D.jserrors,
            i = "nr@seenError";
        },
        6034: (e, t, r) => {
          "use strict";

          r.d(t, {
            t: () => n
          });
          const n = r(2325).D.metrics;
        },
        6486: (e, t, r) => {
          "use strict";

          r.d(t, {
            t: () => n
          });
          const n = r(2325).D.pageAction;
        },
        2484: (e, t, r) => {
          "use strict";

          r.d(t, {
            t: () => n
          });
          const n = r(2325).D.pageViewEvent;
        },
        6382: (e, t, r) => {
          "use strict";

          r.d(t, {
            t: () => n
          });
          const n = r(2325).D.pageViewTiming;
        },
        2628: (e, t, r) => {
          "use strict";

          r.r(t), r.d(t, {
            ADD_EVENT_LISTENER: () => a,
            BST_RESOURCE: () => u,
            BST_TIMER: () => g,
            END: () => l,
            FEATURE_NAME: () => i,
            FN_END: () => p,
            FN_START: () => h,
            ORIG_EVENT: () => m,
            PUSH_STATE: () => v,
            REMOVE_EVENT_LISTENER: () => s,
            RESOURCE: () => d,
            RESOURCE_TIMING_BUFFER_FULL: () => c,
            START: () => f,
            learResourceTimings: () => o
          });
          var n = r(6562);
          const i = r(2325).D.sessionTrace,
            o = "learResourceTimings",
            a = "addEventListener",
            s = "removeEventListener",
            c = "resourcetimingbufferfull",
            u = "bstResource",
            d = "resource",
            f = "-start",
            l = "-end",
            h = "fn" + f,
            p = "fn" + l,
            g = "bstTimer",
            v = "pushState",
            m = n.Yu.EV;
        },
        755: (e, t, r) => {
          "use strict";

          r.r(t), r.d(t, {
            ADD_EVENT_LISTENER: () => S,
            BODY: () => x,
            CB_END: () => O,
            CB_START: () => d,
            END: () => T,
            FEATURE_NAME: () => o,
            FETCH: () => _,
            FETCH_BODY: () => b,
            FETCH_DONE: () => m,
            FETCH_START: () => v,
            FN_END: () => u,
            FN_START: () => c,
            INTERACTION: () => h,
            INTERACTION_API: () => f,
            INTERACTION_EVENTS: () => a,
            JSONP_END: () => y,
            JSONP_NODE: () => g,
            JS_TIME: () => A,
            LOCATION: () => D,
            MAX_TIMER_BUDGET: () => s,
            REMAINING: () => l,
            SPA_NODE: () => p,
            START: () => E,
            WIN: () => N,
            originalSetTimeout: () => w
          });
          var n = r(6562),
            i = r(2374);
          const o = r(2325).D.spa,
            a = ["click", "submit", "keypress", "keydown", "keyup", "change"],
            s = 999,
            c = "fn-start",
            u = "fn-end",
            d = "cb-start",
            f = "api-ixn-",
            l = "remaining",
            h = "interaction",
            p = "spaNode",
            g = "jsonpNode",
            v = "fetch-start",
            m = "fetch-done",
            b = "fetch-body-",
            y = "jsonp-end",
            w = n.Yu.ST,
            E = "-start",
            T = "-end",
            x = "-body",
            O = "cb" + T,
            A = "jsTime",
            _ = "fetch",
            S = "addEventListener",
            N = i._A,
            D = N.location;
        },
        1509: (e, t, r) => {
          "use strict";

          r.d(t, {
            W: () => s
          });
          var n = r(6562),
            i = r(3752),
            o = r(2384),
            a = r(6797);
          class s {
            constructor(e, t, r) {
              this.agentIdentifier = e, this.aggregator = t, this.ee = i.ee.get(e, (0, n.OP)(this.agentIdentifier).isolatedBacklog), this.featureName = r, this.blocked = !1, this.checkConfiguration();
            }
            checkConfiguration() {
              if (!(0, n.lF)(this.agentIdentifier)) {
                let e = {
                  ...(0, a.gG)().info?.jsAttributes
                };
                try {
                  e = {
                    ...e,
                    ...(0, n.C5)(this.agentIdentifier)?.jsAttributes
                  };
                } catch (e) {}
                (0, o.j)(this.agentIdentifier, {
                  ...(0, a.gG)(),
                  info: {
                    ...(0, a.gG)().info,
                    jsAttributes: e
                  }
                });
              }
            }
          }
        },
        2384: (e, t, r) => {
          "use strict";

          r.d(t, {
            j: () => y
          });
          var n = r(8683),
            i = r.n(n),
            o = r(2325),
            a = r(6562),
            s = r(9252),
            c = r(7022),
            u = r(3752),
            d = r(2053),
            f = r(4329),
            l = r(2650),
            h = r(2374);
          function p(e, t, n) {
            n || (0, f.R)(e, "api"), function (e) {
              function t(t) {
                for (var r = arguments.length, n = new Array(r > 1 ? r - 1 : 0), i = 1; i < r; i++) n[i - 1] = arguments[i];
                Object.values(e.initializedAgents).forEach(e => {
                  e.exposed && e.api[t] && e.api[t](...n);
                });
              }
              ["setErrorHandler", "finished", "addToTrace", "inlineHit", "addRelease", "addPageAction", "setCurrentRouteName", "setPageViewName", "setCustomAttribute", "interaction", "noticeError"].forEach(r => {
                e[r] = function () {
                  for (var e = arguments.length, n = new Array(e), i = 0; i < e; i++) n[i] = arguments[i];
                  return t(r, ...n);
                };
              });
            }(t);
            var p = u.ee.get(e),
              g = p.get("tracer"),
              v = "api-",
              m = "api-ixn-";
            function b() {}
            (0, c.D)(["setErrorHandler", "finished", "addToTrace", "inlineHit", "addRelease"], function (e, r) {
              t[r] = w(v, r, !0, "api");
            }), t.addPageAction = w(v, "addPageAction", !0, o.D.pageAction), t.setCurrentRouteName = w(v, "routeName", !0, o.D.spa), t.setPageViewName = function (t, r) {
              if ("string" == typeof t) return "/" !== t.charAt(0) && (t = "/" + t), (0, a.OP)(e).customTransaction = (r || "http://custom.transaction") + t, w(v, "setPageViewName", !0, "api")();
            }, t.setCustomAttribute = function (t, r) {
              const n = (0, a.C5)(e);
              return (0, a.CX)(e, {
                ...n,
                jsAttributes: {
                  ...n.jsAttributes,
                  [t]: r
                }
              }), w(v, "setCustomAttribute", !0, "api")();
            }, t.interaction = function () {
              return new b().get();
            };
            var y = b.prototype = {
              createTracer: function (e, t) {
                var r = {},
                  n = this,
                  i = "function" == typeof t;
                return (0, s.p)("api-ixn-tracer", [(0, d.zO)(), e, r], n, o.D.spa, p), function () {
                  if (g.emit((i ? "" : "no-") + "fn-start", [(0, d.zO)(), n, i], r), i) try {
                    return t.apply(this, arguments);
                  } catch (e) {
                    throw g.emit("fn-err", [arguments, this, "string" == typeof e ? new Error(e) : e], r), e;
                  } finally {
                    g.emit("fn-end", [(0, d.zO)()], r);
                  }
                };
              }
            };
            function w(e, t, r, n) {
              return function () {
                return (0, s.p)("record-supportability", ["API/" + t + "/called"], void 0, o.D.metrics, p), (0, s.p)(e + t, [(0, d.zO)()].concat(i()(arguments)), r ? null : this, n, p), r ? void 0 : this;
              };
            }
            function E() {
              r.e(692).then(r.bind(r, 5692)).then(t => {
                let {
                  setAPI: r
                } = t;
                r(e), (0, f.L)(e, "api");
              });
            }
            (0, c.D)("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","), function (e, t) {
              y[t] = w(m, t, void 0, o.D.spa);
            }), t.noticeError = function (e, t) {
              "string" == typeof e && (e = new Error(e)), (0, s.p)("record-supportability", ["API/noticeError/called"], void 0, o.D.metrics, p), (0, s.p)("err", [e, (0, d.zO)(), !1, t], void 0, o.D.jserrors, p);
            }, h.v6 ? E() : (0, l.b)(() => E(), !0);
          }
          var g = r(6797);
          const v = {
            stn: o.D.sessionTrace,
            err: o.D.jserrors,
            ins: o.D.pageAction,
            spa: o.D.spa
          };
          function m(e, t) {
            var r = u.ee.get(t);
            e && "object" == typeof e && ((0, c.D)(e, function (e, t) {
              if (!t) return (0, s.p)("block-" + e, [], void 0, v[e], r);
              t && !b[e] && ((0, s.p)("feat-" + e, [], void 0, v[e], r), b[e] = !0);
            }), (0, f.L)(t, o.D.pageViewEvent));
          }
          const b = {};
          function y(e) {
            let t = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : {},
              r = arguments.length > 2 ? arguments[2] : void 0,
              n = arguments.length > 3 ? arguments[3] : void 0,
              {
                init: i,
                info: o,
                loader_config: s,
                runtime: c = {
                  loaderType: r
                },
                exposed: u = !0
              } = t;
            const d = (0, g.gG)();
            let f = {};
            return o || (i = d.init, o = d.info, s = d.loader_config, f = d), h.v6 && (o.jsAttributes = {
              ...o.jsAttributes,
              isWorker: !0
            }), (0, a.CX)(e, o), (0, a.Dg)(e, i || {}), (0, a.GE)(e, s || {}), (0, a.sU)(e, c), p(e, f, n), (0, g.Qy)(e, d, "api"), (0, g.Qy)(e, u, "exposed"), h.v6 || ((0, g.EZ)("activatedFeatures", b), (0, g.EZ)("setToken", t => m(t, e))), f;
          }
        },
        909: (e, t, r) => {
          "use strict";

          r.d(t, {
            Z: () => i,
            q: () => o
          });
          var n = r(2325);
          function i(e) {
            switch (e) {
              case n.D.ajax:
                return [n.D.jserrors];
              case n.D.sessionTrace:
                return [n.D.ajax, n.D.pageViewEvent];
              case n.D.pageViewTiming:
                return [n.D.pageViewEvent];
              default:
                return [];
            }
          }
          function o(e) {
            return e === n.D.jserrors ? [] : ["auto"];
          }
        },
        2325: (e, t, r) => {
          "use strict";

          r.d(t, {
            D: () => n,
            p: () => i
          });
          const n = {
              ajax: "ajax",
              jserrors: "jserrors",
              metrics: "metrics",
              pageAction: "page_action",
              pageViewEvent: "page_view_event",
              pageViewTiming: "page_view_timing",
              sessionTrace: "session_trace",
              spa: "spa"
            },
            i = {
              [n.pageViewEvent]: 1,
              [n.pageViewTiming]: 2,
              [n.metrics]: 3,
              [n.jserrors]: 4,
              [n.ajax]: 5,
              [n.sessionTrace]: 6,
              [n.pageAction]: 7,
              [n.spa]: 8
            };
        },
        8683: e => {
          e.exports = function (e, t, r) {
            t || (t = 0), void 0 === r && (r = e ? e.length : 0);
            for (var n = -1, i = r - t || 0, o = Array(i < 0 ? 0 : i); ++n < i;) o[n] = e[t + n];
            return o;
          };
        },
        4992: (e, t, r) => {
          var n = {
            "./ajax/aggregate": [960, 779, 960],
            "./jserrors/aggregate": [6112, 779, 112],
            "./metrics/aggregate": [4325, 779, 325],
            "./page_action/aggregate": [6817, 779, 817],
            "./page_view_event/aggregate": [4823, 779, 823],
            "./page_view_timing/aggregate": [8785, 779, 785],
            "./session_trace/aggregate": [1307, 779, 307],
            "./spa/aggregate": [9378, 779, 378]
          };
          function i(e) {
            if (!r.o(n, e)) return Promise.resolve().then(() => {
              var t = new Error("Cannot find module '" + e + "'");
              throw t.code = "MODULE_NOT_FOUND", t;
            });
            var t = n[e],
              i = t[0];
            return Promise.all(t.slice(1).map(r.e)).then(() => r(i));
          }
          i.keys = () => Object.keys(n), i.id = 4992, e.exports = i;
        }
      },
      n = {};
    function i(e) {
      var t = n[e];
      if (void 0 !== t) return t.exports;
      var o = n[e] = {
        exports: {}
      };
      return r[e](o, o.exports, i), o.exports;
    }
    i.m = r, i.n = e => {
      var t = e && e.__esModule ? () => e.default : () => e;
      return i.d(t, {
        a: t
      }), t;
    }, i.d = (e, t) => {
      for (var r in t) i.o(t, r) && !i.o(e, r) && Object.defineProperty(e, r, {
        enumerable: !0,
        get: t[r]
      });
    }, i.f = {}, i.e = e => Promise.all(Object.keys(i.f).reduce((t, r) => (i.f[r](e, t), t), [])), i.u = e => e + "." + i.h().slice(0, 8) + "-1223.js", i.h = () => "215647de1097144af948", i.o = (e, t) => Object.prototype.hasOwnProperty.call(e, t), e = {}, t = "NRBA:", i.l = (r, n, o, a) => {
      if (e[r]) e[r].push(n);else {
        var s, c;
        if (void 0 !== o) for (var u = document.getElementsByTagName("script"), d = 0; d < u.length; d++) {
          var f = u[d];
          if (f.getAttribute("src") == r || f.getAttribute("data-webpack") == t + o) {
            s = f;
            break;
          }
        }
        s || (c = !0, (s = document.createElement("script")).charset = "utf-8", s.timeout = 120, i.nc && s.setAttribute("nonce", i.nc), s.setAttribute("data-webpack", t + o), s.src = r), e[r] = [n];
        var l = (t, n) => {
            s.onerror = s.onload = null, clearTimeout(h);
            var i = e[r];
            if (delete e[r], s.parentNode && s.parentNode.removeChild(s), i && i.forEach(e => e(n)), t) return t(n);
          },
          h = setTimeout(l.bind(null, void 0, {
            type: "timeout",
            target: s
          }), 12e4);
        s.onerror = l.bind(null, s.onerror), s.onload = l.bind(null, s.onload), c && document.head.appendChild(s);
      }
    }, i.r = e => {
      "undefined" != typeof Symbol && Symbol.toStringTag && Object.defineProperty(e, Symbol.toStringTag, {
        value: "Module"
      }), Object.defineProperty(e, "__esModule", {
        value: !0
      });
    }, i.p = "https://js-agent.newrelic.com/", (() => {
      var e = {
        939: 0,
        866: 0
      };
      i.f.j = (t, r) => {
        var n = i.o(e, t) ? e[t] : void 0;
        if (0 !== n) if (n) r.push(n[2]);else {
          var o = new Promise((r, i) => n = e[t] = [r, i]);
          r.push(n[2] = o);
          var a = i.p + i.u(t),
            s = new Error();
          i.l(a, r => {
            if (i.o(e, t) && (0 !== (n = e[t]) && (e[t] = void 0), n)) {
              var o = r && ("load" === r.type ? "missing" : r.type),
                a = r && r.target && r.target.src;
              s.message = "Loading chunk " + t + " failed.\n(" + o + ": " + a + ")", s.name = "ChunkLoadError", s.type = o, s.request = a, n[1](s);
            }
          }, "chunk-" + t, t);
        }
      };
      var t = (t, r) => {
          var n,
            o,
            [a, s, c] = r,
            u = 0;
          if (a.some(t => 0 !== e[t])) {
            for (n in s) i.o(s, n) && (i.m[n] = s[n]);
            if (c) c(i);
          }
          for (t && t(r); u < a.length; u++) o = a[u], i.o(e, o) && e[o] && e[o][0](), e[o] = 0;
        },
        r = window.webpackChunkNRBA = window.webpackChunkNRBA || [];
      r.forEach(t.bind(null, 0)), r.push = t.bind(null, r.push.bind(r));
    })();
    var o = {};
    (() => {
      "use strict";

      i.r(o);
      var e = i(2325),
        t = i(6562);
      const r = Object.values(e.D);
      function n(e) {
        const n = {};
        return r.forEach(r => {
          n[r] = function (e, r) {
            return !1 !== (0, t.Mt)(r, `${e}.enabled`);
          }(r, e);
        }), n;
      }
      var a = i(2384),
        s = i(909),
        c = i(9557),
        u = i(7022);
      class d extends c.w {
        constructor(e) {
          super(e), this.aggregatedData = {};
        }
        store(e, t, r, n, i) {
          var o = this.getBucket(e, t, r, i);
          return o.metrics = function (e, t) {
            t || (t = {
              count: 0
            });
            return t.count += 1, (0, u.D)(e, function (e, r) {
              t[e] = f(r, t[e]);
            }), t;
          }(n, o.metrics), o;
        }
        merge(e, t, r, n, i) {
          var o = this.getBucket(e, t, n, i);
          if (o.metrics) {
            var a = o.metrics;
            a.count += r.count, (0, u.D)(r, function (e, t) {
              if ("count" !== e) {
                var n = a[e],
                  i = r[e];
                i && !i.c ? a[e] = f(i.t, n) : a[e] = function (e, t) {
                  if (!t) return e;
                  t.c || (t = l(t.t));
                  return t.min = Math.min(e.min, t.min), t.max = Math.max(e.max, t.max), t.t += e.t, t.sos += e.sos, t.c += e.c, t;
                }(i, a[e]);
              }
            });
          } else o.metrics = r;
        }
        storeMetric(e, t, r, n) {
          var i = this.getBucket(e, t, r);
          return i.stats = f(n, i.stats), i;
        }
        getBucket(e, t, r, n) {
          this.aggregatedData[e] || (this.aggregatedData[e] = {});
          var i = this.aggregatedData[e][t];
          return i || (i = this.aggregatedData[e][t] = {
            params: r || {}
          }, n && (i.custom = n)), i;
        }
        get(e, t) {
          return t ? this.aggregatedData[e] && this.aggregatedData[e][t] : this.aggregatedData[e];
        }
        take(e) {
          for (var t = {}, r = "", n = !1, i = 0; i < e.length; i++) t[r = e[i]] = h(this.aggregatedData[r]), t[r].length && (n = !0), delete this.aggregatedData[r];
          return n ? t : null;
        }
      }
      function f(e, t) {
        return null == e ? function (e) {
          e ? e.c++ : e = {
            c: 1
          };
          return e;
        }(t) : t ? (t.c || (t = l(t.t)), t.c += 1, t.t += e, t.sos += e * e, e > t.max && (t.max = e), e < t.min && (t.min = e), t) : {
          t: e
        };
      }
      function l(e) {
        return {
          t: e,
          min: e,
          max: e,
          sos: e * e,
          c: 1
        };
      }
      function h(e) {
        return "object" != typeof e ? [] : (0, u.D)(e, p);
      }
      function p(e, t) {
        return t;
      }
      var g = i(6797),
        v = i(5526),
        m = i(8610);
      var b = i(9252),
        y = i(2053),
        w = i(8283),
        E = i(6625),
        T = i(4329),
        x = i(1509),
        O = i(2650),
        A = i(2374);
      class _ extends x.W {
        constructor(e, t, r) {
          let n = !(arguments.length > 3 && void 0 !== arguments[3]) || arguments[3];
          super(e, t, r), this.completed = new Promise((e, t) => {
            this.resolve = e, this.reject = t;
          }), this.hasAggregator = !1, this.auto = n, n && (0, T.R)(e, r);
        }
        importAggregator() {
          try {
            if (this.hasAggregator || !this.auto) return;
            this.hasAggregator = !0;
            const e = async () => {
              try {
                const {
                  Aggregate: e
                } = await i(4992)(`./${this.featureName}/aggregate`);
                new e(this.agentIdentifier, this.aggregator), this.resolve();
              } catch (e) {
                (0, m.Z)(`Failed to import aggregator class for ${this.featureName}`, e);
              }
            };
            A.v6 ? e() : (0, O.b)(() => e(), !0);
          } catch (e) {
            this.reject(e);
          }
        }
      }
      var S,
        N,
        D,
        P = i(2484);
      class I extends _ {
        constructor(e, t) {
          let r = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, P.t, r), A.il && ((0, E.v)(e), (0, w.B)(e, "firstbyte", (0, y.yf)()), (0, O.K)(() => this.measureDomContentLoaded()), (0, O.b)(() => this.measureWindowLoaded(), !0), this.importAggregator());
        }
        measureWindowLoaded() {
          var t = (0, y.zO)();
          (0, w.B)(this.agentIdentifier, "onload", t + (0, y.os)()), (0, b.p)("timing", ["load", t], void 0, e.D.pageViewTiming, this.ee);
        }
        measureDomContentLoaded() {
          (0, w.B)(this.agentIdentifier, "domContent", (0, y.zO)() + (0, y.os)());
        }
      }
      S = I, N = "featureName", D = P.t, N in S ? Object.defineProperty(S, N, {
        value: D,
        enumerable: !0,
        configurable: !0,
        writable: !0
      }) : S[N] = D;
      var R = i(6998),
        C = i(8544),
        j = i(6382);
      class k extends _ {
        constructor(t, r) {
          var n;
          let i = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          if (super(t, r, j.t, i), n = this, A.il) {
            if (this.pageHiddenTime = (0, R.e)(), this.performanceObserver, this.lcpPerformanceObserver, this.clsPerformanceObserver, this.fiRecorded = !1, "PerformanceObserver" in window && "function" == typeof window.PerformanceObserver) {
              this.performanceObserver = new PerformanceObserver(function () {
                return n.perfObserver(...arguments);
              });
              try {
                this.performanceObserver.observe({
                  entryTypes: ["paint"]
                });
              } catch (e) {}
              this.lcpPerformanceObserver = new PerformanceObserver(function () {
                return n.lcpObserver(...arguments);
              });
              try {
                this.lcpPerformanceObserver.observe({
                  entryTypes: ["largest-contentful-paint"]
                });
              } catch (e) {}
              this.clsPerformanceObserver = new PerformanceObserver(function () {
                return n.clsObserver(...arguments);
              });
              try {
                this.clsPerformanceObserver.observe({
                  type: "layout-shift",
                  buffered: !0
                });
              } catch (e) {}
            }
            this.fiRecorded = !1;
            ["click", "keydown", "mousedown", "pointerdown", "touchstart"].forEach(e => {
              (0, C.iz)(e, function () {
                return n.captureInteraction(...arguments);
              });
            }), (0, R.N)(() => {
              this.pageHiddenTime = (0, y.zO)(), (0, b.p)("docHidden", [this.pageHiddenTime], void 0, e.D.pageViewTiming, this.ee);
            }, !0), (0, C.bP)("pagehide", () => (0, b.p)("winPagehide", [(0, y.zO)()], void 0, e.D.pageViewTiming, this.ee)), this.importAggregator();
          }
        }
        perfObserver(t, r) {
          t.getEntries().forEach(t => {
            "first-paint" === t.name ? (0, b.p)("timing", ["fp", Math.floor(t.startTime)], void 0, e.D.pageViewTiming, this.ee) : "first-contentful-paint" === t.name && (0, b.p)("timing", ["fcp", Math.floor(t.startTime)], void 0, e.D.pageViewTiming, this.ee);
          });
        }
        lcpObserver(t, r) {
          var n = t.getEntries();
          if (n.length > 0) {
            var i = n[n.length - 1];
            if (this.pageHiddenTime < i.startTime) return;
            var o = [i],
              a = this.addConnectionAttributes({});
            a && o.push(a), (0, b.p)("lcp", o, void 0, e.D.pageViewTiming, this.ee);
          }
        }
        clsObserver(t) {
          t.getEntries().forEach(t => {
            t.hadRecentInput || (0, b.p)("cls", [t], void 0, e.D.pageViewTiming, this.ee);
          });
        }
        addConnectionAttributes(e) {
          var t = navigator.connection || navigator.mozConnection || navigator.webkitConnection;
          if (t) return t.type && (e["net-type"] = t.type), t.effectiveType && (e["net-etype"] = t.effectiveType), t.rtt && (e["net-rtt"] = t.rtt), t.downlink && (e["net-dlink"] = t.downlink), e;
        }
        captureInteraction(r) {
          if (r instanceof t.Yu.EV && !this.fiRecorded) {
            var n = Math.round(r.timeStamp),
              i = {
                type: r.type
              };
            this.addConnectionAttributes(i), n <= (0, y.zO)() ? i.fid = (0, y.zO)() - n : n > (0, y.os)() && n <= Date.now() ? (n -= (0, y.os)(), i.fid = (0, y.zO)() - n) : n = (0, y.zO)(), this.fiRecorded = !0, (0, b.p)("timing", ["fi", n, i], void 0, e.D.pageViewTiming, this.ee);
          }
        }
      }
      !function (e, t, r) {
        t in e ? Object.defineProperty(e, t, {
          value: r,
          enumerable: !0,
          configurable: !0,
          writable: !0
        }) : e[t] = r;
      }(k, "featureName", j.t);
      var L = i(4045),
        M = "React",
        H = "Angular",
        z = "AngularJS",
        B = "Backbone",
        F = "Ember",
        $ = "Vue",
        U = "Meteor",
        V = "Zepto",
        G = "Jquery";
      function q() {
        if (!A.il) return [];
        var e = [];
        try {
          (function () {
            try {
              if (window.React || window.ReactDOM || window.ReactRedux) return !0;
              if (document.querySelector("[data-reactroot], [data-reactid]")) return !0;
              for (var e = document.querySelectorAll("body > div"), t = 0; t < e.length; t++) if (Object.keys(e[t]).indexOf("_reactRootContainer") >= 0) return !0;
              return !1;
            } catch (e) {
              return !1;
            }
          })() && e.push(M), function () {
            try {
              return !!window.angular || !!document.querySelector(".ng-binding, [ng-app], [data-ng-app], [ng-controller], [data-ng-controller], [ng-repeat], [data-ng-repeat]") || !!document.querySelector('script[src*="angular.js"], script[src*="angular.min.js"]');
            } catch (e) {
              return !1;
            }
          }() && e.push(z), function () {
            try {
              return !!(window.hasOwnProperty("ng") && window.ng.hasOwnProperty("coreTokens") && window.ng.coreTokens.hasOwnProperty("NgZone")) || !!document.querySelectorAll("[ng-version]").length;
            } catch (e) {
              return !1;
            }
          }() && e.push(H), window.Backbone && e.push(B), window.Ember && e.push(F), window.Vue && e.push($), window.Meteor && e.push(U), window.Zepto && e.push(V), window.jQuery && e.push(G);
        } catch (e) {}
        return e;
      }
      var W = i(9548),
        X = i(9226),
        Z = i(8226);
      const Y = Boolean(A._A?.Worker),
        J = Boolean(A._A?.SharedWorker),
        Q = Boolean(A._A?.navigator?.serviceWorker);
      let K, ee, te;
      var re = i(6034);
      class ne extends _ {
        constructor(e, t) {
          var r;
          let n = arguments.length > 2 && void 0 !== arguments[2] ? arguments[2] : {},
            i = !(arguments.length > 3 && void 0 !== arguments[3]) || arguments[3];
          super(e, t, re.t, i), r = this, this.PfFeatStatusEnum = n, this.singleChecks(), this.eachSessionChecks(), (0, L.X)("record-supportability", function () {
            return r.recordSupportability(...arguments);
          }, this.featureName, this.ee), (0, L.X)("record-custom", function () {
            return r.recordCustom(...arguments);
          }, this.featureName, this.ee), this.importAggregator();
        }
        recordSupportability(e, t) {
          var r = ["sm", e, {
            name: e
          }, t];
          return (0, b.p)("storeMetric", r, null, this.featureName, this.ee), r;
        }
        recordCustom(e, t) {
          var r = ["cm", e, {
            name: e
          }, t];
          return (0, b.p)("storeEventMetrics", r, null, this.featureName, this.ee), r;
        }
        singleChecks() {
          this.recordSupportability(`Generic/Version/${Z.q}/Detected`);
          const {
            loaderType: e
          } = (0, t.OP)(this.agentIdentifier);
          e && this.recordSupportability(`Generic/LoaderType/${e}/Detected`), A.il && (0, O.K)(() => {
            q().forEach(e => {
              this.recordSupportability("Framework/" + e + "/Detected");
            });
          }), W.T.isFileProtocol() && (this.recordSupportability("Generic/FileProtocol/Detected"), W.T.supportabilityMetricSent = !0);
          const r = (0, X.$c)(this.agentIdentifier);
          r.length > 0 && this.recordSupportability("Generic/Obfuscate/Detected"), r.length > 0 && !(0, X.Ng)(r) && this.recordSupportability("Generic/Obfuscate/Invalid"), A.il && this.reportPolyfillsNeeded(), function (e) {
            if (!K) {
              if (Y) {
                K = Worker;
                try {
                  A._A.Worker = r(K, "Dedicated");
                } catch (e) {
                  o(e, "Dedicated");
                }
                if (J) {
                  ee = SharedWorker;
                  try {
                    A._A.SharedWorker = r(ee, "Shared");
                  } catch (e) {
                    o(e, "Shared");
                  }
                } else n("Shared");
                if (Q) {
                  te = navigator.serviceWorker.register;
                  try {
                    A._A.navigator.serviceWorker.register = (t = te, function () {
                      for (var e = arguments.length, r = new Array(e), n = 0; n < e; n++) r[n] = arguments[n];
                      return i("Service", r[1]?.type), t.apply(navigator.serviceWorker, r);
                    });
                  } catch (e) {
                    o(e, "Service");
                  }
                } else n("Service");
                var t;
                return;
              }
              n("All");
            }
            function r(e, t) {
              return new Proxy(e, {
                construct: (e, r) => (i(t, r[1]?.type), new e(...r))
              });
            }
            function n(t) {
              A.v6 || e(`Workers/${t}/Unavailable`);
            }
            function i(t, r) {
              e("module" === r ? `Workers/${t}/Module` : `Workers/${t}/Classic`);
            }
            function o(t, r) {
              e(`Workers/${r}/SM/Unsupported`), (0, m.Z)(`NR Agent: Unable to capture ${r} workers.`, t);
            }
          }(this.recordSupportability.bind(this));
        }
        reportPolyfillsNeeded() {
          this.recordSupportability(`Generic/Polyfill/Promise/${this.PfFeatStatusEnum.PROMISE}`), this.recordSupportability(`Generic/Polyfill/ArrayIncludes/${this.PfFeatStatusEnum.ARRAY_INCLUDES}`), this.recordSupportability(`Generic/Polyfill/ObjectAssign/${this.PfFeatStatusEnum.OBJECT_ASSIGN}`), this.recordSupportability(`Generic/Polyfill/ObjectEntries/${this.PfFeatStatusEnum.OBJECT_ENTRIES}`);
        }
        eachSessionChecks() {
          A.il && (0, C.bP)("pageshow", e => {
            e.persisted && this.recordCustom("Custom/BFCache/PageRestored");
          });
        }
      }
      !function (e, t, r) {
        t in e ? Object.defineProperty(e, t, {
          value: r,
          enumerable: !0,
          configurable: !0,
          writable: !0
        }) : e[t] = r;
      }(ne, "featureName", re.t);
      var ie = i(3916),
        oe = i(3752),
        ae = i(8683),
        se = i.n(ae),
        ce = A._A,
        ue = "fetch-",
        de = "fetch-body-",
        fe = ["arrayBuffer", "blob", "json", "text", "formData"],
        le = ce.Request,
        he = ce.Response,
        pe = "prototype",
        ge = "nr@context";
      const ve = {};
      function me(e) {
        const t = function (e) {
          return (e || oe.ee).get("fetch");
        }(e);
        if (!(le && he && ce.fetch)) return t;
        if (ve[t.debugId]) return t;
        function r(e, r, n) {
          var i = e[r];
          "function" == typeof i && (e[r] = function () {
            var e,
              r = se()(arguments),
              o = {};
            t.emit(n + "before-start", [r], o), o[ge] && o[ge].dt && (e = o[ge].dt);
            var a = i.apply(this, r);
            return t.emit(n + "start", [r, e], a), a.then(function (e) {
              return t.emit(n + "end", [null, e], a), e;
            }, function (e) {
              throw t.emit(n + "end", [e], a), e;
            });
          });
        }
        return ve[t.debugId] = !0, (0, u.D)(fe, function (e, t) {
          r(le[pe], t, de), r(he[pe], t, de);
        }), r(ce, "fetch", ue), t.on("fetch-end", function (e, r) {
          var n = this;
          if (r) {
            var i = r.headers.get("content-length");
            null !== i && (n.rxSize = i), t.emit("fetch-done", [null, r], n);
          } else t.emit("fetch-done", [e], n);
        }), t;
      }
      var be = "nr@original",
        ye = Object.prototype.hasOwnProperty,
        we = !1;
      function Ee(e, t) {
        return e || (e = oe.ee), r.inPlace = function (e, t, n, i, o) {
          n || (n = "");
          var a,
            s,
            c,
            u = "-" === n.charAt(0);
          for (c = 0; c < t.length; c++) s = t[c], Oe(a = e[s]) || (e[s] = r(a, u ? s + n : n, i, s, o));
        }, r.flag = be, r;
        function r(t, r, i, o, a) {
          return Oe(t) ? t : (r || (r = ""), nrWrapper[be] = t, xe(t, nrWrapper, e), nrWrapper);
          function nrWrapper() {
            var s, c, u, d;
            try {
              c = this, s = se()(arguments), u = "function" == typeof i ? i(s, c) : i || {};
            } catch (t) {
              Te([t, "", [s, c, o], u], e);
            }
            n(r + "start", [s, c, o], u, a);
            try {
              return d = t.apply(c, s);
            } catch (e) {
              throw n(r + "err", [s, c, e], u, a), e;
            } finally {
              n(r + "end", [s, c, d], u, a);
            }
          }
        }
        function n(r, n, i, o) {
          if (!we || t) {
            var a = we;
            we = !0;
            try {
              e.emit(r, n, i, t, o);
            } catch (t) {
              Te([t, r, n, i], e);
            }
            we = a;
          }
        }
      }
      function Te(e, t) {
        t || (t = oe.ee);
        try {
          t.emit("internal-error", e);
        } catch (e) {}
      }
      function xe(e, t, r) {
        if (Object.defineProperty && Object.keys) try {
          return Object.keys(e).forEach(function (r) {
            Object.defineProperty(t, r, {
              get: function () {
                return e[r];
              },
              set: function (t) {
                return e[r] = t, t;
              }
            });
          }), t;
        } catch (e) {
          Te([e], r);
        }
        for (var n in e) ye.call(e, n) && (t[n] = e[n]);
        return t;
      }
      function Oe(e) {
        return !(e && e instanceof Function && e.apply && !e[be]);
      }
      function Ae(e, t, r) {
        var n = e[t];
        e[t] = function (e, t) {
          var r = t(e);
          return r[be] = e, xe(e, r, oe.ee), r;
        }(n, r);
      }
      function _e() {
        for (var e = arguments.length, t = new Array(e), r = 0; r < e; ++r) t[r] = arguments[r];
        return t;
      }
      const Se = {};
      function Ne(e) {
        const t = function (e) {
          return (e || oe.ee).get("timer");
        }(e);
        if (Se[t.debugId]) return t;
        Se[t.debugId] = !0;
        var r = Ee(t),
          n = "setTimeout",
          i = "setInterval",
          o = "clearTimeout",
          a = "-start";
        return r.inPlace(A._A, [n, "setImmediate"], "setTimeout-"), r.inPlace(A._A, [i], "setInterval-"), r.inPlace(A._A, [o, "clearImmediate"], "clearTimeout-"), t.on(i + a, function (e, t, n) {
          e[0] = r(e[0], "fn-", null, n);
        }), t.on(n + a, function (e, t, n) {
          this.method = n, this.timerDuration = isNaN(e[1]) ? 0 : +e[1], e[0] = r(e[0], "fn-", this, n);
        }), t;
      }
      const De = {};
      function Pe(e) {
        const t = function (e) {
          return (e || oe.ee).get("raf");
        }(e);
        if (De[t.debugId] || !A.il) return t;
        De[t.debugId] = !0;
        var r = Ee(t);
        return r.inPlace(window, ["requestAnimationFrame", "mozRequestAnimationFrame", "webkitRequestAnimationFrame", "msRequestAnimationFrame"], "raf-"), t.on("raf-start", function (e) {
          e[0] = r(e[0], "fn-");
        }), t;
      }
      const Ie = {};
      function Re(e) {
        const t = function (e) {
          return (e || oe.ee).get("history");
        }(e);
        if (Ie[t.debugId] || !A.il) return t;
        Ie[t.debugId] = !0;
        var r = Ee(t),
          n = window.history && window.history.constructor && window.history.constructor.prototype,
          i = window.history;
        return n && n.pushState && n.replaceState && (i = n), r.inPlace(i, ["pushState", "replaceState"], "-"), t;
      }
      const Ce = {};
      function je(e) {
        const t = function (e) {
          return (e || oe.ee).get("jsonp");
        }(e);
        if (Ce[t.debugId] || !A.il) return t;
        Ce[t.debugId] = !0;
        var r = Ee(t),
          n = /[?&](?:callback|cb)=([^&#]+)/,
          i = /(.*)\.([^.]+)/,
          o = /^(\w+)(\.|$)(.*)$/,
          a = ["appendChild", "insertBefore", "replaceChild"];
        function s(e, t) {
          var r = e.match(o),
            n = r[1],
            i = r[3];
          return i ? s(i, t[n]) : t[n];
        }
        return "addEventListener" in window && (Node && Node.prototype && Node.prototype.appendChild ? r.inPlace(Node.prototype, a, "dom-") : (r.inPlace(HTMLElement.prototype, a, "dom-"), r.inPlace(HTMLHeadElement.prototype, a, "dom-"), r.inPlace(HTMLBodyElement.prototype, a, "dom-"))), t.on("dom-start", function (e) {
          !function (e) {
            if (!e || "string" != typeof e.nodeName || "script" !== e.nodeName.toLowerCase()) return;
            if ("function" != typeof e.addEventListener) return;
            var o = (a = e.src, c = a.match(n), c ? c[1] : null);
            var a, c;
            if (!o) return;
            var u = function (e) {
              var t = e.match(i);
              if (t && t.length >= 3) return {
                key: t[2],
                parent: s(t[1], window)
              };
              return {
                key: e,
                parent: window
              };
            }(o);
            if ("function" != typeof u.parent[u.key]) return;
            var d = {};
            function f() {
              t.emit("jsonp-end", [], d), e.removeEventListener("load", f, (0, C.m$)(!1)), e.removeEventListener("error", l, (0, C.m$)(!1));
            }
            function l() {
              t.emit("jsonp-error", [], d), t.emit("jsonp-end", [], d), e.removeEventListener("load", f, (0, C.m$)(!1)), e.removeEventListener("error", l, (0, C.m$)(!1));
            }
            r.inPlace(u.parent, [u.key], "cb-", d), e.addEventListener("load", f, (0, C.m$)(!1)), e.addEventListener("error", l, (0, C.m$)(!1)), t.emit("new-jsonp", [e.src], d);
          }(e[0]);
        }), t;
      }
      const ke = {};
      function Le(e) {
        const r = function (e) {
          return (e || oe.ee).get("mutation");
        }(e);
        if (ke[r.debugId] || !A.il) return r;
        ke[r.debugId] = !0;
        var n = Ee(r),
          i = t.Yu.MO;
        return i && (window.MutationObserver = function (e) {
          return this instanceof i ? new i(n(e, "fn-")) : i.apply(this, arguments);
        }, MutationObserver.prototype = i.prototype), r;
      }
      const Me = {};
      function He(e) {
        const r = function (e) {
          return (e || oe.ee).get("promise");
        }(e);
        if (Me[r.debugId]) return r;
        Me[r.debugId] = !0;
        var n = oe.c,
          i = Ee(r),
          o = t.Yu.PR;
        return o && function () {
          function e(e) {
            var t = r.context(),
              n = i(e, "executor-", t, null, !1),
              a = new o(n);
            return r.context(a).getCtx = function () {
              return t;
            }, a;
          }
          A._A.Promise = e, Object.defineProperty(A._A.Promise, "name", {
            value: "Promise"
          }), ["all", "race"].forEach(function (e) {
            var t = o[e];
            o[e] = function (n) {
              var i = !1;
              (0, u.D)(n, function (t, r) {
                Promise.resolve(r).then(s("all" === e), s(!1));
              });
              var a = t.apply(o, arguments);
              return o.resolve(a);
              function s(e) {
                return function () {
                  r.emit("propagate", [null, !i], a, !1, !1), i = i || !e;
                };
              }
            };
          }), ["resolve", "reject"].forEach(function (e) {
            var t = o[e];
            o[e] = function (e) {
              var n = t.apply(o, arguments);
              return e !== n && r.emit("propagate", [e, !0], n, !1, !1), n;
            };
          }), o.prototype.catch = function (e) {
            return this.then(null, e);
          }, Object.assign(o.prototype, {
            constructor: {
              value: e
            }
          }), (0, u.D)(Object.getOwnPropertyNames(o), function (t, r) {
            try {
              e[r] = o[r];
            } catch (e) {}
          }), Ae(o.prototype, "then", function (e) {
            return function () {
              var t = this,
                o = _e.apply(this, arguments),
                a = n(t);
              a.promise = t, o[0] = i(o[0], "cb-", a, null, !1), o[1] = i(o[1], "cb-", a, null, !1);
              var s = e.apply(this, o);
              return a.nextPromise = s, r.emit("propagate", [t, !0], s, !1, !1), s;
            };
          }), r.on("executor-start", function (e) {
            e[0] = i(e[0], "resolve-", this, null, !1), e[1] = i(e[1], "resolve-", this, null, !1);
          }), r.on("executor-err", function (e, t, r) {
            e[1](r);
          }), r.on("cb-end", function (e, t, n) {
            r.emit("propagate", [n, !0], this.nextPromise, !1, !1);
          }), r.on("propagate", function (e, t, n) {
            this.getCtx && !t || (this.getCtx = function () {
              if (e instanceof Promise) var t = r.context(e);
              return t && t.getCtx ? t.getCtx() : this;
            });
          }), e.toString = function () {
            return "" + o;
          };
        }(), r;
      }
      const ze = {};
      function Be(e) {
        var t = function (e) {
          return (e || oe.ee).get("events");
        }(e);
        if (ze[t.debugId]) return t;
        ze[t.debugId] = !0;
        var r = Ee(t, !0),
          n = XMLHttpRequest,
          i = "addEventListener",
          o = "removeEventListener";
        function a(e) {
          for (var t = e; t && !t.hasOwnProperty(i);) t = Object.getPrototypeOf(t);
          t && s(t);
        }
        function s(e) {
          r.inPlace(e, [i, o], "-", c);
        }
        function c(e, t) {
          return e[1];
        }
        return "getPrototypeOf" in Object ? (A.il && a(document), a(A._A), a(n.prototype)) : n.prototype.hasOwnProperty(i) && (s(A._A), s(n.prototype)), t.on("addEventListener-start", function (e, t) {
          var n = e[1];
          if (null !== n && ("function" == typeof n || "object" == typeof n)) {
            var i = (0, ie.X)(n, "nr@wrapped", function () {
              var e = {
                object: function () {
                  if ("function" != typeof n.handleEvent) return;
                  return n.handleEvent.apply(n, arguments);
                },
                function: n
              }[typeof n];
              return e ? r(e, "fn-", null, e.name || "anonymous") : n;
            });
            this.wrapped = e[1] = i;
          }
        }), t.on("removeEventListener-start", function (e) {
          e[1] = this.wrapped || e[1];
        }), t;
      }
      const Fe = {};
      function $e(e) {
        var r = e || oe.ee;
        const n = function (e) {
          return (e || oe.ee).get("xhr");
        }(r);
        if (Fe[n.debugId]) return n;
        Fe[n.debugId] = !0, Be(r);
        var i = Ee(n),
          o = t.Yu.XHR,
          a = t.Yu.MO,
          s = t.Yu.PR,
          c = t.Yu.SI,
          u = "readystatechange",
          d = ["onload", "onerror", "onabort", "onloadstart", "onloadend", "onprogress", "ontimeout"],
          f = [],
          l = A._A.XMLHttpRequest.listeners,
          h = A._A.XMLHttpRequest = function (e) {
            var t = new o(e);
            function r() {
              try {
                n.emit("new-xhr", [t], t), t.addEventListener(u, g, (0, C.m$)(!1));
              } catch (e) {
                (0, m.Z)("An error occured while intercepting XHR", e);
                try {
                  n.emit("internal-error", [e]);
                } catch (e) {}
              }
            }
            return this.listeners = l ? [...l, r] : [r], this.listeners.forEach(e => e()), t;
          };
        function p(e, t) {
          i.inPlace(t, ["onreadystatechange"], "fn-", E);
        }
        function g() {
          var e = this,
            t = n.context(e);
          e.readyState > 3 && !t.resolved && (t.resolved = !0, n.emit("xhr-resolved", [], e)), i.inPlace(e, d, "fn-", E);
        }
        if (function (e, t) {
          for (var r in e) t[r] = e[r];
        }(o, h), h.prototype = o.prototype, i.inPlace(h.prototype, ["open", "send"], "-xhr-", E), n.on("send-xhr-start", function (e, t) {
          p(e, t), function (e) {
            f.push(e), a && (v ? v.then(w) : c ? c(w) : (b = -b, y.data = b));
          }(t);
        }), n.on("open-xhr-start", p), a) {
          var v = s && s.resolve();
          if (!c && !s) {
            var b = 1,
              y = document.createTextNode(b);
            new a(w).observe(y, {
              characterData: !0
            });
          }
        } else r.on("fn-end", function (e) {
          e[0] && e[0].type === u || w();
        });
        function w() {
          for (var e = 0; e < f.length; e++) p(0, f[e]);
          f.length && (f = []);
        }
        function E(e, t) {
          return t;
        }
        return n;
      }
      function Ue(e) {
        return Be(e);
      }
      function Ve(e) {
        return me(e);
      }
      function Ge(e) {
        return Re(e);
      }
      function qe(e) {
        return Pe(e);
      }
      function We(e) {
        return Ne(e);
      }
      function Xe(e) {
        return $e(e);
      }
      var Ze,
        Ye = {};
      try {
        Ze = localStorage.getItem("__nr_flags").split(","), console && "function" == typeof console.log && (Ye.console = !0, -1 !== Ze.indexOf("dev") && (Ye.dev = !0), -1 !== Ze.indexOf("nr_dev") && (Ye.nrDev = !0));
      } catch (e) {}
      function Je(e) {
        try {
          Ye.console && Je(e);
        } catch (e) {}
      }
      Ye.nrDev && oe.ee.on("internal-error", function (e) {
        Je(e.stack);
      }), Ye.dev && oe.ee.on("fn-err", function (e, t, r) {
        Je(r.stack);
      }), Ye.dev && (Je("NR AGENT IN DEVELOPMENT MODE"), Je("flags: " + (0, u.D)(Ye, function (e, t) {
        return e;
      }).join(", ")));
      var Qe = i(8322);
      class Ke extends _ {
        constructor(r, n) {
          var i;
          let o = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(r, n, Qe.t, o), i = this, this.skipNext = 0, this.handleErrors = !1, this.origOnerror = A._A?.onerror;
          const a = this,
            s = (0, t.OP)(this.agentIdentifier);
          s.features.err = !0, a.ee.on("fn-start", function (e, t, r) {
            a.handleErrors && (a.skipNext += 1);
          }), a.ee.on("fn-err", function (e, t, r) {
            a.handleErrors && !r[Qe.A] && ((0, ie.X)(r, Qe.A, function () {
              return !0;
            }), this.thrown = !0, tt(r, void 0, a.ee));
          }), a.ee.on("fn-end", function () {
            a.handleErrors && !this.thrown && a.skipNext > 0 && (a.skipNext -= 1);
          }), a.ee.on("internal-error", t => {
            (0, b.p)("ierr", [t, (0, y.zO)(), !0], void 0, e.D.jserrors, a.ee);
          });
          const c = A._A?.onerror;
          A._A.onerror = function () {
            return c && c(...arguments), i.onerrorHandler(...arguments), !1;
          };
          try {
            A._A?.addEventListener("unhandledrejection", t => {
              const r = new Error(`${t.reason}`);
              (0, b.p)("err", [r, (0, y.zO)(), !1, {
                unhandledPromiseRejection: 1
              }], void 0, e.D.jserrors, this.ee);
            });
          } catch (e) {}
          try {
            throw new Error();
          } catch (e) {
            "stack" in e && (We(this.ee), qe(this.ee), "addEventListener" in A._A && Ue(this.ee), s.xhrWrappable && Xe(this.ee), a.handleErrors = !0);
          }
          this.importAggregator();
        }
        onerrorHandler(t, r, n, i, o) {
          try {
            this.skipNext ? this.skipNext -= 1 : tt(o || new et(t, r, n), !0, this.ee);
          } catch (t) {
            try {
              (0, b.p)("ierr", [t, (0, y.zO)(), !0], void 0, e.D.jserrors, this.ee);
            } catch (e) {}
          }
          return "function" == typeof this.origOnerror && this.origOnerror.apply(this, se()(arguments));
        }
      }
      function et(e, t, r) {
        this.message = e || "Uncaught error with no additional information", this.sourceURL = t, this.line = r;
      }
      function tt(t, r, n) {
        var i = r ? null : (0, y.zO)();
        (0, b.p)("err", [t, i], void 0, e.D.jserrors, n);
      }
      !function (e, t, r) {
        t in e ? Object.defineProperty(e, t, {
          value: r,
          enumerable: !0,
          configurable: !0,
          writable: !0
        }) : e[t] = r;
      }(Ke, "featureName", Qe.t);
      var rt = 1;
      function nt(e) {
        var t = typeof e;
        return !e || "object" !== t && "function" !== t ? -1 : e === A._A ? 0 : (0, ie.X)(e, "nr@id", function () {
          return rt++;
        });
      }
      var it = i(9071);
      function ot(e) {
        if ("string" == typeof e && e.length) return e.length;
        if ("object" == typeof e) {
          if ("undefined" != typeof ArrayBuffer && e instanceof ArrayBuffer && e.byteLength) return e.byteLength;
          if ("undefined" != typeof Blob && e instanceof Blob && e.size) return e.size;
          if (!("undefined" != typeof FormData && e instanceof FormData)) try {
            return JSON.stringify(e).length;
          } catch (e) {
            return;
          }
        }
      }
      var at = i(6368);
      class st {
        constructor(e) {
          this.agentIdentifier = e, this.generateTracePayload = this.generateTracePayload.bind(this), this.shouldGenerateTrace = this.shouldGenerateTrace.bind(this);
        }
        generateTracePayload(e) {
          if (!this.shouldGenerateTrace(e)) return null;
          var r = (0, t.DL)(this.agentIdentifier);
          if (!r) return null;
          var n = (r.accountID || "").toString() || null,
            i = (r.agentID || "").toString() || null,
            o = (r.trustKey || "").toString() || null;
          if (!n || !i) return null;
          var a = (0, v.M)(),
            s = (0, v.Ht)(),
            c = Date.now(),
            u = {
              spanId: a,
              traceId: s,
              timestamp: c
            };
          return (e.sameOrigin || this.isAllowedOrigin(e) && this.useTraceContextHeadersForCors()) && (u.traceContextParentHeader = this.generateTraceContextParentHeader(a, s), u.traceContextStateHeader = this.generateTraceContextStateHeader(a, c, n, i, o)), (e.sameOrigin && !this.excludeNewrelicHeader() || !e.sameOrigin && this.isAllowedOrigin(e) && this.useNewrelicHeaderForCors()) && (u.newrelicHeader = this.generateTraceHeader(a, s, c, n, i, o)), u;
        }
        generateTraceContextParentHeader(e, t) {
          return "00-" + t + "-" + e + "-01";
        }
        generateTraceContextStateHeader(e, t, r, n, i) {
          return i + "@nr=0-1-" + r + "-" + n + "-" + e + "----" + t;
        }
        generateTraceHeader(e, t, r, n, i, o) {
          if (!("function" == typeof A._A?.btoa)) return null;
          var a = {
            v: [0, 1],
            d: {
              ty: "Browser",
              ac: n,
              ap: i,
              id: e,
              tr: t,
              ti: r
            }
          };
          return o && n !== o && (a.d.tk = o), btoa(JSON.stringify(a));
        }
        shouldGenerateTrace(e) {
          return this.isDtEnabled() && this.isAllowedOrigin(e);
        }
        isAllowedOrigin(e) {
          var r = !1,
            n = {};
          if ((0, t.Mt)(this.agentIdentifier, "distributed_tracing") && (n = (0, t.P_)(this.agentIdentifier).distributed_tracing), e.sameOrigin) r = !0;else if (n.allowed_origins instanceof Array) for (var i = 0; i < n.allowed_origins.length; i++) {
            var o = (0, at.e)(n.allowed_origins[i]);
            if (e.hostname === o.hostname && e.protocol === o.protocol && e.port === o.port) {
              r = !0;
              break;
            }
          }
          return r;
        }
        isDtEnabled() {
          var e = (0, t.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !!e.enabled;
        }
        excludeNewrelicHeader() {
          var e = (0, t.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !!e.exclude_newrelic_header;
        }
        useNewrelicHeaderForCors() {
          var e = (0, t.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !1 !== e.cors_use_newrelic_header;
        }
        useTraceContextHeadersForCors() {
          var e = (0, t.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !!e.cors_use_tracecontext_headers;
        }
      }
      var ct = i(8675);
      var ut = ["load", "error", "abort", "timeout"],
        dt = ut.length,
        ft = t.Yu.REQ,
        lt = A._A?.XMLHttpRequest;
      class ht extends _ {
        constructor(r, n) {
          let i = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(r, n, ct.t, i);
          const o = (0, t.OP)(this.agentIdentifier);
          o.xhrWrappable && (o.features.xhr = !0, this.dt = new st(this.agentIdentifier), this.handler = (e, t, r, n) => (0, b.p)(e, t, r, n, this.ee), this.wrappedFetch = Ve(this.ee), Xe(this.ee), function (r, n, i, o) {
            function a(e) {
              var t = this;
              t.totalCbs = 0, t.called = 0, t.cbTime = 0, t.end = T, t.ended = !1, t.xhrGuids = {}, t.lastSize = null, t.loadCaptureCalled = !1, t.params = this.params || {}, t.metrics = this.metrics || {}, e.addEventListener("load", function (r) {
                O(t, e);
              }, (0, C.m$)(!1)), it.I && (it.I > 34 || it.I < 10) || e.addEventListener("progress", function (e) {
                t.lastSize = e.loaded;
              }, (0, C.m$)(!1));
            }
            function s(e) {
              this.params = {
                method: e[0]
              }, x(this, e[1]), this.metrics = {};
            }
            function c(e, n) {
              var i = (0, t.DL)(r);
              "xpid" in i && this.sameOrigin && n.setRequestHeader("X-NewRelic-ID", i.xpid);
              var a = o.generateTracePayload(this.parsedOrigin);
              if (a) {
                var s = !1;
                a.newrelicHeader && (n.setRequestHeader("newrelic", a.newrelicHeader), s = !0), a.traceContextParentHeader && (n.setRequestHeader("traceparent", a.traceContextParentHeader), a.traceContextStateHeader && n.setRequestHeader("tracestate", a.traceContextStateHeader), s = !0), s && (this.dt = a);
              }
            }
            function u(e, t) {
              var r = this.metrics,
                i = e[0],
                o = this;
              if (r && i) {
                var a = ot(i);
                a && (r.txSize = a);
              }
              this.startTime = (0, y.zO)(), this.listener = function (e) {
                try {
                  "abort" !== e.type || o.loadCaptureCalled || (o.params.aborted = !0), ("load" !== e.type || o.called === o.totalCbs && (o.onloadCalled || "function" != typeof t.onload)) && o.end(t);
                } catch (e) {
                  try {
                    n.emit("internal-error", [e]);
                  } catch (e) {}
                }
              };
              for (var s = 0; s < dt; s++) t.addEventListener(ut[s], this.listener, (0, C.m$)(!1));
            }
            function d(e, t, r) {
              this.cbTime += e, t ? this.onloadCalled = !0 : this.called += 1, this.called !== this.totalCbs || !this.onloadCalled && "function" == typeof r.onload || this.end(r);
            }
            function f(e, t) {
              var r = "" + nt(e) + !!t;
              this.xhrGuids && !this.xhrGuids[r] && (this.xhrGuids[r] = !0, this.totalCbs += 1);
            }
            function l(e, t) {
              var r = "" + nt(e) + !!t;
              this.xhrGuids && this.xhrGuids[r] && (delete this.xhrGuids[r], this.totalCbs -= 1);
            }
            function h() {
              this.endTime = (0, y.zO)();
            }
            function p(e, t) {
              t instanceof lt && "load" === e[0] && n.emit("xhr-load-added", [e[1], e[2]], t);
            }
            function g(e, t) {
              t instanceof lt && "load" === e[0] && n.emit("xhr-load-removed", [e[1], e[2]], t);
            }
            function v(e, t, r) {
              t instanceof lt && ("onload" === r && (this.onload = !0), ("load" === (e[0] && e[0].type) || this.onload) && (this.xhrCbStart = (0, y.zO)()));
            }
            function m(e, t) {
              this.xhrCbStart && n.emit("xhr-cb-time", [(0, y.zO)() - this.xhrCbStart, this.onload, t], t);
            }
            function b(e) {
              var t,
                r = e[1] || {};
              "string" == typeof e[0] ? t = e[0] : e[0] && e[0].url ? t = e[0].url : A._A?.URL && e[0] && e[0] instanceof URL && (t = e[0].href), t && (this.parsedOrigin = (0, at.e)(t), this.sameOrigin = this.parsedOrigin.sameOrigin);
              var n = o.generateTracePayload(this.parsedOrigin);
              if (n && (n.newrelicHeader || n.traceContextParentHeader)) if ("string" == typeof e[0] || A._A?.URL && e[0] && e[0] instanceof URL) {
                var i = {};
                for (var a in r) i[a] = r[a];
                i.headers = new Headers(r.headers || {}), s(i.headers, n) && (this.dt = n), e.length > 1 ? e[1] = i : e.push(i);
              } else e[0] && e[0].headers && s(e[0].headers, n) && (this.dt = n);
              function s(e, t) {
                var r = !1;
                return t.newrelicHeader && (e.set("newrelic", t.newrelicHeader), r = !0), t.traceContextParentHeader && (e.set("traceparent", t.traceContextParentHeader), t.traceContextStateHeader && e.set("tracestate", t.traceContextStateHeader), r = !0), r;
              }
            }
            function w(e, t) {
              this.params = {}, this.metrics = {}, this.startTime = (0, y.zO)(), this.dt = t, e.length >= 1 && (this.target = e[0]), e.length >= 2 && (this.opts = e[1]);
              var r,
                n = this.opts || {},
                i = this.target;
              "string" == typeof i ? r = i : "object" == typeof i && i instanceof ft ? r = i.url : A._A?.URL && "object" == typeof i && i instanceof URL && (r = i.href), x(this, r);
              var o = ("" + (i && i instanceof ft && i.method || n.method || "GET")).toUpperCase();
              this.params.method = o, this.txSize = ot(n.body) || 0;
            }
            function E(t, r) {
              var n;
              this.endTime = (0, y.zO)(), this.params || (this.params = {}), this.params.status = r ? r.status : 0, "string" == typeof this.rxSize && this.rxSize.length > 0 && (n = +this.rxSize);
              var o = {
                txSize: this.txSize,
                rxSize: n,
                duration: (0, y.zO)() - this.startTime
              };
              i("xhr", [this.params, o, this.startTime, this.endTime, "fetch"], this, e.D.ajax);
            }
            function T(t) {
              var r = this.params,
                n = this.metrics;
              if (!this.ended) {
                this.ended = !0;
                for (var o = 0; o < dt; o++) t.removeEventListener(ut[o], this.listener, !1);
                r.aborted || (n.duration = (0, y.zO)() - this.startTime, this.loadCaptureCalled || 4 !== t.readyState ? null == r.status && (r.status = 0) : O(this, t), n.cbTime = this.cbTime, i("xhr", [r, n, this.startTime, this.endTime, "xhr"], this, e.D.ajax));
              }
            }
            function x(e, t) {
              var r = (0, at.e)(t),
                n = e.params;
              n.hostname = r.hostname, n.port = r.port, n.protocol = r.protocol, n.host = r.hostname + ":" + r.port, n.pathname = r.pathname, e.parsedOrigin = r, e.sameOrigin = r.sameOrigin;
            }
            function O(e, t) {
              e.params.status = t.status;
              var r = function (e, t) {
                var r = e.responseType;
                return "json" === r && null !== t ? t : "arraybuffer" === r || "blob" === r || "json" === r ? ot(e.response) : "text" === r || "" === r || void 0 === r ? ot(e.responseText) : void 0;
              }(t, e.lastSize);
              if (r && (e.metrics.rxSize = r), e.sameOrigin) {
                var n = t.getResponseHeader("X-NewRelic-App-Data");
                n && (e.params.cat = n.split(", ").pop());
              }
              e.loadCaptureCalled = !0;
            }
            n.on("new-xhr", a), n.on("open-xhr-start", s), n.on("open-xhr-end", c), n.on("send-xhr-start", u), n.on("xhr-cb-time", d), n.on("xhr-load-added", f), n.on("xhr-load-removed", l), n.on("xhr-resolved", h), n.on("addEventListener-end", p), n.on("removeEventListener-end", g), n.on("fn-end", m), n.on("fetch-before-start", b), n.on("fetch-start", w), n.on("fn-start", v), n.on("fetch-done", E);
          }(this.agentIdentifier, this.ee, this.handler, this.dt), this.importAggregator());
        }
      }
      !function (e, t, r) {
        t in e ? Object.defineProperty(e, t, {
          value: r,
          enumerable: !0,
          configurable: !0,
          writable: !0
        }) : e[t] = r;
      }(ht, "featureName", ct.t);
      var pt = i(6408),
        gt = i(2628);
      const {
        ADD_EVENT_LISTENER: vt,
        BST_RESOURCE: mt,
        BST_TIMER: bt,
        END: yt,
        FEATURE_NAME: wt,
        FN_END: Et,
        FN_START: Tt,
        learResourceTimings: xt,
        PUSH_STATE: Ot,
        REMOVE_EVENT_LISTENER: At,
        RESOURCE: _t,
        RESOURCE_TIMING_BUFFER_FULL: St,
        START: Nt,
        ORIG_EVENT: Dt
      } = gt;
      class Pt extends _ {
        constructor(r, n) {
          if (super(r, n, wt, !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2]), !A.il) return;
          if (!(window.performance && window.performance.timing && window.performance.getEntriesByType)) return;
          (0, t.OP)(this.agentIdentifier).features.stn = !0;
          const i = this.ee;
          function o(t) {
            if ((0, b.p)(mt, [window.performance.getEntriesByType(_t)], void 0, e.D.sessionTrace, i), window.performance["c" + xt]) try {
              window.performance[At](St, o, !1);
            } catch (t) {} else try {
              window.performance[At]("webkit" + St, o, !1);
            } catch (t) {}
          }
          this.timerEE = We(this.ee), this.rafEE = qe(this.ee), Ge(this.ee), Ue(this.ee), this.ee.on(Tt, function (e, t) {
            e[0] instanceof Dt && (this.bstStart = (0, y.zO)());
          }), this.ee.on(Et, function (t, r) {
            var n = t[0];
            n instanceof Dt && (0, b.p)("bst", [n, r, this.bstStart, (0, y.zO)()], void 0, e.D.sessionTrace, i);
          }), this.timerEE.on(Tt, function (e, t, r) {
            this.bstStart = (0, y.zO)(), this.bstType = r;
          }), this.timerEE.on(Et, function (t, r) {
            (0, b.p)(bt, [r, this.bstStart, (0, y.zO)(), this.bstType], void 0, e.D.sessionTrace, i);
          }), this.rafEE.on(Tt, function () {
            this.bstStart = (0, y.zO)();
          }), this.rafEE.on(Et, function (t, r) {
            (0, b.p)(bt, [r, this.bstStart, (0, y.zO)(), "requestAnimationFrame"], void 0, e.D.sessionTrace, i);
          }), this.ee.on(Ot + Nt, function (e) {
            this.time = (0, y.zO)(), this.startPath = location.pathname + location.hash;
          }), this.ee.on(Ot + yt, function (t) {
            (0, b.p)("bstHist", [location.pathname + location.hash, this.startPath, this.time], void 0, e.D.sessionTrace, i);
          }), (0, pt.W)() ? ((0, b.p)(mt, [window.performance.getEntriesByType("resource")], void 0, e.D.sessionTrace, i), function () {
            var t = new PerformanceObserver((t, r) => {
              var n = t.getEntries();
              (0, b.p)(mt, [n], void 0, e.D.sessionTrace, i);
            });
            try {
              t.observe({
                entryTypes: ["resource"]
              });
            } catch (e) {}
          }()) : vt in window.performance && (window.performance["c" + xt] ? window.performance[vt](St, o, (0, C.m$)(!1)) : window.performance[vt]("webkit" + St, o, (0, C.m$)(!1))), document[vt]("scroll", this.noOp, (0, C.m$)(!1)), document[vt]("keypress", this.noOp, (0, C.m$)(!1)), document[vt]("click", this.noOp, (0, C.m$)(!1)), this.importAggregator();
        }
        noOp(e) {}
      }
      !function (e, t, r) {
        t in e ? Object.defineProperty(e, t, {
          value: r,
          enumerable: !0,
          configurable: !0,
          writable: !0
        }) : e[t] = r;
      }(Pt, "featureName", wt);
      var It = i(755);
      const {
        FEATURE_NAME: Rt,
        START: Ct,
        END: jt,
        BODY: kt,
        CB_END: Lt,
        JS_TIME: Mt,
        FETCH: Ht,
        ADD_EVENT_LISTENER: zt,
        WIN: Bt,
        LOCATION: Ft,
        FN_START: $t,
        CB_START: Ut,
        FN_END: Vt
      } = It;
      class Gt extends _ {
        constructor(e, r) {
          if (super(e, r, Rt, !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2]), !A.il) return;
          const n = (0, t.OP)(this.agentIdentifier);
          if (!Bt[zt] || !n.xhrWrappable) return;
          n.features.spa = !0;
          let i,
            o = 0;
          const a = this.ee.get("tracer"),
            s = je(this.ee);
          const c = function (e) {
              return He(e);
            }(this.ee),
            u = Ue(this.ee),
            d = We(this.ee),
            f = Xe(this.ee),
            l = Ve(this.ee),
            h = Ge(this.ee),
            p = function (e) {
              return Le(e);
            }(this.ee);
          function g(e, t) {
            h.emit("newURL", ["" + Ft, t]);
          }
          function v() {
            o++, i = Ft.hash, this[$t] = (0, y.zO)();
          }
          function m() {
            o--, Ft.hash !== i && g(0, !0);
            var e = (0, y.zO)();
            this[Mt] = ~~this[Mt] + e - this[$t], this[Vt] = e;
          }
          function b(e, t) {
            e.on(t, function () {
              this[t] = (0, y.zO)();
            });
          }
          this.ee.on($t, v), c.on(Ut, v), s.on(Ut, v), this.ee.on(Vt, m), c.on(Lt, m), s.on(Lt, m), this.ee.buffer([$t, Vt, "xhr-resolved"], this.featureName), u.buffer([$t], this.featureName), d.buffer(["setTimeout" + jt, "clearTimeout" + Ct, $t], this.featureName), f.buffer([$t, "new-xhr", "send-xhr" + Ct], this.featureName), l.buffer([Ht + Ct, Ht + "-done", Ht + kt + Ct, Ht + kt + jt], this.featureName), h.buffer(["newURL"], this.featureName), p.buffer([$t], this.featureName), c.buffer(["propagate", Ut, Lt, "executor-err", "resolve" + Ct], this.featureName), a.buffer([$t, "no-" + $t], this.featureName), s.buffer(["new-jsonp", "cb-start", "jsonp-error", "jsonp-end"], this.featureName), b(l, Ht + Ct), b(l, Ht + "-done"), b(s, "new-jsonp"), b(s, "jsonp-end"), b(s, "cb-start"), h.on("pushState-end", g), h.on("replaceState-end", g), Bt[zt]("hashchange", g, (0, C.m$)(!0)), Bt[zt]("load", g, (0, C.m$)(!0)), Bt[zt]("popstate", function () {
            g(0, o > 1);
          }, (0, C.m$)(!0)), this.importAggregator();
        }
      }
      !function (e, t, r) {
        t in e ? Object.defineProperty(e, t, {
          value: r,
          enumerable: !0,
          configurable: !0,
          writable: !0
        }) : e[t] = r;
      }(Gt, "featureName", Rt);
      var qt = i(6486);
      class Wt extends _ {
        constructor(e, t) {
          let r = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, qt.t, r), this.importAggregator();
        }
      }
      !function (e, t, r) {
        t in e ? Object.defineProperty(e, t, {
          value: r,
          enumerable: !0,
          configurable: !0,
          writable: !0
        }) : e[t] = r;
      }(Wt, "featureName", qt.t), new class {
        constructor(t) {
          let r = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : (0, v.ky)(16);
          this.agentIdentifier = r, this.sharedAggregator = new d({
            agentIdentifier: this.agentIdentifier
          }), this.features = {}, this.desiredFeatures = t.features || [], this.desiredFeatures.sort((t, r) => e.p[t.featureName] - e.p[r.featureName]), Object.assign(this, (0, a.j)(this.agentIdentifier, t, t.loaderType || "agent")), this.start();
        }
        get config() {
          return {
            info: (0, t.C5)(this.agentIdentifier),
            init: (0, t.P_)(this.agentIdentifier),
            loader_config: (0, t.DL)(this.agentIdentifier),
            runtime: (0, t.OP)(this.agentIdentifier)
          };
        }
        start() {
          try {
            const e = n(this.agentIdentifier);
            this.desiredFeatures.forEach(t => {
              if (e[t.featureName]) {
                const r = (0, s.Z)(t.featureName);
                r.every(t => e[t]) || (0, m.Z)(`${t.featureName} is enabled but one or more dependent features has been disabled (${JSON.stringify(r)}). This may cause unintended consequences or missing data...`), this.features[t.featureName] = new t(this.agentIdentifier, this.sharedAggregator);
              }
            }), (0, g.Qy)(this.agentIdentifier, this.features, "features");
          } catch (e) {
            return (0, m.Z)("Failed to initialize instrument classes", e), !1;
          }
        }
      }({
        features: [ht, I, k, Pt, ne, Wt, Ke, Gt],
        loaderType: "spa"
      });
    })(), window.NRBA = o;
  })();
})()</script>
<link rel='shortcut icon' href='https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/13/images/favSD.ico' type='image/x-icon' />
<link rel='icon' href='https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/13/images/favSD.ico' type='image/x-icon'>
<link rel='stylesheet' href='https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/f428ac6e436879961ed6bb261fe095ecf4ea09e0/arp.css'>
<link href="//cdn.pendo.io" rel="dns-prefetch" />
<link href="https://cdn.pendo.io" rel="preconnect" crossorigin="anonymous" />
<link rel='dns-prefetch' href='https://smetrics.elsevier.com'>
<script type="3855cf66cf87175080abf308-text/javascript">
        
        window.pageTargeting = {"region":"us-east-1","platform":"sdtech","entitled":false,"crawler":"","journal":"NeuroImage","auth":"AE"};
        window.pageData = {"content":[{"entitlementType":"unsubscribed","format":"MIME-XHTML","id":"sd:article:pii:S1053811914004686","type":"sd:article:JL:scope-abstract","detail":"sd:article:subtype:fla","publicationType":"journal","issn":"1053-8119","volumeNumber":"100","suppl":"C","provider":"elsevier"}],"page":{"businessUnit":"ELS:RP:ST","language":"en","name":"product:journal:article","noTracking":"false","productAppVersion":"abstract-redirected","productName":"SD","type":"CP-CA","environment":"prod","loadTimestamp":1687992313619,"loadTime":""},"visitor":{"accessType":"ae:ANON_GUEST","accountId":"ae:228598","accountName":"ae:ScienceDirect Guests","loginStatus":"anonymous","userId":"ae:12975512","ipAddress":"118.217.148.220","appSessionId":"dbe7ca4f-621f-4070-94c2-93ec0fb035fb"}};
        window.arp = {
          config: {"adobeSuite":"elsevier-sd-prod","arsUrl":"https://ars.els-cdn.com","bosTimeOut":60000,"bosUrl":"https://feedback.recs.d.elsevier.com/raw/events","googleMapsApiKey":"AIzaSyCBYU6I6lrbEU6wQXUEIte3NwGtm3jwHQc","mediaBaseUrl":"https://ars.els-cdn.com/content/image/","reduxLogging":false,"sendToEnhancedReader":true,"strictMode":false,"seamlessAccess":{"enableSeamlessAccess":true,"scriptUrl":"https://unpkg.com/@theidentityselector/thiss-ds@1.0.13/dist/thiss-ds.js","persistenceUrl":"https://service.seamlessaccess.org/ps/","persistenceContext":"seamlessaccess.org","scienceDirectUrl":"https://www.sciencedirect.com","shibAuthUrl":"https://auth.elsevier.com/ShibAuth/institutionLogin"},"universalPdf":{"enableUniversalPdf":true,"universalPdfUrl":"https://static.mendeley.com/view-pdf-component/0.8.9/dist/view-pdf-element.js","integratorId":{"other":8301,"accessbar":10879}},"reaxys":{"url":"https://www.reaxys.com","apiUrl":"https://reaxys-sdlc.reaxys.com","queryBuilderHostPath":"https://www.reaxys.com/reaxys/secured/hopinto.do","origin":"sciencedirect"},"enableOneTrustCookie":false},
          subscriptions: [],
          subscribe: function(cb) {
            var self = this;
            var i = this.subscriptions.push(cb) - 1;
            return function unsubscribe() {
              self.subscriptions.splice(i, 1);
            }
          },
        };
        window.addEventListener('beforeprint', () => pendo.onGuideDismissed());
      </script>
</head>
<body>
<noscript>
      JavaScript is disabled on your browser.
      Please enable JavaScript to use all the features on this page.
      <img src=https://smetrics.elsevier.com/b/ss/elsevier-sd-prod/1/G.4--NS/1687992313628?pageName=sd%3Aproduct%3Ajournal%3Aarticle&c16=els%3Arp%3Ast&c2=sd&v185=img&v33=ae%3AANON_GUEST&c1=ae%3A228598&c12=ae%3A12975512 />
    </noscript>
<a class="anchor sr-only sr-only-focusable u-display-inline anchor-default" href="#screen-reader-main-content" data-reactroot=""><span class="anchor-text">Skip to main content</span></a><a class="anchor sr-only sr-only-focusable u-display-inline anchor-default" href="#screen-reader-main-title" data-reactroot=""><span class="anchor-text">Skip to article</span></a>
<!--[if lt IE 9]>
      <div id="ie8Warning" class="warning">
        <script>function ie8click() {
  const node = document.getElementById('ie8Warning');
  document.cookie = 'ie_warning_state=1';
  node.parentNode.removeChild(node);
}</script>
        <p>Please note that Internet Explorer version 8.x is not supported as of January 1, 2016.
        Please refer to <a href="https://service.elsevier.com/app/answers/detail/a_id/9831">this support page</a> for more information.</p>
        <a class="warning-close" onclick="ie8click()" title="Close IE warning">&times;</a>
      </div>
    <![endif]-->
<div data-iso-key="_0"><div class="App" id="app" data-aa-name="root" data-reactroot=""><div class="page"><div class="sd-flex-container"><div class="sd-flex-content"><header id="gh-cnt"><div id="gh-main-cnt" class="u-flex-center-ver u-position-relative u-padding-s-hor u-padding-l-hor-from-xl"><a id="gh-branding" class="u-flex-center-ver" href="/" aria-label="ScienceDirect home page" data-aa-region="header" data-aa-name="ScienceDirect"><img class="gh-logo" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/24/images/elsevier-non-solus-new-grey.svg" alt="Elsevier logo" height="48" width="54" /><svg xmlns="http://www.w3.org/2000/svg" version="1.1" height="15" viewBox="0 0 190 23" role="img" class="gh-wordmark u-margin-s-left" aria-labelledby="gh-wm-science-direct" focusable="false" aria-hidden="true" alt="ScienceDirect Wordmark"><title id="gh-wm-science-direct">ScienceDirect</title><g><path fill="#EB6500" d="M3.81 6.9c0-1.48 0.86-3.04 3.7-3.04 1.42 0 3.1 0.43 4.65 1.32l0.13-2.64c-1.42-0.63-2.97-0.96-4.78-0.96 -4.62 0-6.6 2.44-6.6 5.45 0 5.61 8.78 6.14 8.78 9.93 0 1.48-1.15 3.04-3.86 3.04 -1.72 0-3.4-0.56-4.72-1.39l-0.36 2.64c1.55 0.76 3.57 1.06 5.15 1.06 4.26 0 6.7-2.48 6.7-5.51C12.59 11.49 3.81 10.76 3.81 6.9M20.27 9.01c0.23-0.13 0.69-0.26 1.72-0.26 1.72 0 2.41 0.3 2.41 1.58h2.38c0-0.36 0-0.79-0.03-1.09 -0.23-1.98-2.15-2.67-4.88-2.67 -3 0-6.7 2.31-6.7 7.76 0 5.22 2.77 7.99 6.63 7.99 1.68 0 3.47-0.36 4.95-1.39l-0.2-2.31c-0.99 0.82-2.84 1.52-4.06 1.52 -2.14 0-4.55-1.71-4.55-5.91C17.93 10.2 20.01 9.18 20.27 9.01"></path><rect x="29.42" y="6.97" fill="#EB6500" width="2.54" height="14.95"></rect><path fill="#EB6500" d="M30.67 0.7c-0.92 0-1.65 0.92-1.65 1.81 0 0.93 0.76 1.85 1.65 1.85 0.89 0 1.68-0.96 1.68-1.88C32.35 1.55 31.56 0.7 30.67 0.7M48.06 14.13c0-5.18-1.42-7.56-6.01-7.56 -3.86 0-6.67 2.77-6.67 7.92 0 4.92 2.97 7.82 6.73 7.82 2.81 0 4.36-0.63 5.68-1.42l-0.2-2.31c-0.89 0.79-2.94 1.55-4.69 1.55 -3.14 0-4.88-1.95-4.88-5.51v-0.49H48.06M39.91 9.18c0.17-0.17 1.29-0.46 1.98-0.46 2.48 0 3.76 0.53 3.86 3.43h-7.46C38.56 10.27 39.71 9.37 39.91 9.18zM58.82 6.57c-2.24 0-3.63 1.12-4.85 2.61l-0.4-2.21h-2.34l0.13 1.19c0.1 0.76 0.13 1.78 0.13 2.97v10.79h2.54V11.88c0.69-0.96 2.15-2.48 2.48-2.64 0.23-0.13 1.29-0.4 2.08-0.4 2.28 0 2.48 1.15 2.54 3.43 0.03 1.19 0.03 3.17 0.03 3.17 0.03 3-0.1 6.47-0.1 6.47h2.54c0 0 0.07-4.49 0.07-6.96 0-1.48 0.03-2.97-0.1-4.46C63.31 7.43 61.49 6.57 58.82 6.57M72.12 9.01c0.23-0.13 0.69-0.26 1.72-0.26 1.72 0 2.41 0.3 2.41 1.58h2.38c0-0.36 0-0.79-0.03-1.09 -0.23-1.98-2.15-2.67-4.88-2.67 -3 0-6.7 2.31-6.7 7.76 0 5.22 2.77 7.99 6.63 7.99 1.68 0 3.47-0.36 4.95-1.39l-0.2-2.31c-0.99 0.82-2.84 1.52-4.06 1.52 -2.15 0-4.55-1.71-4.55-5.91C69.77 10.2 71.85 9.18 72.12 9.01M92.74 14.13c0-5.18-1.42-7.56-6.01-7.56 -3.86 0-6.67 2.77-6.67 7.92 0 4.92 2.97 7.82 6.73 7.82 2.81 0 4.36-0.63 5.68-1.42l-0.2-2.31c-0.89 0.79-2.94 1.55-4.69 1.55 -3.14 0-4.88-1.95-4.88-5.51v-0.49H92.74M84.59 9.18c0.17-0.17 1.29-0.46 1.98-0.46 2.48 0 3.76 0.53 3.86 3.43h-7.46C83.24 10.27 84.39 9.37 84.59 9.18zM103.9 1.98h-7.13v19.93h6.83c7.26 0 9.77-5.68 9.77-10.03C113.37 7.33 110.93 1.98 103.9 1.98M103.14 19.8h-3.76V4.1h4.09c5.38 0 6.96 4.39 6.96 7.79C110.43 16.87 108.19 19.8 103.14 19.8zM118.38 0.7c-0.92 0-1.65 0.92-1.65 1.81 0 0.93 0.76 1.85 1.65 1.85 0.89 0 1.69-0.96 1.69-1.88C120.07 1.55 119.28 0.7 118.38 0.7"></path><rect x="117.13" y="6.97" fill="#EB6500" width="2.54" height="14.95"></rect><path fill="#EB6500" d="M130.2 6.6c-1.62 0-2.87 1.45-3.4 2.74l-0.43-2.37h-2.34l0.13 1.19c0.1 0.76 0.13 1.75 0.13 2.9v10.86h2.54v-9.51c0.53-1.29 1.72-3.7 3.17-3.7 0.96 0 1.06 0.99 1.06 1.22l2.08-0.6V9.18c0-0.03-0.03-0.17-0.06-0.4C132.8 7.36 131.91 6.6 130.2 6.6M145.87 14.13c0-5.18-1.42-7.56-6.01-7.56 -3.86 0-6.67 2.77-6.67 7.92 0 4.92 2.97 7.82 6.73 7.82 2.81 0 4.36-0.63 5.68-1.42l-0.2-2.31c-0.89 0.79-2.94 1.55-4.69 1.55 -3.14 0-4.89-1.95-4.89-5.51v-0.49H145.87M137.72 9.18c0.17-0.17 1.29-0.46 1.98-0.46 2.48 0 3.76 0.53 3.86 3.43h-7.46C136.37 10.27 137.52 9.37 137.72 9.18zM153.23 9.01c0.23-0.13 0.69-0.26 1.72-0.26 1.72 0 2.41 0.3 2.41 1.58h2.38c0-0.36 0-0.79-0.03-1.09 -0.23-1.98-2.14-2.67-4.88-2.67 -3 0-6.7 2.31-6.7 7.76 0 5.22 2.77 7.99 6.63 7.99 1.69 0 3.47-0.36 4.95-1.39l-0.2-2.31c-0.99 0.82-2.84 1.52-4.06 1.52 -2.15 0-4.55-1.71-4.55-5.91C150.89 10.2 152.97 9.18 153.23 9.01M170 19.44c-0.92 0.36-1.72 0.69-2.51 0.69 -1.16 0-1.58-0.66-1.58-2.34V8.95h3.93V6.97h-3.93V2.97h-2.48v3.99h-2.71v1.98h2.71v9.67c0 2.64 1.39 3.73 3.33 3.73 1.15 0 2.54-0.39 3.43-0.79L170 19.44M173.68 5.96c-1.09 0-2-0.87-2-1.97 0-1.1 0.91-1.97 2-1.97s1.98 0.88 1.98 1.98C175.66 5.09 174.77 5.96 173.68 5.96zM173.67 2.46c-0.85 0-1.54 0.67-1.54 1.52 0 0.85 0.69 1.54 1.54 1.54 0.85 0 1.54-0.69 1.54-1.54C175.21 3.13 174.52 2.46 173.67 2.46zM174.17 5.05c-0.09-0.09-0.17-0.19-0.25-0.3l-0.41-0.56h-0.16v0.87h-0.39V2.92c0.22-0.01 0.47-0.03 0.66-0.03 0.41 0 0.82 0.16 0.82 0.64 0 0.29-0.21 0.55-0.49 0.63 0.23 0.32 0.45 0.62 0.73 0.91H174.17zM173.56 3.22l-0.22 0.01v0.63h0.22c0.26 0 0.43-0.05 0.43-0.34C174 3.28 173.83 3.21 173.56 3.22z"></path></g></svg></a><div class="gh-nav-cnt u-hide-from-print"><div class="gh-nav-links-container gh-nav-links-container-h u-hide-from-print gh-nav-content-container"><nav aria-label="links" class="gh-nav gh-nav-links gh-nav-h"><ul class="gh-nav-list u-list-reset"><li class="gh-nav-item gh-move-to-spine"><a class="anchor gh-nav-action anchor-default" href="/browse/journals-and-books" data-aa-region="header" data-aa-name="Journals &amp; Books"><span class="anchor-text">Journals &amp; Books</span></a></li></ul></nav><nav aria-label="utilities" class="gh-nav gh-nav-utilities gh-nav-h"><ul class="gh-nav-list u-list-reset"><li class="gh-move-to-spine gh-help-button gh-help-icon gh-nav-item"><div class="popover" id="gh-help-icon-popover"><div id="popover-trigger-gh-help-icon-popover"><input type="hidden" /><button class="button-link gh-nav-help-icon gh-icon-btn button-link-primary" aria-expanded="false" aria-label="ScienceDirect Support Center links" type="button"><svg role="img" focusable="false" viewBox="0 0 114 128" aria-hidden="true" alt="ScienceDirect help page" width="21.375" height="24" class="icon icon-help gh-icon" aria-label="Help"><path d="m57 8c-14.7 0-28.5 5.72-38.9 16.1-10.38 10.4-16.1 24.22-16.1 38.9 0 30.32 24.68 55 55 55 14.68 0 28.5-5.72 38.88-16.1 10.4-10.4 16.12-24.2 16.12-38.9 0-30.32-24.68-55-55-55zm0 1e1c24.82 0 45 20.18 45 45 0 12.02-4.68 23.32-13.18 31.82s-19.8 13.18-31.82 13.18c-24.82 0-45-20.18-45-45 0-12.02 4.68-23.32 13.18-31.82s19.8-13.18 31.82-13.18zm-0.14 14c-11.55 0.26-16.86 8.43-16.86 18v2h1e1v-2c0-4.22 2.22-9.66 8-9.24 5.5 0.4 6.32 5.14 5.78 8.14-1.1 6.16-11.78 9.5-11.78 20.5v6.6h1e1v-5.56c0-8.16 11.22-11.52 12-21.7 0.74-9.86-5.56-16.52-16-16.74-0.39-0.01-0.76-0.01-1.14 0zm-4.86 5e1v1e1h1e1v-1e1h-1e1z"></path></svg></button></div></div></li><li class="gh-search-toggle gh-nav-item search-button-link search-with-button-link"><a class="anchor button-link-primary gh-nav-action gh-icon-btn search-input-fallback-link anchor-default" href="/search" data-aa-button="search-in-header-opened-from-article" aria-label="Opens ScienceDirect Search"><svg role="img" focusable="false" viewBox="0 0 100 128" aria-hidden="true" alt="Search" width="18.75" height="24" class="icon icon-search gh-icon" aria-label="Search"><path d="m19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6l-26.32-26.32c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96s-21.17 4.25-28.88 11.96c-7.72 7.71-11.97 17.97-11.97 28.88s4.25 21.17 11.97 28.88c7.71 7.71 17.97 11.96 28.88 11.96 9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></a><a class="link-button link-button-small search-button-outline link-button-primary" href="/search" data-aa-button="search-in-header-opened-from-article" aria-label="Opens ScienceDirect Search"><span class="link-button-text">Search</span><svg role="img" focusable="false" viewBox="0 0 100 128" aria-hidden="true" alt="Search" width="18.75" height="24" class="icon icon-search" aria-label="Search"><path d="m19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6l-26.32-26.32c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96s-21.17 4.25-28.88 11.96c-7.72 7.71-11.97 17.97-11.97 28.88s4.25 21.17 11.97 28.88c7.71 7.71 17.97 11.96 28.88 11.96 9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></a></li></ul></nav></div></div><div class="gh-profile-container gh-move-to-spine u-hide-from-print"><a class="link-button link-button-secondary link-button-small u-margin-s-right" href="/user/register?targetURL=%2Fscience%2Farticle%2Fpii%2FS1053811914004686" id="gh-cta-btn" data-aa-region="header" data-aa-name="Register"><span class="link-button-text">Register</span></a><a class="link-button link-button-primary link-button-small" href="/user/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS1053811914004686&amp;from=globalheader" id="gh-signin-btn" data-aa-region="header" data-aa-name="Sign in"><span class="link-button-text">Sign in</span></a></div><div id="gh-mobile-menu" class="mobile-menu u-hide-from-print"><div class="gh-hamburger u-fill-grey7 u-margin-m-left"><button class="button-link u-flex-center-ver button-link-primary" aria-label="Toggle mobile menu" aria-expanded="false" type="button"><svg class="gh-hamburger-svg-el gh-hamburger-closed" role="img" aria-hidden="true" height="18" width="40"><path d="M0 14h40v2H0zm0-7h40v2H0zm0-7h40v2H0z"></path></svg></button></div><div id="gh-overlay" class="mobile-menu-overlay u-overlay u-display-none" role="button" tabindex="-1"></div><div id="gh-drawer" aria-label="Mobile menu" class="" role="navigation"></div></div></div></header><div class="Article Preview" id="mathjax-container"><div class="accessbar-sticky"><div id="screen-reader-main-content"></div><div class="accessbar" role="region" aria-label="Download options and search"><div class="accessbar-label"></div><ul aria-label="PDF Options"><li class="accessbar-item-show-from-initial accessbar-item-show-from-xs accessbar-item-show-from-md RemoteAccess"><a class="link-button RemoteAccessButton accessbar-utility-component accessbar-utility-link link-button-primary" href="/user/institution/login?targetUrl=%2Fscience%2Farticle%2Fpii%2FS1053811914004686%3Ffr%3DRR-2%26ref%3Dpdf_download%26rr%3D7de975694c1da7e1" aria-label="Access through Chungnam National University"><svg role="img" focusable="false" viewBox="0 0 253.88 253.99" width="10" height="20" class="icon icon-seamless-access inst-icon" aria-label="Seamless access"><g><path d="M37.58,97.76h178.73c5.7,0,10.68-3.56,12.46-8.9c1.42-5.34-0.36-11.04-4.99-14.24l-89.37-64.09 c-4.63-3.2-10.68-3.2-14.95,0L30.11,74.61c-3.2,2.49-5.34,6.05-5.34,10.33C24.77,92.06,30.46,97.76,37.58,97.76z M126.95,36.87 l49.49,35.25H77.46L126.95,36.87z"></path><polygon points="109.45,185.4 109.85,127.82 80.25,113.55 80.25,199.67"></polygon><polygon points="174.19,199.26 174.19,114.37 144.99,128.64 144.99,186.21"></polygon><path d="M242.07,226.42c-0.01,0-0.03,0-0.04,0h-10.75v-10.74c0-3.71-3.01-6.72-6.72-6.72l-10.1,0v-86.21 c0-8.2-6.71-14.9-14.9-14.9c-8.2,0-14.9,6.71-14.9,14.9v86.21l-115.42-0.02v-86.19c0-8.2-6.71-14.9-14.9-14.9 c-8.2,0-14.9,6.71-14.9,14.9v86.18l-10.08,0c-3.71,0-6.72,3.01-6.72,6.72v10.74H11.86c-3.71,0-6.72,3.01-6.72,6.72v11.68h243.62 v-11.63C248.77,229.45,245.78,226.43,242.07,226.42z"></path></g></svg><span class="link-button-text">Access throughÂ <strong>Chungnam National Universâ€¦</strong></span></a></li><li class="accessbar-item-hide-from-initial accessbar-item-hide-from-xs accessbar-item-show-from-md PurchasePDF"><button class="button accessbar-utility-component accessbar-utility-button button-anchor" role="button" aria-disabled="false" aria-label="Purchase PDF" type="button"><span class="button-text">Purchase PDF</span></button></li><li class="accessbar-item-hide-from-initial accessbar-item-hide-from-xs accessbar-item-show-from-md Divider"><span class="accessbar-divider"></span></li><li class="accessbar-item-hide-from-initial accessbar-item-hide-from-xs accessbar-item-show-from-md RemoteAccessOther"><a class="link-button accessbar-utility-component accessbar-utility-link link-button-anchor" href="/user/institution/login?targetUrl=%2Fscience%2Farticle%2Fpii%2FS1053811914004686" aria-label="Access through another institution"><span class="link-button-text">Access through another institution</span></a></li><li class="accessbar-item-show-from-initial accessbar-item-show-from-xs accessbar-item-hide-from-md OverflowPopover"><div class="popover accessbar-overflow-popover" id="OverflowPopoverAnchor"><div id="popover-trigger-OverflowPopoverAnchor"><button class="button accessbar-utility-component accessbar-utility-button button-anchor button-icon-right" role="button" aria-disabled="false" aria-label="Other access options" type="button"><span class="button-text">Other access options</span><svg role="img" focusable="false" viewBox="0 0 92 128" height="20" width="17.25" class="icon icon-navigate-down" aria-label="Navigate Down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div></div></li></ul><form class="QuickSearch" action="/search#submit" method="get" aria-label="form"><div class="search-input"><div class="search-input-container search-input-container-no-label"><label class="search-input-label u-hide-visually">Search ScienceDirect</label><input type="search" id="article-quick-search" name="qs" value="" class="search-input-field" aria-label="Search ScienceDirect" aria-describedby="article-quick-search-description-message" placeholder="Search ScienceDirect" /></div><div class="search-input-message-container"><div aria-live="polite" class="search-input-validation-error"></div><div id="article-quick-search-description-message"></div></div></div><button class="button small u-margin-xs-left button-primary" role="button" aria-disabled="false" type="submit" aria-label="Submit search"><span class="button-text"><svg role="img" focusable="false" viewBox="0 0 100 128" height="20" width="18.75" class="icon icon-search" aria-label="Search"><path d="m19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6l-26.32-26.32c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96s-21.17 4.25-28.88 11.96c-7.72 7.71-11.97 17.97-11.97 28.88s4.25 21.17 11.97 28.88c7.71 7.71 17.97 11.96 28.88 11.96 9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></span></button><input type="hidden" name="origin" value="article" /><input type="hidden" name="zone" value="qSearch" /></form></div></div><div class="article-wrapper u-padding-s-top grid row" role="region" aria-label="Article preview"><div class="preview-sidebar u-show-from-lg col-lg-6"><div class="PreviewTableOfContents text-s" aria-label="outline" role="navigation"><h2 class="u-h4 preview-table-of-contents-title">Article preview</h2><ul class="preview-table-of-contents-list"><li id="preview-section-abstract-item" class=""><a class="anchor anchor-default" href="#preview-section-abstract"><span class="anchor-text">Abstract</span></a></li><li id="preview-section-introduction-item" class=""><a class="anchor anchor-default" href="#preview-section-introduction"><span class="anchor-text">Introduction</span></a></li><li id="preview-section-snippets-item" class=""><a class="anchor anchor-default" href="#preview-section-snippets"><span class="anchor-text">Section snippets</span></a></li><li id="preview-section-references-item" class=""><a class="anchor anchor-default" href="#preview-section-references"><span class="anchor-text">References (52)</span></a></li><li id="preview-section-cited-by-item" class=""><a class="anchor anchor-default" href="#preview-section-cited-by"><span class="anchor-text">Cited by (15)</span></a></li><li id="preview-section-recommended-articles-item" class=""><a class="anchor anchor-default" href="#preview-section-recommended-articles"><span class="anchor-text">Recommended articles (6)</span></a></li></ul></div></div><article class="col-lg-12 col-md-16 pad-left pad-right" role="main" lang="en"><div class="Publication" id="publication"><div class="publication-brand u-show-from-sm"><a class="anchor anchor-default" href="/journal/neuroimage" title="Go to NeuroImage on ScienceDirect"><span class="anchor-text"><img class="publication-brand-image" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/f428ac6e436879961ed6bb261fe095ecf4ea09e0/image/elsevier-non-solus.png" alt="Elsevier" /></span></a></div><div class="publication-volume u-text-center"><h2 class="publication-title u-h3" id="publication-title"><a class="anchor publication-title-link anchor-navigation" href="/journal/neuroimage" title="Go to NeuroImage on ScienceDirect"><span class="anchor-text">NeuroImage</span></a></h2><div class="text-xs"><a class="anchor anchor-default" href="/journal/neuroimage/vol/100/suppl/C" title="Go to table of contents for this volume/issue"><span class="anchor-text">Volume 100</span></a>, <!-- -->15 October 2014<!-- -->, Pages 301-315</div></div><div class="publication-cover u-show-from-sm"><a class="anchor anchor-default" href="/journal/neuroimage/vol/100/suppl/C"><span class="anchor-text"><img class="publication-cover-image" src="https://ars.els-cdn.com/content/image/1-s2.0-S1053811914X00157-cov150h.gif" alt="NeuroImage" /></span></a></div></div><div class="PageDivider"></div><h1 id="screen-reader-main-title" class="Head u-font-gulliver u-h2 u-margin-s-ver"><span class="title-text">Non-parametric Bayesian graph models reveal community structure in resting state fMRI</span></h1><div class="Banner" id="banner"><div class="wrapper truncated"><div class="AuthorGroups text-s"><div class="author-group" id="author-group"><span class="sr-only">Author links open overlay panel</span><button class="button-link workspace-trigger button-link-primary" name="bau0005" type="button"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Kasper Winther</span> <span class="text surname">Andersen</span> </span><span class="author-ref" id="baf0005"><sup>a</sup></span> <span class="author-ref" id="baf0010"><sup>b</sup></span><svg role="img" focusable="false" viewBox="0 0 106 128" width="20" height="20" class="icon icon-person react-xocs-author-icon" aria-label="Person"><path d="m11.07 1.2e2l0.84-9.29c1.97-18.79 23.34-22.93 41.09-22.93 17.74 0 39.11 4.13 41.08 22.84l0.84 9.38h10.04l-0.93-10.34c-2.15-20.43-20.14-31.66-51.03-31.66s-48.89 11.22-51.05 31.73l-0.91 10.27h10.03m41.93-102.29c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98s18.24-10.31 18.24-23.98c0-9.9-8.52-18.59-18.24-18.59zm0 52.29c-15.96 0-28-14.48-28-33.67 0-15.36 12.82-28.33 28-28.33s28 12.97 28 28.33c0 19.19-12.04 33.67-28 33.67"></path></svg><svg role="img" focusable="false" viewBox="0 0 102 128" width="20" height="20" class="icon icon-envelope react-xocs-author-icon" aria-label="Envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></button>, <button class="button-link workspace-trigger button-link-primary" name="bau0010" type="button"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Kristoffer H.</span> <span class="text surname">Madsen</span> </span><span class="author-ref" id="baf0010"><sup>b</sup></span><svg role="img" focusable="false" viewBox="0 0 102 128" width="20" height="20" class="icon icon-envelope react-xocs-author-icon" aria-label="Envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></button>, <button class="button-link workspace-trigger button-link-primary" name="bau0015" type="button"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Hartwig Roman</span> <span class="text surname">Siebner</span> </span><span class="author-ref" id="baf0010"><sup>b</sup></span> <span class="author-ref" id="baf0015"><sup>c</sup></span> <span class="author-ref" id="baf0020"><sup>d</sup></span><svg role="img" focusable="false" viewBox="0 0 102 128" width="20" height="20" class="icon icon-envelope react-xocs-author-icon" aria-label="Envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></button>, <button class="button-link workspace-trigger button-link-primary" name="bau0020" type="button"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Mikkel N.</span> <span class="text surname">Schmidt</span> </span><span class="author-ref" id="baf0005"><sup>a</sup></span><svg role="img" focusable="false" viewBox="0 0 102 128" width="20" height="20" class="icon icon-envelope react-xocs-author-icon" aria-label="Envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></button>, <button class="button-link workspace-trigger button-link-primary" name="bau0025" type="button"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Morten</span> <span class="text surname">MÃ¸rup</span> </span><span class="author-ref" id="baf0005"><sup>a</sup></span><svg role="img" focusable="false" viewBox="0 0 102 128" width="20" height="20" class="icon icon-envelope react-xocs-author-icon" aria-label="Envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></button>, <button class="button-link workspace-trigger button-link-primary" name="bau0030" type="button"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Lars Kai</span> <span class="text surname">Hansen</span> </span><span class="author-ref" id="baf0005"><sup>a</sup></span><svg role="img" focusable="false" viewBox="0 0 102 128" width="20" height="20" class="icon icon-envelope react-xocs-author-icon" aria-label="Envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></button></div></div></div><button class="button-link button-link-secondary u-margin-s-ver text-s show-more-button" id="show-more-btn" type="button" data-aa-button="icon-expand"><span class="button-link-text">Show more</span><svg role="img" focusable="false" viewBox="0 0 92 128" height="20" width="17.25" class="icon icon-navigate-down" aria-label="Navigate Down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="banner-options u-padding-xs-bottom text-s"><button class="button-link AddToMendeley u-margin-s-right u-show-inline-from-md button-link-primary" type="button"><svg role="img" focusable="false" viewBox="0 0 86 128" height="16" width="16" class="icon icon-plus" aria-label="Plus"><path d="m48 58v-38h-1e1v38h-38v1e1h38v38h1e1v-38h38v-1e1z"></path></svg><span class="button-link-text">Add to Mendeley</span></button><div class="Social u-display-inline-block" id="social"><div class="popover social-popover" id="social-popover"><div id="popover-trigger-social-popover"><button class="button-link u-margin-s-right button-link-primary" aria-expanded="false" aria-haspopup="true" type="button"><svg role="img" focusable="false" viewBox="0 0 128 128" height="16" width="16" class="icon icon-share" aria-label="Share"><path d="m9e1 112c-6.62 0-12-5.38-12-12s5.38-12 12-12 12 5.38 12 12-5.38 12-12 12zm-66-36c-6.62 0-12-5.38-12-12s5.38-12 12-12 12 5.38 12 12-5.38 12-12 12zm66-6e1c6.62 0 12 5.38 12 12s-5.38 12-12 12-12-5.38-12-12 5.38-12 12-12zm0 62c-6.56 0-12.44 2.9-16.48 7.48l-28.42-15.28c0.58-1.98 0.9-4.04 0.9-6.2s-0.32-4.22-0.9-6.2l28.42-15.28c4.04 4.58 9.92 7.48 16.48 7.48 12.14 0 22-9.86 22-22s-9.86-22-22-22-22 9.86-22 22c0 1.98 0.28 3.9 0.78 5.72l-28.64 15.38c-4.02-4.34-9.76-7.1-16.14-7.1-12.14 0-22 9.86-22 22s9.86 22 22 22c6.38 0 12.12-2.76 16.14-7.12l28.64 15.38c-0.5 1.84-0.78 3.76-0.78 5.74 0 12.14 9.86 22 22 22s22-9.86 22-22-9.86-22-22-22z"></path></svg><span class="button-link-text">Share</span></button></div></div></div><div class="ExportCitation u-display-inline-block" id="export-citation"><div class="popover export-citation-popover" id="export-citation-popover"><div id="popover-trigger-export-citation-popover"><button class="button-link button-link-primary" aria-expanded="false" aria-haspopup="true" type="button"><svg role="img" focusable="false" viewBox="0 0 106 128" height="16" width="16" class="icon icon-cited-by-66" aria-label="Cited By"><path xmlns="http://www.w3.org/2000/svg" d="m2 58.78v47.22h44v-42h-34v-5.22c0-18.5 17.08-26.78 34-26.78v-1e1c-25.9 0-44 15.12-44 36.78zm1e2 -26.78v-1e1c-25.9 0-44 15.12-44 36.78v47.22h44v-42h-34v-5.22c0-18.5 17.08-26.78 34-26.78z"></path></svg><span class="button-link-text">Cite</span></button></div></div></div></div></div><div class="ArticleIdentifierLinks u-margin-xs-bottom text-xs" id="article-identifier-links"><a class="anchor doi anchor-default anchor-external-link" href="https://doi.org/10.1016/j.neuroimage.2014.05.083" target="_blank" rel="noreferrer noopener" aria-label="Persistent link using digital object identifier" title="Persistent link using digital object identifier"><span class="anchor-text">https://doi.org/10.1016/j.neuroimage.2014.05.083</span></a><a class="anchor rights-and-content anchor-default anchor-external-link" href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S1053811914004686&amp;orderBeanReset=true" target="_blank" rel="noreferrer noopener"><span class="anchor-text">Get rights and content</span></a></div><section class="ReferencedArticles"></section><section class="ReferencedArticles"></section><div id="preview-section-abstract"><div class="PageDivider"></div><div class="Abstracts u-font-gulliver text-s" id="abstracts"><div class="abstract author" id="ab0005"><h2 class="section-title u-h4 u-margin-l-top u-margin-xs-bottom">Abstract</h2><div id="as0005"><p id="sp0005">Modeling of resting state functional magnetic resonance imaging (rs-fMRI) data using network models is of increasing interest. It is often desirable to group nodes into clusters to interpret the communication patterns between nodes. In this study we consider three different nonparametric Bayesian models for node clustering in complex networks. In particular, we test their ability to predict unseen data and their ability to reproduce clustering across datasets. The three generative models considered are the Infinite Relational Model (IRM), Bayesian Community Detection (BCD), and the Infinite Diagonal Model (IDM). The models define probabilities of generating links within and between clusters and the difference between the models lies in the restrictions they impose upon the between-cluster link probabilities. IRM is the most flexible model with no restrictions on the probabilities of links between clusters. BCD restricts the between-cluster link probabilities to be strictly lower than within-cluster link probabilities to conform to the community structure typically seen in social networks. IDM only models a single between-cluster link probability, which can be interpreted as a background noise probability. These probabilistic models are compared against three other approaches for node clustering, namely Infomap, Louvain modularity, and hierarchical clustering. Using 3 different datasets comprising healthy volunteers&#x27; rs-fMRI we found that the BCD model was in general the most predictive and reproducible model. This suggests that rs-fMRI data exhibits community structure and furthermore points to the significance of modeling heterogeneous between-cluster link probabilities.</p></div></div></div></div><div id="preview-section-introduction"><div class="PageDivider"></div><div class="Introduction u-font-gulliver text-s u-margin-l-ver"><h2 class="u-h4 u-margin-s-bottom">Introduction</h2><section id="s0005"><p id="p0005">Analysis of resting state functional magnetic resonance imaging (rs-fMRI) has emerged as a powerful research tool to study whole-brain functional connectivity. Since rs-fMRI provides information about intrinsic fluctuations in functional connectivity within and among brain networks, many conventional analysis schemes applied in task-related fMRI studies are irrelevant. Hence, a number of new techniques have been developed based on identification of stable spatio-temporal multivariate structure in the brain wide set of blood oxygen level dependent (BOLD) time series.</p><p id="p0010">Using correlation methods or flexible multivariate techniques like independent component analysis (ICA) it has been shown that the BOLD signals of distant brain regions are coordinated suggesting interaction as they form so-called resting-state networks. The number and precise definition of these networks are controversial but several networks are broadly accepted, including the default mode network, motor network, visual network, fronto-parietal, and dorsal attention network (Damoiseaux et al., 2006). In addition to signals reflecting neuronal activity, the BOLD signal may be contaminated by physiological noise stemming from respiratory and cardiac cycles and head motion (Birn et al., 2006, Power et al., 2014).</p><p id="p0015">Complex network analysis is a very active research field (BarabÃ¡si, 2003) that has already found application in neuroimaging and in modeling resting state connectivity (Bullmore and Bassett, 2011, Sporns, 2011). The basic object is the â€˜network graphâ€™. When applied to neuroimage analysis the network graph is formed by brain regions represented as nodes. Nodes are connected by a link if brain regions are co-activated above a certain threshold. In rs-fMRI co-activation is often measured simply by calculating correlation between time series.</p><p id="p0020">Network structure can be studied at many levels, from local motifs to global features like scale free link distributions signifying long-range coordination (van den Heuvel et al., 2008). Likewise, dense connections between high degree nodes are referred to as â€˜rich club organizationâ€™ (van den Heuvel and Sporns, 2011). At the intermediate level we may identify clusters of highly linked nodes, i.e., high within-cluster link density and low link density to nodes in other clusters. By analogy to social networks such groups are referred to as <em>communities</em>. The presence of community structure in a network can be quantified by the global <em>modularity</em> index (Newman, 2006). Modularity can also be used to identify communities, i.e., by clustering nodes such that the modularity index is maximized (Lehmann and Hansen, 2007, Newman, 2006). Bassett et al. (2011) showed that â€˜flexibilityâ€™, a measure for the number of cluster-assignment changes for nodes in a modularity optimized node-partition across time, is predictive for the amount of learning in a motor task in a subsequent session. Stevens et al. (2012) showed that modularity predicts visual working memory capacity, and Meunier et al. (2009) found that modularity is reduced during normal aging. Likewise, evidence is emerging that global modularity can be used as a bio-marker. For instance patients with childhood-onset schizophrenia have reduced modularity of their resting state networks (Alexander-Bloch et al., 2010). However, focusing on modularity as the single summary of a complex network may be overly simplistic as the modularity measure does not account for variability in the inter-linking relations between functional clusters. Hence, modularity driven clustering might not reveal all salient aspects of community structure in a network. Indeed, modularity has been criticized for its lack of flexibility as a measure of community structure (Fortunato and BarthÃ©lemy, 2007).</p><p id="p0025">A better understanding of this important mid-level structure in brain networks requires methods that can capture more informative representations of community structure. For this we turn to a family of expressive generative network models. Relational models are statistical generalizations of graph clustering that not only consider the within-cluster density but also take the specific relations between clusters into consideration. The Infinite Relational Model (IRM) (Kemp et al., 1999, Xu et al., 2006) is a non-parametric generalization of the stochastic block model (Nowicki and Snijders, 2001) for inference of such generalized group structure in complex networks. As the IRM representation considers both linking within and between groups, a highly inter-linked group of nodes could in fact be clustered in different groups if they link in different ways to other clusters, i.e., the IRM can infer more general group structures beyond the conventional community structure. An additional feature of the IRM type of model is that it conveniently allows for analysis of multi-graph networks, which for neuroimaging data could be graphs from multiple sessions or subjects. For multi-subject analysis one could look for a common node clustering structure over subjects but allow individual subject cluster linking densities (MÃ¸rup et al., 2010) or test the hypothesis that both clustering and link structure are shared between all subjects (Andersen et al., 2012b).</p><p id="p0030">A constrained variant of the IRM representing the community structure of graphs in the sense of grouping highly connected node sets was proposed recently by MÃ¸rup and Schmidt (2012). The Bayesian Community Detection (BCD) scheme restricts the between-cluster link densities to be strictly lower than within-cluster link densities, thus constraining the more general IRM to conform with the notion of a community in a social network. Another constraint is introduced by the so-called Infinite Diagonal Model (IDM) (MÃ¸rup and Schmidt, 2012, Schmidt and MÃ¸rup, 2013). The IDM allows for differential within-cluster link densities but models only a single between-cluster density and as such the variability in the link densities between clusters is neglected when inferring the clustering structure. Since the between-cluster link density is shared across clusters, it can be thought of as a background-noise density.</p><p id="p0035">It should be noted that certain metrical properties can be expected when basing the graph on simple time series correlation, thereby assuming stationarity. If node <em>A</em> is highly correlated with node <em>B</em>, and <em>B</em> is highly correlated with <em>C</em>, then there is a lower limit on the correlation between nodes <em>A</em> and <em>C</em> which can be inferred by the triangle inequality (Zalesky et al., 2012). This bound will support the formation of community structure, as in social relations: â€˜Friends of friends are friendsâ€™, however, we also note that by thresholding the correlation, the impact on the community structure of these geometrical constraints is non-trivial.</p><p id="p0040">Spatial grouping of brain regions by similarity of BOLD time series as pursued in the present work can be seen as complementary to classical approaches to spatial grouping such as time series clustering (Goutte et al., 1999) and independent component analysis (ICA) (McKeown et al., 1998, McKeown et al., 2003). Compared with conventional clustering, the relational modeling approach has the advantage that clusters are formed by considering the connectivity patterns both within and between clusters, and furthermore relational models avoid the formation of a group prototype, hence allow for more flexible group structures to be found (Kemp et al., 1999). The use of ICA is based on assumptions of independence either in spatial or temporal dimensions, which can be questioned in the resting state as it has been observed that components are negatively correlated in time and have extensive overlaps in space (Fox et al., 2005).</p><p id="p0045">In this study, we apply the above-mentioned community detection schemes to rs-fMRI data acquired in three cohorts of healthy volunteers and investigate to which degree functional brain networks as measured by rs-fMRI exhibit community structure. The three Bayesian relational methods, i.e. IRM, BCD, and IDM, for inference of group structure in complex networks differ only in the way the link probabilities between clusters are modeled. The rich link structures of the relational models can be seen as a way of inferring functional integration at the inter-community level as discussed in Hagmann et al. (2008) and Sporns (2013). We evaluate the performance of these models with respect to their ability to predict out-of-sample data (predictability) and the robustness of their clustering under re-sampling of data (reproducibility) using the NPAIRS split-half framework (Strother et al., 2002). The evaluation is carried out on three datasets from different sites and the models are evaluated both within and between sites for several thresholds of the correlation matrices. In addition, we compare the three models with three other methods for grouping nodes into clusters, namely Infomap, Louvain modularity, and hierarchical clustering. The work in this paper builds on work presented in Andersen et al. (2012b).</p></section></div></div><div id="preview-section-snippets"><div class="PageDivider"></div><div class="Snippets u-font-gulliver text-s"><h2 class="u-h4 u-margin-l-ver">Section snippets</h2><section><section id="s0010"><h2 class="section-title u-h4 u-margin-l-top u-margin-xs-bottom">Methods</h2><p id="p0050">For generality we investigate three rs-fMRI datasets. One dataset acquired locally at the Danish Research Centre for Magnetic Resonance (Copenhagen) and two other rs-fMRI datasets publicly available in the FCON1000 database (Biswal et al., 2010) (viz., the â€˜Beijingâ€™ and the â€˜Leipzigâ€™ datasets).</p></section></section><section><section id="s0095"><section id="s0100"><h2 class="section-title u-h4 u-margin-l-top u-margin-xs-bottom">Estimated clusters</h2><p id="p0205">We thresholded the graphs to maintain the top 8% correlations. The threshold corresponds to a mean (std) p-value across subjects of 4.75<!-- -->Â <!-- -->âˆ—<!-- -->Â <!-- -->10<sup>âˆ’<!-- -->Â <!-- -->5</sup>(1.80<!-- -->Â <!-- -->âˆ—<!-- -->Â <!-- -->10<sup>âˆ’<!-- -->Â <!-- -->4</sup>). The reproducibility between solutions found with different restarts was measured as the NMI between samples with the highest value of the posterior distribution for each run. This was done within all three methods and between the methods and results are shown in TableÂ 1 along with the number of clusters estimated by each of the methods. For</p></section></section></section><section><section id="s0115"><h2 class="section-title u-h4 u-margin-l-top u-margin-xs-bottom">Discussion and conclusion</h2><p id="p0255">Our aim was to explore statistical models for finding structure in networks at the intermediate level. Accumulated evidence points to the importance of community structure in brain networks, hence, we tested three statistical link models, which differed in terms of the different restrictions that were imposed on how nodes are clustered. The IRM is a very flexible representation for graph clustering, in which nodes can be grouped together without having a high link density among them. The BCD is </p></section></section><section><section id="ac0005"><h2 id="st0140" class="u-h4 u-margin-l-top u-margin-xs-bottom">Acknowledgment</h2><p id="p0305">This work is funded by a project grant from the <span id="gts0005">Lundbeck Foundation</span> to Hartwig Siebner (grant-nr <span>R48 A4846</span>). The Magnetom Trio MR scanner was donated by the Simon Spies Foundation.</p></section></section></div></div><div class="Tail text-s"></div><div id="preview-section-references"><div class="paginatedReferences u-font-gulliver text-s"><div class="PageDivider"></div><header><h2 class="u-h4 u-margin-l-ver"><span>References</span><span> (52)</span></h2></header><ul><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>J. </span>Ashburner</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S1053811905001102" target="_self"><span class="anchor-text">Unified segmentation</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">NeuroImage</h3></div><div class="series">(2005)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>R.M. </span>Birn</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S1053811906001248" target="_self"><span class="anchor-text">Separating respiratory-variation-related fluctuations from neuronal-activity-related fluctuations in fMRI</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">NeuroImage</h3></div><div class="series">(2006)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>M.S. </span>Dagli</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S1053811998904244" target="_self"><span class="anchor-text">Localization of cardiac-induced signal change in fMRI</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">Neuroimage</h3></div><div class="series">(1999)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="author u-font-sans"><span>S. </span>Fortunato</span><h3><a class="anchor title anchor-default" href="/science/article/pii/S0370157309002841" target="_self"><span class="anchor-text">Community detection in graphs</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">Phys. Rep.</h3></div><div class="series">(2010)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>C. </span>Goutte</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S1053811998903913" target="_self"><span class="anchor-text">On clustering fMRI time series</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">NeuroImage</h3></div><div class="series">(1999)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>T.E. </span>Lund</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S105381190500501X" target="_self"><span class="anchor-text">Non-white noise in fMRI: does modelling have an impact?</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">NeuroImage</h3></div><div class="series">(2006)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>M.J. </span>McKeown</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S0959438803001338" target="_self"><span class="anchor-text">Independent component analysis of functional MRI: what is signal and what is noise?</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">Curr. Opin. Neurobiol.</h3></div><div class="series">(2003)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>D. </span>Meunier</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S1053811908011026" target="_self"><span class="anchor-text">Age-related changes in modular organization of human brain functional networks</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">NeuroImage</h3></div><div class="series">(2009)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="author u-font-sans"><span>R.C. </span>Oldfield</span><h3><a class="anchor title anchor-default" href="/science/article/pii/0028393271900674" target="_self"><span class="anchor-text">The assessment and analysis of handedness: the Edinburgh inventory</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">Neuropsychologia</h3></div><div class="series">(1971)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>J.D. </span>Power</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S0896627311007926" target="_self"><span class="anchor-text">Functional network organization of the human brain</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">Neuron</h3></div><div class="series">(2011)</div></span></li></ul><div class="u-display-none"><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>J.D. </span>Power</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S1053811913009117" target="_self"><span class="anchor-text">Methods to detect, characterize, and remove motion artifact in resting state fMRI</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">NeuroImage</h3></div><div class="series">(2014)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>M. </span>Rubinov</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S105381190901074X" target="_self"><span class="anchor-text">Complex network measures of brain connectivity: uses and interpretations</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">NeuroImage</h3></div><div class="series">(2010)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>A.M. </span>Smith</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S1053811999904354" target="_self"><span class="anchor-text">Investigation of low frequency drift in fMRI signal</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">Neuroimage</h3></div><div class="series">(1999)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="author u-font-sans"><span>O. </span>Sporns</span><h3><a class="anchor title anchor-default" href="/science/article/pii/S0959438812001894" target="_self"><span class="anchor-text">Network attributes for segregation and integration in the human brain</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">Curr. Opin. Neurobiol.</h3></div><div class="series">(2013)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>S.C. </span>Strother</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S1053811901910341" target="_self"><span class="anchor-text">The quantitative evaluation of functional neuroimaging experiments: the NPAIRS data analysis framework</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">NeuroImage</h3></div><div class="series">(2002)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>N. </span>Tzourio-Mazoyer</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S1053811901909784" target="_self"><span class="anchor-text">Automated anatomical labeling of activations in SPM using a macroscopic anatomical parcellation of the MNI MRI single-subject brain</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">Neuroimage</h3></div><div class="series">(2002)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>M.P. </span>van den Heuvel</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S1053811908009130" target="_self"><span class="anchor-text">Small-world and scale-free organization of voxel-based resting-state functional connectivity in the human brain</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">NeuroImage</h3></div><div class="series">(2008)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>G. </span>Varoquaux</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S1053811913003340" target="_self"><span class="anchor-text">Learning and comparing functional connectomes across subjects</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">NeuroImage</h3></div><div class="series">(2013)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>A. </span>Zalesky</span><em> et al.</em></span><h3><a class="anchor title anchor-default" href="/science/article/pii/S1053811912001784" target="_self"><span class="anchor-text">On the use of correlation as a measure of network connectivity</span></a></h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">Neuroimage</h3></div><div class="series">(2012)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="author u-font-sans"><span>D.J. </span>Aldous</span><h3 class="title">Exchangeability and related topics</h3><span class="host u-clr-grey6 u-font-sans"><div class="series">(1985)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>A.F. </span>Alexander-Bloch</span><em> et al.</em></span><h3 class="title">Disrupted modularity and local connectivity of brain functional networks in childhood-onset schizophrenia</h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">Front. Syst. Neurosci.</h3></div><div class="series">(2010)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>K.W. </span>Andersen</span><em> et al.</em></span><h3 class="title">Identification of functional clusters in the striatum using infinite relational modeling</h3><span class="host u-clr-grey6 u-font-sans"></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>K.W. </span>Andersen</span><em> et al.</em></span><h3 class="title">Identifying modular relations in complex brain networks</h3><span class="host u-clr-grey6 u-font-sans"></span></li><li class="bib-reference u-margin-s-bottom"><span class="author u-font-sans"><span>A.L. </span>BarabÃ¡si</span><h3 class="title">Linked: How Everything is Connected to Everything Else and What It Means for Business, Science, and Everyday Life</h3><span class="host u-clr-grey6 u-font-sans"><div class="series">(2003)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>D.S. </span>Bassett</span><em> et al.</em></span><h3 class="title">Dynamic reconfiguration of human brain networks during learning</h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">Proc. Natl. Acad. Sci. U. S. A.</h3></div><div class="series">(2011)</div></span></li><li class="bib-reference u-margin-s-bottom"><span class="u-font-sans"><span class="author u-font-sans"><span>B.B. </span>Biswal</span><em> et al.</em></span><h3 class="title">Toward discovery science of human brain function</h3><span class="host u-clr-grey6 u-font-sans"><div class="series"><h3 class="title">Proc. Natl. Acad. Sci. U. S. A.</h3></div><div class="series">(2010)</div></span></li></div><button class="button-alternative button-alternative-secondary text-m u-font-sans u-margin-l-bottom" type="button" id="show-more-refs-btn"><svg role="img" focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down u-fill-grey8" aria-label="Navigate Down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg><span class="button-alternative-text">View more references</span></button></div></div><div id="preview-section-cited-by"><section aria-label="Cited by" class="ListArticles preview"><div class="PageDivider"></div><header id="citing-articles-header"><h2 class="u-h4 u-margin-l-ver u-font-gulliver">Cited by (15)</h2></header><div aria-describedby="citing-articles-header"><div class="citing-articles u-margin-l-bottom"><ul><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-gulliver" id="citing-articles-article-0-title"><a class="anchor anchor-default" href="/science/article/pii/S105381192100447X" target="_self"><span class="anchor-text">Using connectomics for predictive assessment of brain parcellations</span></a></h3><div class="text-s">2021, NeuroImage</div><div class="CitedSection u-margin-s-top"><div class="u-margin-s-left"><div class="cite-header text-s u-text-italic u-font-sans">Citation Excerpt :</div><p class="u-font-gulliver text-xs">In contrast to cluster validity measures that quantify similarity within the cluster to the similarity of other clusters Arslan et al. (2018); Dornas and Braun (2018) or statistical prediction based on feature maps or time-series Thirion et al. (2014); Wang et al. (2018), our proposed approach quantifies how well the network organization is preserved within the representation induced by a parcellation. In particular, our statistical prediction framework poses quantification of parcellation quality as a link-prediction problem Ambrosen et al. (2014); Andersen et al. (2014); Clauset et al. (2008); Liben-Nowell and Kleinberg (2007). A parcellation is thereby assessed by its ability to characterize brain connectivity data that may derive from an orthogonal modality to that from which the parcellation was defined.</p></div></div></div><div class="buttons text-s"><button class="button-link button-link-secondary" data-aa-button="sd:product:journal:article:location=citing-articles:type=view-details" aria-describedby="citing-articles-article-0-title" aria-controls="citing-articles-article-0" aria-expanded="false" type="button"><span class="button-link-text">Show abstract</span><svg role="img" focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down" aria-label="Navigate Down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-gulliver text-s" id="reference-abstract"><div class="u-margin-ver-m"><p id="sp0001">The organization of the human brain remains elusive, yet is of great importance to the mechanisms of integrative brain function. At the macroscale, its structural and functional interpretation is conventionally assessed at the level of cortical units. However, the definition and validation of such cortical parcellations are problematic due to the absence of a true gold standard. We propose a framework for quantitative evaluation of brain parcellations via statistical prediction of connectomics data. Specifically, we evaluate the extent in which the network representation at the level of cortical units (defined as parcels) accounts for high-resolution brain connectivity. Herein, we assess the pertinence and comparative ranking of ten existing parcellation atlases to account for functional (FC) and structural connectivity (SC) data based on data from the Human Connectome Project (HCP), and compare them to data-driven as well as spatially-homogeneous geometric parcellations including geodesic parcellations with similar size distributions as the atlases. We find substantial discrepancy in parcellation structures that well characterize FC and SC and differences in what well represents an individualâ€™s functional connectome when compared against the FC structure that is preserved across individuals. Surprisingly, simple spatial homogenous parcellations generally provide good representations of both FC and SC, but are inferior when their within-parcellation distribution of individual parcel sizes is matched to that of a valid atlas. This suggests that the choice of fine grained and coarse representations used by existing atlases are important. However, we find that resolution is more critical than the exact border location of parcels.</p></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-gulliver" id="citing-articles-article-1-title"><a class="anchor anchor-default" href="/science/article/pii/S1053811920300987" target="_self"><span class="anchor-text">Multi-subject Stochastic Blockmodels for adaptive analysis of individual differences in human brain network cluster structure</span></a></h3><div class="text-s">2020, NeuroImage</div></div><div class="buttons text-s"><button class="button-link button-link-secondary" data-aa-button="sd:product:journal:article:location=citing-articles:type=view-details" aria-describedby="citing-articles-article-1-title" aria-controls="citing-articles-article-1" aria-expanded="false" type="button"><span class="button-link-text">Show abstract</span><svg role="img" focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down" aria-label="Navigate Down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-gulliver text-s" id="reference-abstract"><div class="u-margin-ver-m"><p id="abspara0010">There is considerable interest in elucidating the cluster structure of brain networks in terms of modules, blocks or clusters of similar nodes. However, it is currently challenging to handle data on multiple subjects since most of the existing methods are applicable only on a subject-by-subject basis or for analysis of an average group network. The main limitation of per-subject models is that there is no obvious way to combine the results for group comparisons, and of group-averaged models that they do not reflect the variability between subjects. Here, we propose two new extensions of the classical Stochastic Blockmodel (SBM) that use a mixture model to estimate blocks or clusters of connected nodes, combined with a regression model to capture the effects of subject-level covariates on individual differences in cluster structure. The proposed Multi-Subject Stochastic Blockmodels (MS-SBMs) can flexibly account for between-subject variability in terms of homogeneous or heterogeneous covariate effects on connectivity using subject demographics such as age or diagnostic status. Using synthetic data, representing a range of block sizes and cluster structures, we investigate the accuracy of the estimated MS-SBM parameters as well as the validity of inference procedures based on the Wald, likelihood ratio and permutation tests. We show that the proposed multi-subject SBMs recover the true cluster structure of synthetic networks more accurately and adaptively than standard methods for modular decomposition (i.e. the Fast Louvain and Newman Spectral algorithms). Permutation tests of MS-SBM parameters were more robustly valid for statistical inference and Type I error control than tests based on standard asymptotic assumptions. Applied to analysis of multi-subject resting-state fMRI networks (13 healthy volunteers; 12 people with schizophrenia; <span class="math"><math><mrow is="true"><mi is="true">n</mi><mo linebreak="goodbreak" linebreakstyle="after" is="true">=</mo><mn is="true">268</mn></mrow></math></span> brain regions), we show that Heterogeneous Stochastic Blockmodel (Het-SBM) identifies a range of network topologies simultaneously, including modular and core structures.</p></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-gulliver" id="citing-articles-article-2-title"><a class="anchor anchor-default" href="/science/article/pii/S1053811917303816" target="_self"><span class="anchor-text">Inference in the age of big data: Future perspectives on neuroscience</span></a></h3><div class="text-s">2017, NeuroImage</div><div class="CitedSection u-margin-s-top"><div class="u-margin-s-left"><div class="cite-header text-s u-text-italic u-font-sans">Citation Excerpt :</div><p class="u-font-gulliver text-xs">We therefore believe that the strength of flexible non-parametric models to automatically adjust the number of model parameters will probably turn out to be a crucial property of statistical models used in data-rich neuroscience. Although non-parametric models have been used in neuroimaging (e.g., Lashkari et al., 2012; Andersen et al., 2014), parametric models are today the predominant approach in neuroscience. Many big-sample studies (i.e., data from hundreds of animals or humans) currently apply the same parametric models as previous small-sample studies (i.e., a few dozen animals or humans).</p></div></div></div><div class="buttons text-s"><button class="button-link button-link-secondary" data-aa-button="sd:product:journal:article:location=citing-articles:type=view-details" aria-describedby="citing-articles-article-2-title" aria-controls="citing-articles-article-2" aria-expanded="false" type="button"><span class="button-link-text">Show abstract</span><svg role="img" focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down" aria-label="Navigate Down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-gulliver text-s" id="reference-abstract"><div class="u-margin-ver-m"><p id="sp0055">Neuroscience is undergoing faster changes than ever before. Over 100 years our field qualitatively described and invasively manipulated single or few organisms to gain anatomical, physiological, and pharmacological insights. In the last 10 years neuroscience spawned quantitative datasets of unprecedented breadth (e.g., microanatomy, synaptic connections, and optogenetic brain-behavior assays) and size (e.g., cognition, brain imaging, and genetics). While growing data availability and information granularity have been amply discussed, we direct attention to a less explored question: <em>How will the unprecedented data richness shape data analysis practices?</em> Statistical reasoning is becoming more important to distill neurobiological knowledge from healthy and pathological brain measurements. We argue that large-scale data analysis will use more statistical models that are non-parametric, generative, and mixing frequentist and Bayesian aspects, while supplementing classical hypothesis testing with out-of-sample predictions.</p></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-gulliver" id="citing-articles-article-3-title"><a class="anchor anchor-default" href="https://doi.org/10.48550/arXiv.2301.06182" target="_blank"><span class="anchor-text">Bayesian Models of Functional Connectomics and Behavior</span></a></h3><div class="text-s">2023, arXiv</div></div><div class="buttons text-s"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-gulliver text-s" id="reference-abstract"><div class="u-margin-ver-m"></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-gulliver" id="citing-articles-article-4-title"><a class="anchor anchor-default" href="http://www.scopus.com/scopus/inward/record.url?partnerID=10&amp;rel=3.0.0&amp;view=basic&amp;eid=2-s2.0-85143440956&amp;md5=1380e55bfddec334757cd492b2da9423" target="_blank"><span class="anchor-text">Improved Brain Community Structure Detection by Two-Step Weighted Modularity Maximization</span></a></h3><div class="text-s">2022, SSRN</div></div><div class="buttons text-s"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-gulliver text-s" id="reference-abstract"><div class="u-margin-ver-m"></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-gulliver" id="citing-articles-article-5-title"><a class="anchor anchor-default" href="https://doi.org/10.1162/netn_a_00242" target="_blank"><span class="anchor-text">Constraining functional coactivation with a cluster-based structural connectivity network</span></a></h3><div class="text-s">2022, Network Neuroscience</div></div><div class="buttons text-s"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-gulliver text-s" id="reference-abstract"><div class="u-margin-ver-m"></div></div></div></li></ul><a class="button-alternative button-alternative-secondary text-m" href="http://www.scopus.com/scopus/inward/citedby.url?partnerID=10&amp;rel=3.0.0&amp;eid=2-s2.0-84904056582&amp;md5=73bc9ed5ba858bf59715dddd72d2a95" id="citing-articles-view-all-btn" target="_blank"><svg role="img" focusable="false" viewBox="0 0 78 128" width="14.625" height="24" class="icon icon-arrow-up-right u-fill-grey8" aria-label="Arrow Up and Right"><path d="m4 36h57.07l-59.5 59.5 7.07 7.08 59.36-59.36v56.78h1e1v-74h-74z"></path></svg><span class="button-alternative-text">View all citing articles on Scopus</span></a></div></div></section></div><div id="preview-section-recommended-articles"><section class="ListArticles"><div class="PageDivider"></div><header id="recommended-articles-header"><h2 class="u-h4 u-margin-l-ver u-font-gulliver">Recommended articles (6)</h2></header><div aria-describedby="recommended-articles-header"><div id="recommended-articles"><ul><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><div class="u-margin-xs-ver text-xs">Research article</div><h3 class="u-font-gulliver" id="recommended-articles-article-0-title"><a class="anchor anchor-default" href="/science/article/pii/S1053811914005886"><span class="anchor-text"><span>Nutritional influences on early white matter development: Response to Anderson and Burggren</span></span></a></h3><div class="text-s"><div class="source">NeuroImage, Volume 100, 2014, pp. 703-705</div></div></div><div class="buttons text-s"><button class="button-link button-link-secondary" data-aa-button="sd:product:journal:article:location=recommended-articles:type=view-details" aria-describedby="recommended-articles-article-0-title" aria-controls="recommended-articles-article-0" aria-expanded="false" type="button"><span class="button-link-text">Show abstract</span><svg role="img" focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down" aria-label="Navigate Down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-gulliver text-s" id="reference-abstract"><div class="u-margin-ver-m"><p id="sp0005">Does breastfeeding alter early brain development? In a recent retrospective study, our group examined the cross-sectional relationship between early infant feeding practice and white matter maturation and cognitive development. In groups matched for child and mother age, gestation duration, birth weight, gender distribution, and socio-economic status; we observed that children who were breastfed exclusively for at least 3<!-- -->Â <!-- -->months showed, on average, increased white matter myelin development compared to children who either were exclusively formula-fed, or received a mixture of breast milk and formula. In secondary analysis on sub-sets of these children, again matched for important confounding variables, we found improved cognitive test scores of receptive language in the exclusively breast-fed children compared to formula or formula<!-- -->Â <!-- -->+<!-- -->Â <!-- -->breast-fed children; and that prolonged breastfeeding was associated with increased motor, language, and visual functioning in exclusively breast-fed children. In response to this work, Anderson and Burggren have questioned our methodology and, by association, our findings. Further, they use their critique as a platform for advancing an alternative interpretation of our findings: that observed results were not associated with prolonged breast-feeding, but rather delayed the introduction of cow&#x27;s milk. In this response, we address and clarify some of the misconceptions presented by Anderson and Burggren.</p></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><div class="u-margin-xs-ver text-xs">Research article</div><h3 class="u-font-gulliver" id="recommended-articles-article-1-title"><a class="anchor anchor-default" href="/science/article/pii/S1053811914005254"><span class="anchor-text"><span>Longer gestation is associated with more efficient brain networks in preadolescent children</span></span></a></h3><div class="text-s"><div class="source">NeuroImage, Volume 100, 2014, pp. 619-627</div></div></div><div class="buttons text-s"><button class="button-link button-link-secondary" data-aa-button="sd:product:journal:article:location=recommended-articles:type=view-details" aria-describedby="recommended-articles-article-1-title" aria-controls="recommended-articles-article-1" aria-expanded="false" type="button"><span class="button-link-text">Show abstract</span><svg role="img" focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down" aria-label="Navigate Down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-gulliver text-s" id="reference-abstract"><div class="u-margin-ver-m"><p id="sp0005">Neurodevelopmental benefits of increased gestation have not been fully characterized in terms of network organization. Since brain function can be understood as an integrated network of neural information from distributed brain regions, investigation of the effects of gestational length on network properties is a critical goal of human developmental neuroscience. Using diffusion tensor imaging and fiber tractography, we investigated the effects of gestational length on the small-world attributes and rich club organization of 147 preadolescent children, whose gestational length ranged from 29 to 42<!-- -->Â <!-- -->weeks. Higher network efficiency was positively associated with longer gestation. The longer gestation was correlated with increased local efficiency in the posterior medial cortex, including the precuneus, cuneus, and superior parietal regions. Rich club organization was also observed indicating the existence of highly interconnected structural hubs formed in preadolescent children. Connectivity <em>among</em> rich club members and <em>from</em> rich club regions was positively associated with the length of gestation, indicating the higher level of topological benefits of structural connectivity from longer gestation in the predominant regions of brain networks. The findings provide evidence that longer gestation is associated with improved topological organization of the preadolescent brain, characterized by the increased communication capacity of the brain network and enhanced directional strength of brain connectivity with central hub regions.</p></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><div class="u-margin-xs-ver text-xs">Research article</div><h3 class="u-font-gulliver" id="recommended-articles-article-2-title"><a class="anchor anchor-default" href="/science/article/pii/S1053811914005096"><span class="anchor-text"><span>Cross-validation of serial optical coherence scanning and diffusion tensor imaging: A study on neural fiber maps in human medulla oblongata</span></span></a></h3><div class="text-s"><div class="source">NeuroImage, Volume 100, 2014, pp. 395-404</div></div></div><div class="buttons text-s"><button class="button-link button-link-secondary" data-aa-button="sd:product:journal:article:location=recommended-articles:type=view-details" aria-describedby="recommended-articles-article-2-title" aria-controls="recommended-articles-article-2" aria-expanded="false" type="button"><span class="button-link-text">Show abstract</span><svg role="img" focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down" aria-label="Navigate Down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-gulliver text-s" id="reference-abstract"><div class="u-margin-ver-m"><p id="sp0005">We established a strategy to perform cross-validation of serial optical coherence scanner imaging (SOCS) and diffusion tensor imaging (DTI) on a postmortem human medulla. Following DTI, the sample was serially scanned by SOCS, which integrates a vibratome slicer and a multi-contrast optical coherence tomography rig for large-scale three-dimensional imaging at microscopic resolution. The DTI dataset was registered to the SOCS space. An average correlation coefficient of 0.9 was found between the co-registered fiber maps constructed by fractional anisotropy and retardance contrasts. Pixelwise comparison of fiber orientations demonstrated good agreement between the DTI and SOCS measures. Details of the comparison were studied in regions exhibiting a variety of fiber organizations. DTI estimated the preferential orientation of small fiber tracts; however, it didn&#x27;t capture their complex patterns as SOCS did. In terms of resolution and imaging depth, SOCS and DTI complement each other, and open new avenues for cross-modality investigations of the brain.</p></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><div class="u-margin-xs-ver text-xs">Research article</div><h3 class="u-font-gulliver" id="recommended-articles-article-3-title"><a class="anchor anchor-default" href="/science/article/pii/S1053811914004571"><span class="anchor-text"><span>Self-regulation of inter-hemispheric visual cortex balance through real-time fMRI neurofeedback training</span></span></a></h3><div class="text-s"><div class="source">NeuroImage, Volume 100, 2014, pp. 1-14</div></div></div><div class="buttons text-s"><button class="button-link button-link-secondary" data-aa-button="sd:product:journal:article:location=recommended-articles:type=view-details" aria-describedby="recommended-articles-article-3-title" aria-controls="recommended-articles-article-3" aria-expanded="false" type="button"><span class="button-link-text">Show abstract</span><svg role="img" focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down" aria-label="Navigate Down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-gulliver text-s" id="reference-abstract"><div class="u-margin-ver-m"><p id="sp0005">Recent advances in neurofeedback based on real-time functional magnetic resonance imaging (fMRI) allow for learning to control spatially localized brain activity in the range of millimeters across the entire brain. Real-time fMRI neurofeedback studies have demonstrated the feasibility of self-regulating activation in specific areas that are involved in a variety of functions, such as perception, motor control, language, and emotional processing. In most of these previous studies, participants trained to control activity within one region of interest (ROI). In the present study, we extended the neurofeedback approach by now training healthy participants to control the interhemispheric balance between their left and right visual cortices. This was accomplished by providing feedback based on the difference in activity between a target visual ROI and the corresponding homologue region in the opposite hemisphere. Eight out of 14 participants learned to control the differential feedback signal over the course of 3 neurofeedback training sessions spread over 3<!-- -->Â <!-- -->days, i.e., they produced consistent increases in the visual target ROI relative to the opposite visual cortex. Those who learned to control the differential feedback signal were subsequently also able to exert that control in the absence of neurofeedback. Such learning to voluntarily control the balance between cortical areas of the two hemispheres might offer promising rehabilitation approaches for neurological or psychiatric conditions associated with pathological asymmetries in brain activity patterns, such as hemispatial neglect, dyslexia, or mood disorders.</p></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><div class="u-margin-xs-ver text-xs">Research article</div><h3 class="u-font-gulliver" id="recommended-articles-article-4-title"><a class="anchor anchor-default" href="/science/article/pii/S1053811914006259"><span class="anchor-text"><span>Neuroimaging paradigms for tonotopic mapping (II): The influence of acquisition protocol</span></span></a></h3><div class="text-s"><div class="source">NeuroImage, Volume 100, 2014, pp. 663-675</div></div></div><div class="buttons text-s"><button class="button-link button-link-secondary" data-aa-button="sd:product:journal:article:location=recommended-articles:type=view-details" aria-describedby="recommended-articles-article-4-title" aria-controls="recommended-articles-article-4" aria-expanded="false" type="button"><span class="button-link-text">Show abstract</span><svg role="img" focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down" aria-label="Navigate Down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-gulliver text-s" id="reference-abstract"><div class="u-margin-ver-m"><p id="sp0005">Numerous studies on the tonotopic organisation of auditory cortex in humans have employed a wide range of neuroimaging protocols to assess cortical frequency tuning. In the present functional magnetic resonance imaging (fMRI) study, we made a systematic comparison between acquisition protocols with variable levels of interference from acoustic scanner noise. Using sweep stimuli to evoke travelling waves of activation, we measured sound-evoked response signals using <em>sparse</em>, <em>clustered</em>, and <em>continuous</em> imaging protocols that were characterised by inter-scan intervals of 8.8, 2.2, or 0.0<!-- -->Â <!-- -->s, respectively. With regard to sensitivity to sound-evoked activation, the sparse and clustered protocols performed similarly, and both detected more activation than the continuous method. Qualitatively, tonotopic maps in activated areas proved highly similar, in the sense that the overall pattern of tonotopic gradients was reproducible across all three protocols. However, quantitatively, we observed substantial reductions in response amplitudes to moderately low stimulus frequencies that coincided with regions of strong energy in the scanner noise spectrum for the clustered and continuous protocols compared to the sparse protocol. At the same time, extreme frequencies became over-represented for these two protocols, and high best frequencies became relatively more abundant. Our results indicate that although all three scanning protocols are suitable to determine the layout of tonotopic fields, an exact quantitative assessment of the representation of various sound frequencies is substantially confounded by the presence of scanner noise. In addition, we noticed anomalous signal dynamics in response to our travelling wave paradigm that suggest that the assessment of frequency-dependent tuning is non-trivially influenced by time-dependent (hemo)dynamics when using sweep stimuli.</p></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><div class="u-margin-xs-ver text-xs">Research article</div><h3 class="u-font-gulliver" id="recommended-articles-article-5-title"><a class="anchor anchor-default" href="/science/article/pii/S1053811914004649"><span class="anchor-text"><span>Phase based venous suppression in resting-state BOLD GE-fMRI</span></span></a></h3><div class="text-s"><div class="source">NeuroImage, Volume 100, 2014, pp. 51-59</div></div></div><div class="buttons text-s"><button class="button-link button-link-secondary" data-aa-button="sd:product:journal:article:location=recommended-articles:type=view-details" aria-describedby="recommended-articles-article-5-title" aria-controls="recommended-articles-article-5" aria-expanded="false" type="button"><span class="button-link-text">Show abstract</span><svg role="img" focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down" aria-label="Navigate Down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-gulliver text-s" id="reference-abstract"><div class="u-margin-ver-m"><p id="sp0005">Resting-state functional MRI (RS-fMRI) is a widely used method for inferring connectivity between brain regions or nodes. As with task-based fMRI, the spatial specificity of the connectivity maps can be distorted by the strong biasing effect of the BOLD signal in macroscopic veins. In RS-fMRI this effect is exacerbated by the temporal coherences of physiological origin between large veins that are widely distributed in the brain. In gradient echo based EPI, used for the vast majority of RS-fMRI, macroscopic veins that carry BOLD-related changes exhibit a strong phase response. This allows for post-processing identification and removal of venous signals using a phase regressor technique. Here, we employ this approach to suppress macrovascular venous contributions in high-field whole-brain RS-fMRI data sets, resulting in significant changes to both the spatial localization of the networks and the correlations between the network nodes. These effects were observed at both the individual and group analysis level, suggesting that venous contamination is a confounding factor for RS-fMRI studies even at relatively low image resolutions. Suppression of the macrovascular signal using the phase regression approach may therefore help to better identify, delineate, and interpret the true structure of large-scale brain networks.</p></div></div></div></li></ul></div></div></section></div><div class="PageDivider"></div><a class="anchor full-text-link text-s anchor-default" href="/science/article/pii/S1053811914004686" aria-disabled="true" tabindex="-1"><span class="anchor-text">View full text</span></a><div class="Copyright"><span class="copyright-line">Copyright Â© 2014 Elsevier Inc. All rights reserved.</span></div></article></div><div></div></div></div><footer role="contentinfo" class="els-footer u-bg-white text-xs u-padding-s-hor u-padding-m-hor-from-sm u-padding-l-hor-from-md u-padding-l-ver u-margin-l-top u-margin-xl-top-from-sm u-margin-l-top-from-md"><div class="els-footer-elsevier u-margin-m-bottom u-margin-0-bottom-from-md u-margin-s-right u-margin-m-right-from-md u-margin-l-right-from-lg"><a class="anchor anchor-default" href="https://www.elsevier.com/" aria-label="Elsevier home page (opens in a new tab)" target="_blank" rel="nofollow"><span class="anchor-text"><img class="footer-logo" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/47/images/elsevier-non-solus-new-with-wordmark.svg" alt="Elsevier logo with wordmark" height="64" width="58" loading="lazy" /></span></a></div><div class="els-footer-content"><div class="u-remove-if-print"><ul class="els-footer-links u-margin-xs-bottom" style="list-style:none"><li><a class="anchor u-display-block u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-default" href="https://www.elsevier.com/solutions/sciencedirect" id="els-footer-about-science-direct" target="_blank" rel="nofollow"><span class="anchor-text">About ScienceDirect</span></a></li><li><a class="anchor u-display-block u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-default" href="/user/institution/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS1053811914004686" id="els-footer-remote-access" target="_blank" rel="nofollow"><span class="anchor-text">Remote access</span></a></li><li><a class="anchor u-display-block u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-default" href="https://sd-cart.elsevier.com/?" id="els-footer-shopping-cart" target="_blank" rel="nofollow"><span class="anchor-text">Shopping cart</span></a></li><li><a class="anchor u-display-block u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-default" href="http://elsmediakits.com" id="els-footer-advertise" target="_blank" rel="nofollow"><span class="anchor-text">Advertise</span></a></li><li><a class="anchor u-display-block u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-default" href="https://service.elsevier.com/app/contact/supporthub/sciencedirect/" id="els-footer-contact-support" target="_blank" rel="nofollow"><span class="anchor-text">Contact and support</span></a></li><li><a class="anchor u-display-block u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-default" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions" id="els-footer-terms-condition" target="_blank" rel="nofollow"><span class="anchor-text">Terms and conditions</span></a></li><li><a class="anchor u-display-block u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-default" href="https://www.elsevier.com/legal/privacy-policy" id="els-footer-privacy-policy" target="_blank" rel="nofollow"><span class="anchor-text">Privacy policy</span></a></li></ul></div><p id="els-footer-cookie-message" class="u-remove-if-print">We use cookies to help provide and enhance our service and tailor content and ads. By continuing you agree to the<!-- --> <a class="anchor u-clr-grey8 u-margin-0-right anchor-default" href="https://www.sciencedirect.com/legal/use-of-cookies" target="_blank" rel="nofollow"><span class="anchor-text"><strong>use of cookies</strong></span></a>.</p><p id="els-footer-copyright">Copyright Â© <!-- -->2023<!-- --> Elsevier B.V. or its licensors or contributors.<!-- --> <span class="u-remove-if-print">ScienceDirectÂ® is a registered trademark of Elsevier B.V.</span></p><p class="u-remove-if-not-print">ScienceDirectÂ® is a registered trademark of Elsevier B.V.</p></div><div class="els-footer-relx u-margin-0-top u-margin-m-top-from-xs u-margin-0-top-from-md"><a class="anchor anchor-default" href="https://www.relx.com/" aria-label="RELX home page (opens in a new tab)" id="els-footer-relx" target="_blank" rel="nofollow"><span class="anchor-text"><img loading="lazy" src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/60/images/logo-relx-tm.svg" width="93" height="20" alt="RELX group home page" /></span></a></div></footer></div></div></div></div>
<script type="application/json" data-iso-key="_0">{"abstracts":{"content":[{"$$":[{"$":{"id":"st0005"},"#name":"section-title","_":"Abstract"},{"$$":[{"$":{"view":"all","id":"sp0005"},"#name":"simple-para","_":"Modeling of resting state functional magnetic resonance imaging (rs-fMRI) data using network models is of increasing interest. It is often desirable to group nodes into clusters to interpret the communication patterns between nodes. In this study we consider three different nonparametric Bayesian models for node clustering in complex networks. In particular, we test their ability to predict unseen data and their ability to reproduce clustering across datasets. The three generative models considered are the Infinite Relational Model (IRM), Bayesian Community Detection (BCD), and the Infinite Diagonal Model (IDM). The models define probabilities of generating links within and between clusters and the difference between the models lies in the restrictions they impose upon the between-cluster link probabilities. IRM is the most flexible model with no restrictions on the probabilities of links between clusters. BCD restricts the between-cluster link probabilities to be strictly lower than within-cluster link probabilities to conform to the community structure typically seen in social networks. IDM only models a single between-cluster link probability, which can be interpreted as a background noise probability. These probabilistic models are compared against three other approaches for node clustering, namely Infomap, Louvain modularity, and hierarchical clustering. Using 3 different datasets comprising healthy volunteers' rs-fMRI we found that the BCD model was in general the most predictive and reproducible model. This suggests that rs-fMRI data exhibits community structure and furthermore points to the significance of modeling heterogeneous between-cluster link probabilities."}],"$":{"view":"all","id":"as0005"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0005","class":"author"},"#name":"abstract"},{"$$":[{"$":{"id":"st0010"},"#name":"section-title","_":"Highlights"},{"$$":[{"$$":[{"$$":[{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0415"},"#name":"para","_":"Three nonparametric Bayesian models for node clustering are used to model rs-fMRI."}],"$":{"id":"u0005"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0420"},"#name":"para","_":"Models' predictability and reproducibility are extensively evaluated using resampling."}],"$":{"id":"u0010"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0425"},"#name":"para","_":"The community structure model shows better predictability and reproducibility."}],"$":{"id":"u0015"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0430"},"#name":"para","_":"This finding suggests that rs-fMRI graphs exhibit community structure."}],"$":{"id":"u0020"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0435"},"#name":"para","_":"Modeling between-cluster link probabilities adds important information."}],"$":{"id":"u0025"},"#name":"list-item"}],"$":{"id":"l0005"},"#name":"list"}],"$":{"view":"all","id":"sp0060"},"#name":"simple-para"}],"$":{"view":"all","id":"as0010"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0010","class":"author-highlights"},"#name":"abstract"}],"floats":[],"footnotes":[],"attachments":[]},"accessbarConfig":{"fallback":false,"id":"accessbar","version":"0.0.1","analytics":{"location":"accessbar","eventName":"ctaImpression"},"ariaLabel":{"accessbar":"Download options and search","componentsList":"PDF Options"},"banner":{"id":"Banner"},"components":[{"analytics":[{"ids":["accessbar:clinicalkeycta:inst-known:sdcust-unknown:ent-no:ra-no:source-linkinghub:method-ip"],"eventName":"ctaClick"}],"label":"Access through&nbsp;**ClinicalKey**","id":"ClinicalKey"},{"ariaLabel":"Access through your institution","institutionLogoAltText":"Seamless access","analytics":[{"ids":["accessbar:accesscta:inst-known:sdcust-unknown:ent-unknown:ra-yes:source-sdcookie:method-email"],"eventName":"ctaClick"}],"labelPrefix":"Access through","defaultLabelSuffix":"your institution","institutionName":"Chungnam National University","href":"/user/institution/login?targetUrl=%2Fscience%2Farticle%2Fpii%2FS1053811914004686%3Ffr%3DRR-2%26ref%3Dpdf_download%26rr%3D7de975694c1da7e1","remoteAccessType":"SELFMANRA","id":"RemoteAccess"},{"analytics":[{"ids":["accessbar:purchase-pdf"],"eventName":"ctaClick"}],"label":"Purchase PDF","id":"PurchasePDF"},{"analytics":[{"ids":["accessbar:another-institution"],"eventName":"ctaClick"}],"label":"Access through another institution","href":"/user/institution/login?targetUrl=%2Fscience%2Farticle%2Fpii%2FS1053811914004686","id":"RemoteAccessOther"},{"id":"LinkResolver","analytics":[{"eventName":"navigationClick","ids":["sd:accessbar-linkresolver"]}],"rel":"noreferrer noopener","target":"_blank","href":"https://www.elsevier.com/?ctx_ver=Z39.88-2004&ctx_enc=info:ofi/enc:UTF-8&url_ver=Z39.88-2004&rfr_id=info:sid/Elsevier:SD&svc_val_fmt=info:ofi/fmt:kev:mtx:sch_svc&rft_val_fmt=info:ofi/fmt:kev:mtx:journal&rft.aulast=ANDERSEN&rft.auinit=K&rft.date=2014&rft.issn=10538119&rft.volume=100C&rft.spage=301&rft.epage=315&rft.title=NeuroImage&rft.atitle=Non-parametric%20Bayesian%20graph%20models%20reveal%20community%20structure%20in%20resting%20state%20fMRI&rft_id=info:doi/10.1016/j.neuroimage.2014.05.083","label":"Find it here!"}],"search":{"inputPlaceHolder":"Search ScienceDirect","ariaLabel":{"input":"Search ScienceDirect","submit":"Submit search"},"formAction":"/search#submit","analytics":[{"ids":["accessbar:search"],"eventName":"searchStart"}],"id":"QuickSearch"}},"adobeTarget":{},"article":{"accessOptions":{"remoteAccessOptions":{"institutionName":"Chungnam National University","institutionUrl":"/user/institution/login?targetUrl=%2Fscience%2Farticle%2Fpii%2FS1053811914004686%3Ffr%3DRR-2%26ref%3Dpdf_download%26rr%3D7de975694c1da7e1","remoteAccessType":"SELFMANRA"},"linkType":"PURCHASE","linkUrl":"/getaccess/pii/S1053811914004686","price":{"currency":"USD","totalPrice":35.95},"purchaseNonDiscounted":{"linkUrl":"/getaccess/pii/S1053811914004686?corporate=true","price":{"currency":"USD","totalPrice":43.95}}},"analyticsMetadata":{"accountId":"228598","accountName":"ScienceDirect Guests","loginStatus":"anonymous","userId":"12975512","isLoggedIn":false},"cid":"272508","content-family":"serial","copyright-line":"Copyright Â© 2014 Elsevier Inc. All rights reserved.","cover-date-years":["2014"],"cover-date-start":"2014-10-15","cover-date-text":"15 October 2014","document-subtype":"fla","document-type":"article","eid":"1-s2.0-S1053811914004686","doi":"10.1016/j.neuroimage.2014.05.083","first-fp":"301","hub-eid":"1-s2.0-S1053811914X00157","issuePii":"S1053811914X00157","item-weight":"FULL-TEXT","language":"en","last-lp":"315","last-author":{"#name":"last-author","$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"$$":[{"#name":"author","$":{"id":"au0030"},"$$":[{"#name":"given-name","_":"Lars Kai"},{"#name":"surname","_":"Hansen"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0055"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"encoded-e-address","__encoded":"JTdCJTIyJTIzbmFtZSUyMiUzQSUyMmUtYWRkcmVzcyUyMiUyQyUyMiUyNCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyZW0wMDMwJTIyJTJDJTIydHlwZSUyMiUzQSUyMmVtYWlsJTIyJTdEJTJDJTIyXyUyMiUzQSUyMmxrYWklNDBkdHUuZGslMjIlN0Q="}]}]},"normalized-first-auth-initial":"K","normalized-first-auth-surname":"ANDERSEN","pages":[{"last-page":"315","first-page":"301"}],"pii":"S1053811914004686","srctitle":"NeuroImage","suppl":"C","timestamp":"2016-02-25T22:59:15.708252-05:00","title":{"content":[{"#name":"title","$":{"id":"ti0005"},"_":"Non-parametric Bayesian graph models reveal community structure in resting state fMRI"}],"floats":[],"footnotes":[],"attachments":[]},"vol-first":"100","vol-iss-suppl-text":"Volume 100","userSettings":{"forceAbstract":false,"creditCardPurchaseAllowed":true,"blockFullTextForAnonymousAccess":false,"disableWholeIssueDownload":false,"preventTransactionalAccess":false,"preventDocumentDelivery":true},"contentType":"JL","crossmark":true,"document-references":52,"freeHtmlGiven":false,"userProfile":{"departmentName":"ScienceDirect Guests","accessType":"GUEST","accountId":"228598","webUserId":"12975512","accountName":"ScienceDirect Guests","departmentId":"291352","userType":"NORMAL","hasMultipleOrganizations":false},"access":{"openAccess":false,"openArchive":false},"aipType":"none","articleEntitlement":{"entitled":false,"isCasaUser":false,"usageInfo":"(12975512,U|291352,D|228598,A|3,P|2,PL)(SDFE,CON|9f8a920758fc61466a2a8ca5a137872cc201gxrqa,SSO|ANON_GUEST,ACCESS_TYPE)"},"crawlerInformation":{"canCrawlPDFContent":false,"isCrawler":false},"dates":{"Available online":"7 June 2014","Revised":[],"Accepted":"30 May 2014","Publication date":"15 October 2014"},"downloadFullIssue":false,"entitlementReason":"unsubscribed","hasBody":true,"has-large-authors":false,"hasScholarlyAbstract":true,"headerConfig":{"contactUrl":"https://service.elsevier.com/app/contact/supporthub/sciencedirect/","userName":"","userEmail":"","orgName":"ScienceDirect Guests","webUserId":"12975512","libraryBanner":{},"shib_regUrl":"","tick_regUrl":"","recentInstitutions":[],"canActivatePersonalization":false,"hasInstitutionalAssociation":false,"hasMultiOrg":false,"userType":"GUEST","userAnonymity":"ANON_GUEST","allowCart":true,"environment":"prod","cdnAssetsHost":"https://sdfestaticassets-us-east-1.sciencedirectassets.com"},"isCorpReq":false,"isPdfFullText":false,"issn":"10538119","issn-primary-formatted":"1053-8119","issRange":"","isThirdParty":false,"pageCount":15,"publication-content":{"noElsevierLogo":false,"imprintPublisher":{"displayName":"Academic Press","id":"350"},"isSpecialIssue":false,"isSampleIssue":false,"transactionsBlocked":false,"publicationOpenAccess":{"oaStatus":"Partial","oaArticleCount":4754,"openArchiveStatus":false,"openArchiveArticleCount":49,"openAccessStartDate":"2020-01-01T00:00:00Z","oaAllowsAuthorPaid":true},"issue-cover":{"attachment":[{"attachment-eid":"1-s2.0-S1053811914X00157-cov200h.gif","file-basename":"cov200h","extension":"gif","filename":"cov200h.gif","ucs-locator":["https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1053811914X00157/cover/DOWNSAMPLED200/image/gif/e6a44f27906373e19b01c40d89e8071b/cov200h.gif","https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1053811914X00157/cover/DOWNSAMPLED200/image/gif/e6a44f27906373e19b01c40d89e8071b/cov200h.gif"],"attachment-type":"IMAGE-COVER-H200","filesize":"27655","pixel-height":"200","pixel-width":"150"},{"attachment-eid":"1-s2.0-S1053811914X00157-cov150h.gif","file-basename":"cov150h","extension":"gif","filename":"cov150h.gif","ucs-locator":["https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1053811914X00157/cover/DOWNSAMPLED/image/gif/4aeea41967a8de2ceaed906ec5392efa/cov150h.gif","https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1053811914X00157/cover/DOWNSAMPLED/image/gif/4aeea41967a8de2ceaed906ec5392efa/cov150h.gif"],"attachment-type":"IMAGE-COVER-H150","filesize":"19925","pixel-height":"150","pixel-width":"113"}]},"smallCoverUrl":"https://ars.els-cdn.com/content/image/S10538119.gif","title":"neuroimage","contentTypeCode":"JL","images":{"coverImage":"https://ars.els-cdn.com/content/image/1-s2.0-S1053811914X00157-cov150h.gif","logo":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/f428ac6e436879961ed6bb261fe095ecf4ea09e0/image/elsevier-non-solus.png","logoAltText":"Elsevier"},"publicationCoverImageUrl":"https://ars.els-cdn.com/content/image/1-s2.0-S1053811914X00157-cov150h.gif"},"useEnhancedReader":true,"volRange":"100","features":["keywords","references","preview"],"open-research":{},"self-archiving":{},"titleString":"Non-parametric Bayesian graph models reveal community structure in resting state fMRI","renderingMode":"Preview","isAbstract":true,"isContentVisible":false,"ajaxLinks":{"authorMetadata":true,"substances":true},"eligibleForUniversalPdf":true,"pdfEmbed":false,"displayViewFullText":false},"authors":{"content":[{"#name":"author-group","$":{"id":"ag0005"},"$$":[{"#name":"author","$":{"id":"au0005"},"$$":[{"#name":"given-name","_":"Kasper Winther"},{"#name":"surname","_":"Andersen"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0010"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0015"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"cr0005","id":"cf0020"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"âŽ"}]},{"#name":"encoded-e-address","__encoded":"JTdCJTIyJTIzbmFtZSUyMiUzQSUyMmUtYWRkcmVzcyUyMiUyQyUyMiUyNCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyZW0wMDA1JTIyJTJDJTIydHlwZSUyMiUzQSUyMmVtYWlsJTIyJTdEJTJDJTIyXyUyMiUzQSUyMmthc3BlcndqJTQwZHJjbXIuZGslMjIlN0Q="}]},{"#name":"author","$":{"id":"au0010","orcid":"0000-0001-8606-7641"},"$$":[{"#name":"given-name","_":"Kristoffer H."},{"#name":"surname","_":"Madsen"},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0025"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"encoded-e-address","__encoded":"JTdCJTIyJTIzbmFtZSUyMiUzQSUyMmUtYWRkcmVzcyUyMiUyQyUyMiUyNCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyZW0wMDEwJTIyJTJDJTIydHlwZSUyMiUzQSUyMmVtYWlsJTIyJTdEJTJDJTIyXyUyMiUzQSUyMnN0b2ZmZXIlNDBkcmNtci5kayUyMiU3RA=="}]},{"#name":"author","$":{"id":"au0015"},"$$":[{"#name":"given-name","_":"Hartwig Roman"},{"#name":"surname","_":"Siebner"},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0030"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0035"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]},{"#name":"cross-ref","$":{"refid":"af0020","id":"cf0040"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]},{"#name":"encoded-e-address","__encoded":"JTdCJTIyJTIzbmFtZSUyMiUzQSUyMmUtYWRkcmVzcyUyMiUyQyUyMiUyNCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyZW0wMDE1JTIyJTJDJTIydHlwZSUyMiUzQSUyMmVtYWlsJTIyJTdEJTJDJTIyXyUyMiUzQSUyMmhhcnR3aWcuc2llYm5lciU0MGRyY21yLmRrJTIyJTdE"}]},{"#name":"author","$":{"id":"au0020"},"$$":[{"#name":"given-name","_":"Mikkel N."},{"#name":"surname","_":"Schmidt"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0045"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"encoded-e-address","__encoded":"JTdCJTIyJTIzbmFtZSUyMiUzQSUyMmUtYWRkcmVzcyUyMiUyQyUyMiUyNCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyZW0wMDIwJTIyJTJDJTIydHlwZSUyMiUzQSUyMmVtYWlsJTIyJTdEJTJDJTIyXyUyMiUzQSUyMm1ucyU0MGR0dS5kayUyMiU3RA=="}]},{"#name":"author","$":{"id":"au0025"},"$$":[{"#name":"given-name","_":"Morten"},{"#name":"surname","_":"MÃ¸rup"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"encoded-e-address","__encoded":"JTdCJTIyJTIzbmFtZSUyMiUzQSUyMmUtYWRkcmVzcyUyMiUyQyUyMiUyNCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyZW0wMDI1JTIyJTJDJTIydHlwZSUyMiUzQSUyMmVtYWlsJTIyJTdEJTJDJTIyXyUyMiUzQSUyMm1tJTQwZHR1LmRrJTIyJTdE"}]},{"#name":"author","$":{"id":"au0030"},"$$":[{"#name":"given-name","_":"Lars Kai"},{"#name":"surname","_":"Hansen"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0055"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"encoded-e-address","__encoded":"JTdCJTIyJTIzbmFtZSUyMiUzQSUyMmUtYWRkcmVzcyUyMiUyQyUyMiUyNCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyZW0wMDMwJTIyJTJDJTIydHlwZSUyMiUzQSUyMmVtYWlsJTIyJTdEJTJDJTIyXyUyMiUzQSUyMmxrYWklNDBkdHUuZGslMjIlN0Q="}]},{"#name":"affiliation","$":{"id":"af0005"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","$":{"id":"tn0005"},"_":"Department of Applied Mathematics and Computer Science, Technical University of Denmark, Matematiktorvet, Bygning 303 B, 2800 Kgs. Lyngby, Denmark"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Applied Mathematics and Computer Science"},{"#name":"organization","_":"Technical University of Denmark"},{"#name":"address-line","_":"Matematiktorvet"},{"#name":"address-line","_":"Bygning 303 B"},{"#name":"city","_":"Kgs. Lyngby"},{"#name":"postal-code","_":"2800"},{"#name":"country","_":"Denmark"}]}]},{"#name":"affiliation","$":{"id":"af0010"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","$":{"id":"tn0010"},"_":"Danish Research Centre for Magnetic Resonance, Centre for Functional and Diagnostic Imaging and Research, Copenhagen University Hospital Hvidovre, Kettegaard Alle 30, 2650 Hvidovre, Denmark"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Danish Research Centre for Magnetic Resonance"},{"#name":"organization","_":"Centre for Functional and Diagnostic Imaging and Research"},{"#name":"organization","_":"Copenhagen University Hospital Hvidovre"},{"#name":"address-line","_":"Kettegaard Alle 30"},{"#name":"city","_":"Hvidovre"},{"#name":"postal-code","_":"2650"},{"#name":"country","_":"Denmark"}]}]},{"#name":"affiliation","$":{"id":"af0015"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","$":{"id":"tn0015"},"_":"Department of Clinical Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Blegdamsvej 3B, 2200 KÃ¸benhavn N, Denmark"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Clinical Medicine"},{"#name":"organization","_":"Faculty of Health and Medical Sciences"},{"#name":"organization","_":"University of Copenhagen"},{"#name":"address-line","_":"Blegdamsvej 3B"},{"#name":"city","_":"KÃ¸benhavn N"},{"#name":"postal-code","_":"2200"},{"#name":"country","_":"Denmark"}]}]},{"#name":"affiliation","$":{"id":"af0020"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","$":{"id":"tn0020"},"_":"Department of Neurology, Copenhagen University Hospital Bispebjerg, Bispebjerg Bakke 23, 2400 KÃ¸benhavn NV, Denmark"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Neurology"},{"#name":"organization","_":"Copenhagen University Hospital Bispebjerg"},{"#name":"address-line","_":"Bispebjerg Bakke 23"},{"#name":"city","_":"KÃ¸benhavn NV"},{"#name":"postal-code","_":"2400"},{"#name":"country","_":"Denmark"}]}]},{"#name":"correspondence","$":{"id":"cr0005"},"$$":[{"#name":"label","_":"âŽ"},{"#name":"text","$":{"id":"tx0005"},"_":"Corresponding author at: Danish Research Centre for Magnetic Resonance, Department 714, Centre for Functional and Diagnostic Imaging and Research, Copenhagen University Hospital Hvidovre, Kettegaard Alle 30, DK-2650 Hvidovre, Denmark."}]}]}],"floats":[],"footnotes":[],"affiliations":{"af0005":{"#name":"affiliation","$":{"id":"af0005"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","$":{"id":"tn0005"},"_":"Department of Applied Mathematics and Computer Science, Technical University of Denmark, Matematiktorvet, Bygning 303 B, 2800 Kgs. Lyngby, Denmark"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Applied Mathematics and Computer Science"},{"#name":"organization","_":"Technical University of Denmark"},{"#name":"address-line","_":"Matematiktorvet"},{"#name":"address-line","_":"Bygning 303 B"},{"#name":"city","_":"Kgs. Lyngby"},{"#name":"postal-code","_":"2800"},{"#name":"country","_":"Denmark"}]}]},"af0010":{"#name":"affiliation","$":{"id":"af0010"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","$":{"id":"tn0010"},"_":"Danish Research Centre for Magnetic Resonance, Centre for Functional and Diagnostic Imaging and Research, Copenhagen University Hospital Hvidovre, Kettegaard Alle 30, 2650 Hvidovre, Denmark"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Danish Research Centre for Magnetic Resonance"},{"#name":"organization","_":"Centre for Functional and Diagnostic Imaging and Research"},{"#name":"organization","_":"Copenhagen University Hospital Hvidovre"},{"#name":"address-line","_":"Kettegaard Alle 30"},{"#name":"city","_":"Hvidovre"},{"#name":"postal-code","_":"2650"},{"#name":"country","_":"Denmark"}]}]},"af0015":{"#name":"affiliation","$":{"id":"af0015"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","$":{"id":"tn0015"},"_":"Department of Clinical Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Blegdamsvej 3B, 2200 KÃ¸benhavn N, Denmark"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Clinical Medicine"},{"#name":"organization","_":"Faculty of Health and Medical Sciences"},{"#name":"organization","_":"University of Copenhagen"},{"#name":"address-line","_":"Blegdamsvej 3B"},{"#name":"city","_":"KÃ¸benhavn N"},{"#name":"postal-code","_":"2200"},{"#name":"country","_":"Denmark"}]}]},"af0020":{"#name":"affiliation","$":{"id":"af0020"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","$":{"id":"tn0020"},"_":"Department of Neurology, Copenhagen University Hospital Bispebjerg, Bispebjerg Bakke 23, 2400 KÃ¸benhavn NV, Denmark"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Neurology"},{"#name":"organization","_":"Copenhagen University Hospital Bispebjerg"},{"#name":"address-line","_":"Bispebjerg Bakke 23"},{"#name":"city","_":"KÃ¸benhavn NV"},{"#name":"postal-code","_":"2400"},{"#name":"country","_":"Denmark"}]}]}},"correspondences":{"cr0005":{"#name":"correspondence","$":{"id":"cr0005"},"$$":[{"#name":"label","_":"âŽ"},{"#name":"text","$":{"id":"tx0005"},"_":"Corresponding author at: Danish Research Centre for Magnetic Resonance, Department 714, Centre for Functional and Diagnostic Imaging and Research, Copenhagen University Hospital Hvidovre, Kettegaard Alle 30, DK-2650 Hvidovre, Denmark."}]}},"attachments":[],"scopusAuthorIds":{},"articles":{}},"authorMetadata":[],"banner":{"expanded":false},"biographies":{},"body":{},"browser":{"name":"Chrome","engine":"Blink"},"chapters":{"toc":[],"isLoading":false},"changeViewLinks":{"showFullTextLink":true,"showAbstractLink":false},"citingArticles":{"hitCount":15,"viewMoreUrl":"http://www.scopus.com/scopus/inward/citedby.url?partnerID=10&rel=3.0.0&eid=2-s2.0-84904056582&md5=73bc9ed5ba858bf59715dddd72d2a95","articles":[{"articleTitle":"Using connectomics for predictive assessment of brain parcellations","authors":"Albers K.J., Ambrosen K.S., Liptrot M.G., Dyrby T.B., Schmidt M.N., MÃ¸rup M.","doi":"10.1016/j.neuroimage.2021.118170","externalArticle":false,"issn":"10538119","openAccess":1,"pii":"S105381192100447X","publicationDate":"2021-09-01","publicationTitle":"NeuroImage","publicationYear":"2021","scopusArticleUrl":"http://www.scopus.com/scopus/inward/record.url?partnerID=10&rel=3.0.0&view=basic&eid=2-s2.0-85107614151&md5=7910f24f4ac082a88b53c4096d9f03d","scopusEid":"2-s2.0-85107614151","snippets":["In contrast to cluster validity measures that quantify similarity within the cluster to the similarity of other clusters Arslan et al. (2018); Dornas and Braun (2018) or statistical prediction based on feature maps or time-series Thirion et al. (2014); Wang et al. (2018), our proposed approach quantifies how well the network organization is preserved within the representation induced by a parcellation. In particular, our statistical prediction framework poses quantification of parcellation quality as a link-prediction problem Ambrosen et al. (2014); Andersen et al. (2014); Clauset et al. (2008); Liben-Nowell and Kleinberg (2007). A parcellation is thereby assessed by its ability to characterize brain connectivity data that may derive from an orthogonal modality to that from which the parcellation was defined."],"thirdParty":false,"volume":"Volume 238","abstract":{"$$":[{"$$":[{"$":{"id":"sctt0001"},"#name":"section-title","_":"Abstract"},{"$$":[{"$":{"view":"all","id":"sp0001"},"#name":"simple-para","_":"The organization of the human brain remains elusive, yet is of great importance to the mechanisms of integrative brain function. At the macroscale, its structural and functional interpretation is conventionally assessed at the level of cortical units. However, the definition and validation of such cortical parcellations are problematic due to the absence of a true gold standard. We propose a framework for quantitative evaluation of brain parcellations via statistical prediction of connectomics data. Specifically, we evaluate the extent in which the network representation at the level of cortical units (defined as parcels) accounts for high-resolution brain connectivity. Herein, we assess the pertinence and comparative ranking of ten existing parcellation atlases to account for functional (FC) and structural connectivity (SC) data based on data from the Human Connectome Project (HCP), and compare them to data-driven as well as spatially-homogeneous geometric parcellations including geodesic parcellations with similar size distributions as the atlases. We find substantial discrepancy in parcellation structures that well characterize FC and SC and differences in what well represents an individualâ€™s functional connectome when compared against the FC structure that is preserved across individuals. Surprisingly, simple spatial homogenous parcellations generally provide good representations of both FC and SC, but are inferior when their within-parcellation distribution of individual parcel sizes is matched to that of a valid atlas. This suggests that the choice of fine grained and coarse representations used by existing atlases are important. However, we find that resolution is more critical than the exact border location of parcels."}],"$":{"view":"all","id":"abssec0001"},"#name":"abstract-sec"}],"$":{"view":"all","id":"abs0001","class":"author"},"#name":"abstract"}],"$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"#name":"abstracts"},"pdf":{"urlType":"download","url":"/science/article/pii/S105381192100447X/pdfft?md5=e4d7789b238ae0b699f73a20cc7acd9c&pid=1-s2.0-S105381192100447X-main.pdf"}},{"articleTitle":"Multi-subject Stochastic Blockmodels for adaptive analysis of individual differences in human brain network cluster structure","authors":"PavloviÄ‡ D.M., Guillaume B.R.L., Towlson E.K., Kuek N.M.Y., Afyouni S., VÃ©rtes P.E., Yeo B.T.T., Bullmore E.T., Nichols T.E.","doi":"10.1016/j.neuroimage.2020.116611","externalArticle":false,"issn":"10538119","openAccess":1,"pii":"S1053811920300987","publicationDate":"2020-10-15","publicationTitle":"NeuroImage","publicationYear":"2020","scopusArticleUrl":"http://www.scopus.com/scopus/inward/record.url?partnerID=10&rel=3.0.0&view=basic&eid=2-s2.0-85087930375&md5=76fa7d45a7824cc453835b2376f84d","scopusEid":"2-s2.0-85087930375","thirdParty":false,"volume":"Volume 220","abstract":{"$$":[{"$$":[{"$$":[{"#name":"attachment-eid","_":"1-s2.0-S1053811920300987-si1.svg"},{"#name":"ucs-locator","_":"https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1053811920300987/STRIPIN/image/svg+xml/02a010b8c3250b9e96e4f6b341f9d450/si1.svg"},{"#name":"file-basename","_":"si1"},{"#name":"abstract-attachment","_":"true"},{"#name":"filename","_":"si1.svg"},{"#name":"extension","_":"svg"},{"#name":"filesize","_":"7694"},{"#name":"attachment-type","_":"ALTIMG"}],"$":{"xmlns:xocs":true},"#name":"attachment"}],"#name":"attachments"},{"$$":[{"$":{"id":"sectitle0010"},"#name":"section-title","_":"Abstract"},{"$$":[{"$$":[{"#name":"__text__","_":"There is considerable interest in elucidating the cluster structure of brain networks in terms of modules, blocks or clusters of similar nodes. However, it is currently challenging to handle data on multiple subjects since most of the existing methods are applicable only on a subject-by-subject basis or for analysis of an average group network. The main limitation of per-subject models is that there is no obvious way to combine the results for group comparisons, and of group-averaged models that they do not reflect the variability between subjects. Here, we propose two new extensions of the classical Stochastic Blockmodel (SBM) that use a mixture model to estimate blocks or clusters of connected nodes, combined with a regression model to capture the effects of subject-level covariates on individual differences in cluster structure. The proposed Multi-Subject Stochastic Blockmodels (MS-SBMs) can flexibly account for between-subject variability in terms of homogeneous or heterogeneous covariate effects on connectivity using subject demographics such as age or diagnostic status. Using synthetic data, representing a range of block sizes and cluster structures, we investigate the accuracy of the estimated MS-SBM parameters as well as the validity of inference procedures based on the Wald, likelihood ratio and permutation tests. We show that the proposed multi-subject SBMs recover the true cluster structure of synthetic networks more accurately and adaptively than standard methods for modular decomposition (i.e. the Fast Louvain and Newman Spectral algorithms). Permutation tests of MS-SBM parameters were more robustly valid for statistical inference and Type I error control than tests based on standard asymptotic assumptions. Applied to analysis of multi-subject resting-state fMRI networks (13 healthy volunteers; 12 people with schizophrenia; "},{"$$":[{"$$":[{"#name":"mi","_":"n"},{"$":{"linebreak":"goodbreak","linebreakstyle":"after"},"#name":"mo","_":"="},{"#name":"mn","_":"268"}],"#name":"mrow"}],"$":{"xmlns:mml":true,"altimg":"si1.svg"},"#name":"math"},{"#name":"__text__","_":" brain regions), we show that Heterogeneous Stochastic Blockmodel (Het-SBM) identifies a range of network topologies simultaneously, including modular and core structures."}],"$":{"view":"all","id":"abspara0010"},"#name":"simple-para"}],"$":{"view":"all","id":"abssec0010"},"#name":"abstract-sec"}],"$":{"view":"all","id":"abs0010","lang":"en","class":"author"},"#name":"abstract"}],"$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"#name":"abstracts"},"pdf":{"urlType":"download","url":"/science/article/pii/S1053811920300987/pdfft?md5=a448f80ad86e559c722463d986443044&pid=1-s2.0-S1053811920300987-main.pdf"}},{"articleTitle":"Inference in the age of big data: Future perspectives on neuroscience","authors":"Bzdok D., Yeo B.T.T.","doi":"10.1016/j.neuroimage.2017.04.061","externalArticle":false,"issn":"10538119","openAccess":1,"page":"549-564","pii":"S1053811917303816","publicationDate":"2017-07-15","publicationTitle":"NeuroImage","publicationYear":"2017","scopusArticleUrl":"http://www.scopus.com/scopus/inward/record.url?partnerID=10&rel=3.0.0&view=basic&eid=2-s2.0-85019157622&md5=2023808db8ee13acea1f9b4e4fa7814","scopusEid":"2-s2.0-85019157622","snippets":["We therefore believe that the strength of flexible non-parametric models to automatically adjust the number of model parameters will probably turn out to be a crucial property of statistical models used in data-rich neuroscience. Although non-parametric models have been used in neuroimaging (e.g., Lashkari et al., 2012; Andersen et al., 2014), parametric models are today the predominant approach in neuroscience. Many big-sample studies (i.e., data from hundreds of animals or humans) currently apply the same parametric models as previous small-sample studies (i.e., a few dozen animals or humans)."],"thirdParty":false,"volume":"Volume 155","abstract":{"$$":[{"$$":[{"$":{"id":"sect0005"},"#name":"section-title","_":"Abstract"},{"$$":[{"$$":[{"#name":"__text__","_":"Neuroscience is undergoing faster changes than ever before. Over 100 years our field qualitatively described and invasively manipulated single or few organisms to gain anatomical, physiological, and pharmacological insights. In the last 10 years neuroscience spawned quantitative datasets of unprecedented breadth (e.g., microanatomy, synaptic connections, and optogenetic brain-behavior assays) and size (e.g., cognition, brain imaging, and genetics). While growing data availability and information granularity have been amply discussed, we direct attention to a less explored question: "},{"#name":"italic","_":"How will the unprecedented data richness shape data analysis practices?"},{"#name":"__text__","_":" Statistical reasoning is becoming more important to distill neurobiological knowledge from healthy and pathological brain measurements. We argue that large-scale data analysis will use more statistical models that are non-parametric, generative, and mixing frequentist and Bayesian aspects, while supplementing classical hypothesis testing with out-of-sample predictions."}],"$":{"view":"all","id":"sp0055"},"#name":"simple-para"}],"$":{"view":"all","id":"abs0010"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0010","class":"author"},"#name":"abstract"},{"$$":[{"$":{"id":"sect0010"},"#name":"section-title","_":"Highlights"},{"$$":[{"$$":[{"$$":[{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0160"},"#name":"para","_":"Neuroscience started collecting multi-modal datasets from thousands of individuals."}],"$":{"id":"u0030"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$$":[{"#name":"italic","_":"Non-parametric"},{"#name":"__text__","_":" models will increase neurobiological insight as data accumulate."}],"$":{"view":"all","id":"p0165"},"#name":"para"}],"$":{"id":"u0035"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$$":[{"#name":"italic","_":"Generative"},{"#name":"__text__","_":" models will reveal candidate mechanisms underlying behavior and disease."}],"$":{"view":"all","id":"p0170"},"#name":"para"}],"$":{"id":"u0040"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$$":[{"#name":"__text__","_":"The advantages of "},{"#name":"italic","_":"frequentist"},{"#name":"__text__","_":" and "},{"#name":"italic","_":"Bayesian"},{"#name":"__text__","_":" modeling will be more often combined."}],"$":{"view":"all","id":"p0175"},"#name":"para"}],"$":{"id":"u0045"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$$":[{"#name":"italic","_":"Null-hypothesis testing"},{"#name":"__text__","_":" and "},{"#name":"italic","_":"out-of-sample generalization"},{"#name":"__text__","_":" will draw formal inference."}],"$":{"view":"all","id":"p0180"},"#name":"para"}],"$":{"id":"u0050"},"#name":"list-item"}],"$":{"id":"li0010"},"#name":"list"}],"$":{"view":"all","id":"sp0060"},"#name":"simple-para"}],"$":{"view":"all","id":"abs0015"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0015","class":"author-highlights"},"#name":"abstract"}],"$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"#name":"abstracts"},"pdf":{"urlType":"download","url":"/science/article/pii/S1053811917303816/pdfft?md5=e11ce2ed5959b164b92929c95aef8e0a&pid=1-s2.0-S1053811917303816-main.pdf"}},{"articleTitle":"Bayesian Models of Functional Connectomics and Behavior","authors":"Dâ€™Souza N.S.","doi":"10.48550/arXiv.2301.06182","externalArticle":true,"openAccess":0,"publicationDate":"15 January 2023","publicationTitle":"arXiv","publicationYear":"2023","scopusArticleUrl":"http://www.scopus.com/scopus/inward/record.url?partnerID=10&rel=3.0.0&view=basic&eid=2-s2.0-85146841325&md5=fc79ef1527dede2e0ea4246591bd11","scopusEid":"2-s2.0-85146841325","thirdParty":false,"abstract":{},"pdf":{}},{"articleTitle":"Improved Brain Community Structure Detection by Two-Step Weighted Modularity Maximization","authors":"Guo Z., Zhao X., Yao L., Long Z.","externalArticle":true,"openAccess":0,"publicationDate":"7 November 2022","publicationTitle":"SSRN","publicationYear":"2022","scopusArticleUrl":"http://www.scopus.com/scopus/inward/record.url?partnerID=10&rel=3.0.0&view=basic&eid=2-s2.0-85143440956&md5=1380e55bfddec334757cd492b2da9423","scopusEid":"2-s2.0-85143440956","thirdParty":false,"abstract":{},"pdf":{}},{"articleTitle":"Constraining functional coactivation with a cluster-based structural connectivity network","authors":"Kang I., Galdo M., Turner B.M.","doi":"10.1162/netn_a_00242","externalArticle":true,"openAccess":0,"page":"1032-1065","publicationDate":"1 October 2022","publicationTitle":"Network Neuroscience","publicationYear":"2022","scopusArticleUrl":"http://www.scopus.com/scopus/inward/record.url?partnerID=10&rel=3.0.0&view=basic&eid=2-s2.0-85140890762&md5=8c7bd7cb1edc4ebc2e03f37c0834296","scopusEid":"2-s2.0-85140890762","thirdParty":false,"volume":"Volume 6","abstract":{},"pdf":{}}]},"combinedContentItems":{"content":[{"#name":"keywords","$$":[{"#name":"keywords","$":{"xmlns:ce":true,"xmlns:aep":true,"xmlns:xoe":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:sa":true,"xmlns:ja":true,"xmlns":true,"id":"ks0005","view":"all","class":"keyword"},"$$":[{"#name":"section-title","$":{"id":"st0015"},"_":"Keywords"},{"#name":"keyword","$":{"id":"kw0030"},"$$":[{"#name":"text","$":{"id":"tx0010"},"_":"Complex network"}]},{"#name":"keyword","$":{"id":"kw0035"},"$$":[{"#name":"text","$":{"id":"tx0015"},"_":"Graph theory"}]},{"#name":"keyword","$":{"id":"kw0040"},"$$":[{"#name":"text","$":{"id":"tx0020"},"_":"Infinite Relational Model"}]},{"#name":"keyword","$":{"id":"kw0045"},"$$":[{"#name":"text","$":{"id":"tx0025"},"_":"Bayesian Community Detection"}]},{"#name":"keyword","$":{"id":"kw0050"},"$$":[{"#name":"text","$":{"id":"tx0030"},"_":"Resting state fMRI"}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"crossMark":{"isOpen":false},"domainConfig":{"cdnAssetsHost":"https://sdfestaticassets-us-east-1.sciencedirectassets.com","assetRoute":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/f428ac6e436879961ed6bb261fe095ecf4ea09e0"},"downloadIssue":{"openOnPageLoad":false,"isOpen":false,"downloadCapOpen":false,"articles":[],"selected":[]},"enrichedContent":{"tableOfContents":false,"researchData":{"hasResearchData":false,"dataProfile":{},"openData":{},"mendeleyData":{},"databaseLinking":{}},"geospatialData":{"attachments":[]},"interactiveCaseInsights":{},"virtualMicroscope":{}},"entitledRecommendations":{"openOnPageLoad":false,"isOpen":false,"articles":[],"selected":[],"currentPage":1,"totalPages":1},"exam":{},"glossary":{},"issueNavigation":{"previous":{},"next":{}},"linkingHubLinks":{},"metrics":{"isOpen":true},"preview":{"content":[{"#name":"introduction","$$":[{"#name":"section","$":{"xmlns:ce":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:sa":true,"xmlns:ja":true,"xmlns":true,"id":"s0005","view":"all"},"$$":[{"#name":"section-title","$":{"id":"st0020"},"_":"Introduction"},{"#name":"para","$":{"id":"p0005","view":"all"},"_":"Analysis of resting state functional magnetic resonance imaging (rs-fMRI) has emerged as a powerful research tool to study whole-brain functional connectivity. Since rs-fMRI provides information about intrinsic fluctuations in functional connectivity within and among brain networks, many conventional analysis schemes applied in task-related fMRI studies are irrelevant. Hence, a number of new techniques have been developed based on identification of stable spatio-temporal multivariate structure in the brain wide set of blood oxygen level dependent (BOLD) time series."},{"#name":"para","$":{"id":"p0010","view":"all"},"_":"Using correlation methods or flexible multivariate techniques like independent component analysis (ICA) it has been shown that the BOLD signals of distant brain regions are coordinated suggesting interaction as they form so-called resting-state networks. The number and precise definition of these networks are controversial but several networks are broadly accepted, including the default mode network, motor network, visual network, fronto-parietal, and dorsal attention network (Damoiseaux et al., 2006). In addition to signals reflecting neuronal activity, the BOLD signal may be contaminated by physiological noise stemming from respiratory and cardiac cycles and head motion (Birn et al., 2006, Power et al., 2014)."},{"#name":"para","$":{"id":"p0015","view":"all"},"_":"Complex network analysis is a very active research field (BarabÃ¡si, 2003) that has already found application in neuroimaging and in modeling resting state connectivity (Bullmore and Bassett, 2011, Sporns, 2011). The basic object is the â€˜network graphâ€™. When applied to neuroimage analysis the network graph is formed by brain regions represented as nodes. Nodes are connected by a link if brain regions are co-activated above a certain threshold. In rs-fMRI co-activation is often measured simply by calculating correlation between time series."},{"#name":"para","$":{"id":"p0020","view":"all"},"$$":[{"#name":"__text__","_":"Network structure can be studied at many levels, from local motifs to global features like scale free link distributions signifying long-range coordination (van den Heuvel et al., 2008). Likewise, dense connections between high degree nodes are referred to as â€˜rich club organizationâ€™ (van den Heuvel and Sporns, 2011). At the intermediate level we may identify clusters of highly linked nodes, i.e., high within-cluster link density and low link density to nodes in other clusters. By analogy to social networks such groups are referred to as "},{"#name":"italic","_":"communities"},{"#name":"__text__","_":". The presence of community structure in a network can be quantified by the global "},{"#name":"italic","_":"modularity"},{"#name":"__text__","_":" index (Newman, 2006). Modularity can also be used to identify communities, i.e., by clustering nodes such that the modularity index is maximized (Lehmann and Hansen, 2007, Newman, 2006). Bassett et al. (2011) showed that â€˜flexibilityâ€™, a measure for the number of cluster-assignment changes for nodes in a modularity optimized node-partition across time, is predictive for the amount of learning in a motor task in a subsequent session. Stevens et al. (2012) showed that modularity predicts visual working memory capacity, and Meunier et al. (2009) found that modularity is reduced during normal aging. Likewise, evidence is emerging that global modularity can be used as a bio-marker. For instance patients with childhood-onset schizophrenia have reduced modularity of their resting state networks (Alexander-Bloch et al., 2010). However, focusing on modularity as the single summary of a complex network may be overly simplistic as the modularity measure does not account for variability in the inter-linking relations between functional clusters. Hence, modularity driven clustering might not reveal all salient aspects of community structure in a network. Indeed, modularity has been criticized for its lack of flexibility as a measure of community structure (Fortunato and BarthÃ©lemy, 2007)."}]},{"#name":"para","$":{"id":"p0025","view":"all"},"_":"A better understanding of this important mid-level structure in brain networks requires methods that can capture more informative representations of community structure. For this we turn to a family of expressive generative network models. Relational models are statistical generalizations of graph clustering that not only consider the within-cluster density but also take the specific relations between clusters into consideration. The Infinite Relational Model (IRM) (Kemp et al., 1999, Xu et al., 2006) is a non-parametric generalization of the stochastic block model (Nowicki and Snijders, 2001) for inference of such generalized group structure in complex networks. As the IRM representation considers both linking within and between groups, a highly inter-linked group of nodes could in fact be clustered in different groups if they link in different ways to other clusters, i.e., the IRM can infer more general group structures beyond the conventional community structure. An additional feature of the IRM type of model is that it conveniently allows for analysis of multi-graph networks, which for neuroimaging data could be graphs from multiple sessions or subjects. For multi-subject analysis one could look for a common node clustering structure over subjects but allow individual subject cluster linking densities (MÃ¸rup et al., 2010) or test the hypothesis that both clustering and link structure are shared between all subjects (Andersen et al., 2012b)."},{"#name":"para","$":{"id":"p0030","view":"all"},"_":"A constrained variant of the IRM representing the community structure of graphs in the sense of grouping highly connected node sets was proposed recently by MÃ¸rup and Schmidt (2012). The Bayesian Community Detection (BCD) scheme restricts the between-cluster link densities to be strictly lower than within-cluster link densities, thus constraining the more general IRM to conform with the notion of a community in a social network. Another constraint is introduced by the so-called Infinite Diagonal Model (IDM) (MÃ¸rup and Schmidt, 2012, Schmidt and MÃ¸rup, 2013). The IDM allows for differential within-cluster link densities but models only a single between-cluster density and as such the variability in the link densities between clusters is neglected when inferring the clustering structure. Since the between-cluster link density is shared across clusters, it can be thought of as a background-noise density."},{"#name":"para","$":{"id":"p0035","view":"all"},"$$":[{"#name":"__text__","_":"It should be noted that certain metrical properties can be expected when basing the graph on simple time series correlation, thereby assuming stationarity. If node "},{"#name":"italic","_":"A"},{"#name":"__text__","_":" is highly correlated with node "},{"#name":"italic","_":"B"},{"#name":"__text__","_":", and "},{"#name":"italic","_":"B"},{"#name":"__text__","_":" is highly correlated with "},{"#name":"italic","_":"C"},{"#name":"__text__","_":", then there is a lower limit on the correlation between nodes "},{"#name":"italic","_":"A"},{"#name":"__text__","_":" and "},{"#name":"italic","_":"C"},{"#name":"__text__","_":" which can be inferred by the triangle inequality (Zalesky et al., 2012). This bound will support the formation of community structure, as in social relations: â€˜Friends of friends are friendsâ€™, however, we also note that by thresholding the correlation, the impact on the community structure of these geometrical constraints is non-trivial."}]},{"#name":"para","$":{"id":"p0040","view":"all"},"_":"Spatial grouping of brain regions by similarity of BOLD time series as pursued in the present work can be seen as complementary to classical approaches to spatial grouping such as time series clustering (Goutte et al., 1999) and independent component analysis (ICA) (McKeown et al., 1998, McKeown et al., 2003). Compared with conventional clustering, the relational modeling approach has the advantage that clusters are formed by considering the connectivity patterns both within and between clusters, and furthermore relational models avoid the formation of a group prototype, hence allow for more flexible group structures to be found (Kemp et al., 1999). The use of ICA is based on assumptions of independence either in spatial or temporal dimensions, which can be questioned in the resting state as it has been observed that components are negatively correlated in time and have extensive overlaps in space (Fox et al., 2005)."},{"#name":"para","$":{"id":"p0045","view":"all"},"_":"In this study, we apply the above-mentioned community detection schemes to rs-fMRI data acquired in three cohorts of healthy volunteers and investigate to which degree functional brain networks as measured by rs-fMRI exhibit community structure. The three Bayesian relational methods, i.e. IRM, BCD, and IDM, for inference of group structure in complex networks differ only in the way the link probabilities between clusters are modeled. The rich link structures of the relational models can be seen as a way of inferring functional integration at the inter-community level as discussed in Hagmann et al. (2008) and Sporns (2013). We evaluate the performance of these models with respect to their ability to predict out-of-sample data (predictability) and the robustness of their clustering under re-sampling of data (reproducibility) using the NPAIRS split-half framework (Strother et al., 2002). The evaluation is carried out on three datasets from different sites and the models are evaluated both within and between sites for several thresholds of the correlation matrices. In addition, we compare the three models with three other methods for grouping nodes into clusters, namely Infomap, Louvain modularity, and hierarchical clustering. The work in this paper builds on work presented in Andersen et al. (2012b)."}]}]},{"#name":"section","$$":[{"#name":"section","$":{"xmlns:ce":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:sa":true,"xmlns:ja":true,"xmlns":true,"id":"s0010","view":"all"},"$$":[{"#name":"section-title","$":{"id":"st0025"},"_":"Methods"},{"#name":"para","$":{"id":"p0050","view":"all"},"_":"For generality we investigate three rs-fMRI datasets. One dataset acquired locally at the Danish Research Centre for Magnetic Resonance (Copenhagen) and two other rs-fMRI datasets publicly available in the FCON1000 database (Biswal et al., 2010) (viz., the â€˜Beijingâ€™ and the â€˜Leipzigâ€™ datasets)."},{"#name":"section","$":{"id":"s0015","view":"all"},"$$":[{"#name":"section-title","$":{"id":"st0030"},"_":"Copenhagen data"},{"#name":"para","$":{"id":"p0055","view":"all"},"_":"The Copenhagen dataset included 30 healthy controls with no history of neurological or psychiatric disease. At the day of scanning all subjects were asked to refrain from caffeine, cigarettes or alcohol"}]}]}]},{"#name":"section","$$":[{"#name":"section","$":{"xmlns:ce":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:sa":true,"xmlns:ja":true,"xmlns":true,"id":"s0095","view":"all"},"$$":[{"#name":"section-title","$":{"id":"st0110"},"_":"Results"},{"#name":"section","$":{"id":"s0100","view":"all"},"$$":[{"#name":"section-title","$":{"id":"st0115"},"_":"Estimated clusters"},{"#name":"para","$":{"id":"p0205","view":"all"},"$$":[{"#name":"__text__","_":"We thresholded the graphs to maintain the top 8% correlations. The threshold corresponds to a mean (std) p-value across subjects of 4.75"},{"#name":"hsp","$":{"sp":"0.25"}},{"#name":"__text__","_":"âˆ—"},{"#name":"hsp","$":{"sp":"0.25"}},{"#name":"__text__","_":"10"},{"#name":"sup","$":{"loc":"post"},"$$":[{"#name":"__text__","_":"âˆ’"},{"#name":"hsp","$":{"sp":"0.10"}},{"#name":"__text__","_":"5"}]},{"#name":"__text__","_":"(1.80"},{"#name":"hsp","$":{"sp":"0.25"}},{"#name":"__text__","_":"âˆ—"},{"#name":"hsp","$":{"sp":"0.25"}},{"#name":"__text__","_":"10"},{"#name":"sup","$":{"loc":"post"},"$$":[{"#name":"__text__","_":"âˆ’"},{"#name":"hsp","$":{"sp":"0.10"}},{"#name":"__text__","_":"4"}]},{"#name":"__text__","_":"). The reproducibility between solutions found with different restarts was measured as the NMI between samples with the highest value of the posterior distribution for each run. This was done within all three methods and between the methods and results are shown in TableÂ 1 along with the number of clusters estimated by each of the methods. For"}]}]}]}]},{"#name":"section","$$":[{"#name":"section","$":{"xmlns:ce":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:sa":true,"xmlns:ja":true,"xmlns":true,"id":"s0115","view":"all"},"$$":[{"#name":"section-title","$":{"id":"st0130"},"_":"Discussion and conclusion"},{"#name":"para","$":{"id":"p0255","view":"all"},"_":"Our aim was to explore statistical models for finding structure in networks at the intermediate level. Accumulated evidence points to the importance of community structure in brain networks, hence, we tested three statistical link models, which differed in terms of the different restrictions that were imposed on how nodes are clustered. The IRM is a very flexible representation for graph clustering, in which nodes can be grouped together without having a high link density among them. The BCD is "}]}]},{"#name":"section","$$":[{"#name":"acknowledgment","$":{"xmlns:ce":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:sa":true,"xmlns:ja":true,"xmlns":true,"id":"ac0005","view":"all"},"$$":[{"#name":"section-title","$":{"id":"st0140"},"_":"Acknowledgment"},{"#name":"para","$":{"id":"p0305","view":"all"},"$$":[{"#name":"__text__","_":"This work is funded by a project grant from the "},{"#name":"grant-sponsor","$":{"id":"gts0005","sponsor-id":"http://dx.doi.org/10.13039/501100003554","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"Lundbeck Foundation"},{"#name":"__text__","_":" to Hartwig Siebner (grant-nr "},{"#name":"grant-number","$":{"refid":"gts0005"},"_":"R48 A4846"},{"#name":"__text__","_":"). The Magnetom Trio MR scanner was donated by the Simon Spies Foundation."}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"purchasePdf":{"isOpen":false},"rawtext":"","recommendations":{"articles":[{"pii":"S1053811914005886","doi":"10.1016/j.neuroimage.2014.07.016","journalTitle":"NeuroImage","publicationYear":"2014","publicationDate":"2014-10-15","volumeSupText":"Volume 100","articleNumber":"","pageRange":"703-705","trace-token":"AAAAQJyOCwzcGbLeWZFGdHWs3EDsJX6fmtsAiOVF0QSVqpRioQGdOxpwS4XEI_Bl83Pf18pYDxP1lWc2G3fopMfsb-bhBgqcdls2d3UPZ-HprqFf5qY-EQ","authors":{"content":[{"#name":"author-group","$":{"id":"ag0005"},"$$":[{"#name":"author","$":{"id":"au0005"},"$$":[{"#name":"given-name","_":"Sean C.L."},{"#name":"surname","_":"Deoni"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0010"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cr0005","id":"cf0015"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"âŽ"}]},{"#name":"e-address","$":{"id":"em0005","type":"email"},"_":"sdeonii@brown.edu"}]},{"#name":"author","$":{"id":"au0010"},"$$":[{"#name":"given-name","_":"Douglas C."},{"#name":"surname","_":"Dean"},{"#name":"suffix","_":"III"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0020"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]}]},{"#name":"author","$":{"id":"au0015"},"$$":[{"#name":"given-name","_":"Lindsay"},{"#name":"surname","_":"Walker"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0025"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]}]},{"#name":"author","$":{"id":"au0020"},"$$":[{"#name":"given-name","_":"Holly"},{"#name":"surname","_":"Dirks"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0030"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]}]},{"#name":"author","$":{"id":"au0025"},"$$":[{"#name":"given-name","_":"Jonathan"},{"#name":"surname","_":"O'Muircheartaigh"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0035"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0040"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]},{"#name":"affiliation","$":{"id":"af0005"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","$":{"id":"tn0010"},"_":"Advanced Baby Imaging Lab, School of Engineering, Brown University, Providence, RI 02912, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Advanced Baby Imaging Lab"},{"#name":"organization","_":"School of Engineering"},{"#name":"organization","_":"Brown University"},{"#name":"city","_":"Providence"},{"#name":"state","_":"RI"},{"#name":"postal-code","_":"02912"},{"#name":"country","_":"USA"}]}]},{"#name":"affiliation","$":{"id":"af0010"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","$":{"id":"tn0015"},"_":"Department of Neuroimaging, King's College London, Institute of Psychiatry, De Crespigny Park, London SE5 8AF, UK"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Neuroimaging"},{"#name":"organization","_":"King's College London"},{"#name":"organization","_":"Institute of Psychiatry"},{"#name":"address-line","_":"De Crespigny Park"},{"#name":"city","_":"London"},{"#name":"postal-code","_":"SE5 8AF"},{"#name":"country","_":"UK"}]}]},{"#name":"correspondence","$":{"id":"cr0005"},"$$":[{"#name":"label","_":"âŽ"},{"#name":"text","$":{"id":"tx0005"},"_":"Corresponding author at: Advanced Baby Imaging Lab, School of Engineering, Brown University, Providence, RI 02912, USA."}]}]}],"floats":[],"footnotes":[],"attachments":[]},"lastAuthor":{"content":[{"#name":"author","$":{"id":"au0025"},"$$":[{"#name":"given-name","_":"Jonathan"},{"#name":"surname","_":"O'Muircheartaigh"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0035"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0040"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]}],"floats":[],"footnotes":[],"attachments":[]},"title":{"content":[{"#name":"article-footnote","$":{"id":"aep-article-footnote-id1"},"$$":[{"#name":"label","_":"â˜†"},{"#name":"note-para","$":{"id":"np0005","view":"all"},"$$":[{"#name":"__text__","_":"Manuscript prepared for submission to NeuroImage ("},{"#name":"italic","_":"Comments and Controversies"},{"#name":"__text__","_":")."}]}]},{"#name":"dochead","$":{"id":"dh0005"},"$$":[{"#name":"textfn","$":{"id":"tn0005"},"_":"Comments and Controversies"}]},{"#name":"title","$":{"id":"ti0005"},"_":"Nutritional influences on early white matter development: Response to Anderson and Burggren"}],"floats":[],"footnotes":[{"#name":"article-footnote","$":{"id":"aep-article-footnote-id1"},"$$":[{"#name":"label","_":"â˜†"},{"#name":"note-para","$":{"id":"np0005","view":"all"},"$$":[{"#name":"__text__","_":"Manuscript prepared for submission to NeuroImage ("},{"#name":"italic","_":"Comments and Controversies"},{"#name":"__text__","_":")."}]}]}],"attachments":[]},"openArchive":false,"openAccess":false,"document-subtype":"dis","content-family":"serial","contentType":"JL","abstract":{"$$":[{"$$":[{"$":{"id":"st0005"},"#name":"section-title","_":"Abstract"},{"$$":[{"$$":[{"#name":"__text__","_":"Does breastfeeding alter early brain development? In a recent retrospective study, our group examined the cross-sectional relationship between early infant feeding practice and white matter maturation and cognitive development. In groups matched for child and mother age, gestation duration, birth weight, gender distribution, and socio-economic status; we observed that children who were breastfed exclusively for at least 3"},{"$":{"sp":"0.25"},"#name":"hsp"},{"#name":"__text__","_":"months showed, on average, increased white matter myelin development compared to children who either were exclusively formula-fed, or received a mixture of breast milk and formula. In secondary analysis on sub-sets of these children, again matched for important confounding variables, we found improved cognitive test scores of receptive language in the exclusively breast-fed children compared to formula or formula"},{"$":{"sp":"0.25"},"#name":"hsp"},{"#name":"__text__","_":"+"},{"$":{"sp":"0.25"},"#name":"hsp"},{"#name":"__text__","_":"breast-fed children; and that prolonged breastfeeding was associated with increased motor, language, and visual functioning in exclusively breast-fed children. In response to this work, Anderson and Burggren have questioned our methodology and, by association, our findings. Further, they use their critique as a platform for advancing an alternative interpretation of our findings: that observed results were not associated with prolonged breast-feeding, but rather delayed the introduction of cow's milk. In this response, we address and clarify some of the misconceptions presented by Anderson and Burggren."}],"$":{"view":"all","id":"sp0005"},"#name":"simple-para"}],"$":{"view":"all","id":"as0005"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0005","class":"author"},"#name":"abstract"},{"$$":[{"$":{"id":"st0010"},"#name":"section-title","_":"Highlights"},{"$$":[{"$$":[{"$$":[{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0045"},"#name":"para","_":"Address main concerns highlighted in a comment by Anderson and Burggren"}],"$":{"id":"u0005"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$$":[{"#name":"__text__","_":"Breastfed infants have improved cognitive development by 2"},{"$":{"sp":"0.25"},"#name":"hsp"},{"#name":"__text__","_":"years of age."}],"$":{"view":"all","id":"p0050"},"#name":"para"}],"$":{"id":"u0010"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0055"},"#name":"para","_":"Duration of breastfeeding is positively associated with behavioral performance."}],"$":{"id":"u0015"},"#name":"list-item"}],"$":{"id":"l0005"},"#name":"list"}],"$":{"view":"all","id":"sp0025"},"#name":"simple-para"}],"$":{"view":"all","id":"as0010"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0010","class":"author-highlights"},"#name":"abstract"}],"$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"#name":"abstracts"},"pdf":{"urlType":"download","url":"/science/article/pii/S1053811914005886/pdfft?md5=cf81a764cd1f8574d1e5c48969ec00b9&pid=1-s2.0-S1053811914005886-main.pdf"},"iss-first":"","vol-first":"100","isThirdParty":false,"issn-primary-unformatted":"10538119","issn-primary-formatted":"1053-8119"},{"pii":"S1053811914005254","doi":"10.1016/j.neuroimage.2014.06.048","journalTitle":"NeuroImage","publicationYear":"2014","publicationDate":"2014-10-15","volumeSupText":"Volume 100","articleNumber":"","pageRange":"619-627","trace-token":"AAAAQJyOCwzcGbLeWZFGdHWs3EDsJX6fmtsAiOVF0QSVqpRiaqBdxV3GhHlw27VUkcQn_WrjYKLkQUgu0DcCXQiZo1OpBNAP4cVjLSG015fOh58NkwMExA","authors":{"content":[{"#name":"author-group","$":{"id":"ag0005"},"$$":[{"#name":"author","$":{"id":"au0005"},"$$":[{"#name":"given-name","_":"Dae-Jin"},{"#name":"surname","_":"Kim"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0010"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]}]},{"#name":"author","$":{"id":"au0010"},"$$":[{"#name":"given-name","_":"Elysia Poggi"},{"#name":"surname","_":"Davis"},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0015"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0020"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au0015"},"$$":[{"#name":"given-name","_":"Curt A."},{"#name":"surname","_":"Sandman"},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0025"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au0020"},"$$":[{"#name":"given-name","_":"Olaf"},{"#name":"surname","_":"Sporns"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0030"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]}]},{"#name":"author","$":{"id":"au0025"},"$$":[{"#name":"given-name","_":"Brian F."},{"#name":"surname","_":"O'Donnell"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0035"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]}]},{"#name":"author","$":{"id":"au0030"},"$$":[{"#name":"given-name","_":"Claudia"},{"#name":"surname","_":"Buss"},{"#name":"cross-ref","$":{"refid":"af0020","id":"cf0040"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]}]},{"#name":"author","$":{"id":"au0035"},"$$":[{"#name":"given-name","_":"William P."},{"#name":"surname","_":"Hetrick"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0045"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cr0005","id":"cf0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"âŽ"}]},{"#name":"e-address","$":{"id":"em0005","type":"email"},"_":"whetrick@indiana.edu"}]},{"#name":"affiliation","$":{"id":"af0005"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","$":{"id":"tn0005"},"_":"Department of Psychological and Brain Sciences, Indiana University, 1101 East 10th Street, Bloomington, IN 47405, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Psychological and Brain Sciences"},{"#name":"organization","_":"Indiana University"},{"#name":"address-line","_":"1101 East 10th Street"},{"#name":"city","_":"Bloomington"},{"#name":"state","_":"IN"},{"#name":"postal-code","_":"47405"},{"#name":"country","_":"USA"}]}]},{"#name":"affiliation","$":{"id":"af0010"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","$":{"id":"tn0010"},"_":"Department of Psychology, University of Denver, 2155 South Race Street, Denver, CO 80208, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Psychology"},{"#name":"organization","_":"University of Denver"},{"#name":"address-line","_":"2155 South Race Street"},{"#name":"city","_":"Denver"},{"#name":"state","_":"CO"},{"#name":"postal-code","_":"80208"},{"#name":"country","_":"USA"}]}]},{"#name":"affiliation","$":{"id":"af0015"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","$":{"id":"tn0015"},"_":"Department of Psychiatry and Human Behavior, University of California Irvine, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Psychiatry and Human Behavior"},{"#name":"organization","_":"University of California Irvine"},{"#name":"country","_":"USA"}]}]},{"#name":"affiliation","$":{"id":"af0020"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","$":{"id":"tn0020"},"_":"Institut fÃ¼r Medizinische Psychologie, CharitÃ© Centrum fÃ¼r Human-und Gesundheitswissenschaften, CharitÃ© UniversitÃ¤tsmedizin, Berlin, Germany"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Institut fÃ¼r Medizinische Psychologie"},{"#name":"organization","_":"CharitÃ© Centrum fÃ¼r Human-und Gesundheitswissenschaften"},{"#name":"organization","_":"CharitÃ© UniversitÃ¤tsmedizin"},{"#name":"city","_":"Berlin"},{"#name":"country","_":"Germany"}]}]},{"#name":"correspondence","$":{"id":"cr0005"},"$$":[{"#name":"label","_":"âŽ"},{"#name":"text","$":{"id":"tx0005"},"$$":[{"#name":"__text__","_":"Corresponding author at: Department of Psychological and Brain Sciences, Indiana University, Bloomington, IN 47405, USA. Fax: +"},{"#name":"hsp","$":{"sp":"0.10"}},{"#name":"__text__","_":"1 812 856 4544."}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"lastAuthor":{"content":[{"#name":"author","$":{"id":"au0035"},"$$":[{"#name":"given-name","_":"William P."},{"#name":"surname","_":"Hetrick"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0045"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cr0005","id":"cf0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"âŽ"}]},{"#name":"e-address","$":{"id":"em0005","type":"email"},"_":"whetrick@indiana.edu"}]}],"floats":[],"footnotes":[],"attachments":[]},"title":{"content":[{"#name":"title","$":{"id":"ti0005"},"_":"Longer gestation is associated with more efficient brain networks in preadolescent children"}],"floats":[],"footnotes":[],"attachments":[]},"openArchive":false,"openAccess":false,"document-subtype":"fla","content-family":"serial","contentType":"JL","abstract":{"$$":[{"$$":[{"$":{"id":"st0005"},"#name":"section-title","_":"Abstract"},{"$$":[{"$$":[{"#name":"__text__","_":"Neurodevelopmental benefits of increased gestation have not been fully characterized in terms of network organization. Since brain function can be understood as an integrated network of neural information from distributed brain regions, investigation of the effects of gestational length on network properties is a critical goal of human developmental neuroscience. Using diffusion tensor imaging and fiber tractography, we investigated the effects of gestational length on the small-world attributes and rich club organization of 147 preadolescent children, whose gestational length ranged from 29 to 42"},{"$":{"sp":"0.25"},"#name":"hsp"},{"#name":"__text__","_":"weeks. Higher network efficiency was positively associated with longer gestation. The longer gestation was correlated with increased local efficiency in the posterior medial cortex, including the precuneus, cuneus, and superior parietal regions. Rich club organization was also observed indicating the existence of highly interconnected structural hubs formed in preadolescent children. Connectivity "},{"#name":"italic","_":"among"},{"#name":"__text__","_":" rich club members and "},{"#name":"italic","_":"from"},{"#name":"__text__","_":" rich club regions was positively associated with the length of gestation, indicating the higher level of topological benefits of structural connectivity from longer gestation in the predominant regions of brain networks. The findings provide evidence that longer gestation is associated with improved topological organization of the preadolescent brain, characterized by the increased communication capacity of the brain network and enhanced directional strength of brain connectivity with central hub regions."}],"$":{"view":"all","id":"sp0005"},"#name":"simple-para"}],"$":{"view":"all","id":"as0005"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0005","class":"author"},"#name":"abstract"},{"$$":[{"$":{"id":"st0010"},"#name":"section-title","_":"Highlights"},{"$$":[{"$$":[{"$$":[{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0220"},"#name":"para","_":"Structural network of 147 preadolescents (29â€“42 gestational age: GA) was examined."}],"$":{"id":"u0005"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0225"},"#name":"para","_":"Longer GA was associated with higher global and local network efficiency."}],"$":{"id":"u0010"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0230"},"#name":"para","_":"Presence of â€œrich-clubâ€ organization for preadolescents (6â€“11 ages) was confirmed."}],"$":{"id":"u0015"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0235"},"#name":"para","_":"Connectivity was preferentially enhanced in rich-club regions with longer GA."}],"$":{"id":"u0020"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0240"},"#name":"para","_":"The findings extend the topological organizational benefits from longer gestation."}],"$":{"id":"u0025"},"#name":"list-item"}],"$":{"id":"l0005"},"#name":"list"}],"$":{"view":"all","id":"sp0055"},"#name":"simple-para"}],"$":{"view":"all","id":"as0010"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0010","class":"author-highlights"},"#name":"abstract"}],"$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"#name":"abstracts"},"pdf":{"urlType":"download","url":"/science/article/pii/S1053811914005254/pdfft?md5=73809c45de9d9bc81132cefb2c3a02de&pid=1-s2.0-S1053811914005254-main.pdf"},"iss-first":"","vol-first":"100","isThirdParty":false,"issn-primary-unformatted":"10538119","issn-primary-formatted":"1053-8119"},{"pii":"S1053811914005096","doi":"10.1016/j.neuroimage.2014.06.032","journalTitle":"NeuroImage","publicationYear":"2014","publicationDate":"2014-10-15","volumeSupText":"Volume 100","articleNumber":"","pageRange":"395-404","trace-token":"AAAAQJyOCwzcGbLeWZFGdHWs3EDsJX6fmtsAiOVF0QSVqpRi8QTUHZz6pTX9oaeDy_rOnHMmEUYZHDbZ3Kn9mSdngtUu94aKB4R-dUMv2l0CEaI3dsKdRQ","authors":{"content":[{"#name":"author-group","$":{"id":"ag0005"},"$$":[{"#name":"author","$":{"id":"au0005"},"$$":[{"#name":"given-name","_":"Hui"},{"#name":"surname","_":"Wang"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0010"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]}]},{"#name":"author","$":{"id":"au0010"},"$$":[{"#name":"given-name","_":"Junfeng"},{"#name":"surname","_":"Zhu"},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0015"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]},{"#name":"author","$":{"id":"au0015"},"$$":[{"#name":"given-name","_":"Martin"},{"#name":"surname","_":"Reuter"},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0020"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]},{"#name":"cross-ref","$":{"refid":"af0020","id":"cf0025"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]}]},{"#name":"author","$":{"id":"au0020"},"$$":[{"#name":"given-name","_":"Louis N."},{"#name":"surname","_":"Vinke"},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0030"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au0025"},"$$":[{"#name":"given-name","_":"Anastasia"},{"#name":"surname","_":"Yendiki"},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0035"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au0030"},"$$":[{"#name":"given-name","_":"David A."},{"#name":"surname","_":"Boas"},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0040"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au0035"},"$$":[{"#name":"given-name","_":"Bruce"},{"#name":"surname","_":"Fischl"},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0045"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]},{"#name":"cross-ref","$":{"refid":"af0020","id":"cf0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]}]},{"#name":"author","$":{"id":"au0040"},"$$":[{"#name":"given-name","_":"Taner"},{"#name":"surname","_":"Akkin"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0055"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cr0005","id":"cf0060"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"âŽ"}]},{"#name":"e-address","$":{"id":"em0005","type":"email"},"_":"akkin@umn.edu"}]},{"#name":"affiliation","$":{"id":"af0005"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","$":{"id":"tn0005"},"_":"Department of Biomedical Engineering, University of Minnesota, Minneapolis, MN 55455, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Biomedical Engineering"},{"#name":"organization","_":"University of Minnesota"},{"#name":"city","_":"Minneapolis"},{"#name":"state","_":"MN"},{"#name":"postal-code","_":"55455"},{"#name":"country","_":"USA"}]}]},{"#name":"affiliation","$":{"id":"af0010"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","$":{"id":"tn0010"},"_":"Department of Industrial and Systems Engineering, University of Minnesota, Minneapolis, MN 55455, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Industrial and Systems Engineering"},{"#name":"organization","_":"University of Minnesota"},{"#name":"city","_":"Minneapolis"},{"#name":"state","_":"MN"},{"#name":"postal-code","_":"55455"},{"#name":"country","_":"USA"}]}]},{"#name":"affiliation","$":{"id":"af0015"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","$":{"id":"tn0015"},"_":"Athinoula A. Martinos Center for Biomedical Imaging, Department of Radiology, Massachusetts General Hospital, Harvard Medical School, Charlestown, MA 02129, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Athinoula A. Martinos Center for Biomedical Imaging"},{"#name":"organization","_":"Department of Radiology"},{"#name":"organization","_":"Massachusetts General Hospital"},{"#name":"organization","_":"Harvard Medical School"},{"#name":"city","_":"Charlestown"},{"#name":"state","_":"MA"},{"#name":"postal-code","_":"02129"},{"#name":"country","_":"USA"}]}]},{"#name":"affiliation","$":{"id":"af0020"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","$":{"id":"tn0020"},"_":"Computer Science and Artificial Intelligence Laboratory, Massachusetts Institute of Technology, Cambridge, MA 02139, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Computer Science and Artificial Intelligence Laboratory"},{"#name":"organization","_":"Massachusetts Institute of Technology"},{"#name":"city","_":"Cambridge"},{"#name":"state","_":"MA"},{"#name":"postal-code","_":"02139"},{"#name":"country","_":"USA"}]}]},{"#name":"correspondence","$":{"id":"cr0005"},"$$":[{"#name":"label","_":"âŽ"},{"#name":"text","$":{"id":"tx0005"},"_":"Corresponding author at: 7-105 Hasselmo Hall, 312 Church Street SE, Minneapolis, MN 55455, USA."}]}]}],"floats":[],"footnotes":[],"attachments":[]},"lastAuthor":{"content":[{"#name":"author","$":{"id":"au0040"},"$$":[{"#name":"given-name","_":"Taner"},{"#name":"surname","_":"Akkin"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0055"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cr0005","id":"cf0060"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"âŽ"}]},{"#name":"e-address","$":{"id":"em0005","type":"email"},"_":"akkin@umn.edu"}]}],"floats":[],"footnotes":[],"attachments":[]},"title":{"content":[{"#name":"title","$":{"id":"ti0005"},"_":"Cross-validation of serial optical coherence scanning and diffusion tensor imaging: A study on neural fiber maps in human medulla oblongata"}],"floats":[],"footnotes":[],"attachments":[]},"openArchive":false,"openAccess":false,"document-subtype":"fla","content-family":"serial","contentType":"JL","abstract":{"$$":[{"$$":[{"$":{"id":"st0005"},"#name":"section-title","_":"Abstract"},{"$$":[{"$":{"view":"all","id":"sp0005"},"#name":"simple-para","_":"We established a strategy to perform cross-validation of serial optical coherence scanner imaging (SOCS) and diffusion tensor imaging (DTI) on a postmortem human medulla. Following DTI, the sample was serially scanned by SOCS, which integrates a vibratome slicer and a multi-contrast optical coherence tomography rig for large-scale three-dimensional imaging at microscopic resolution. The DTI dataset was registered to the SOCS space. An average correlation coefficient of 0.9 was found between the co-registered fiber maps constructed by fractional anisotropy and retardance contrasts. Pixelwise comparison of fiber orientations demonstrated good agreement between the DTI and SOCS measures. Details of the comparison were studied in regions exhibiting a variety of fiber organizations. DTI estimated the preferential orientation of small fiber tracts; however, it didn't capture their complex patterns as SOCS did. In terms of resolution and imaging depth, SOCS and DTI complement each other, and open new avenues for cross-modality investigations of the brain."}],"$":{"view":"all","id":"as0005"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0005","class":"author"},"#name":"abstract"},{"$$":[{"$":{"id":"st0010"},"#name":"section-title","_":"Highlights"},{"$$":[{"$$":[{"$$":[{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0235"},"#name":"para","_":"SOCS enables systematic validation of DTI on human brains at microscopic resolution."}],"$":{"id":"u0005"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0240"},"#name":"para","_":"Birefringence and diffusion anisotropy of fiber tracts are highly correlated."}],"$":{"id":"u0010"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0245"},"#name":"para","_":"Global fiber orientations by DTI and SOCS are in remarkably good agreement."}],"$":{"id":"u0015"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0250"},"#name":"para","_":"SOCS resolves complex fiber patterns that are not captured by DTI."}],"$":{"id":"u0020"},"#name":"list-item"}],"$":{"id":"l0005"},"#name":"list"}],"$":{"view":"all","id":"sp0075"},"#name":"simple-para"}],"$":{"view":"all","id":"as0010"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0010","class":"author-highlights"},"#name":"abstract"}],"$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"#name":"abstracts"},"pdf":{"urlType":"download","url":"/science/article/pii/S1053811914005096/pdfft?md5=d3fb33c470041d3c4da55a75ad601d1a&pid=1-s2.0-S1053811914005096-main.pdf"},"iss-first":"","vol-first":"100","isThirdParty":false,"issn-primary-unformatted":"10538119","issn-primary-formatted":"1053-8119"},{"pii":"S1053811914004571","doi":"10.1016/j.neuroimage.2014.05.072","journalTitle":"NeuroImage","publicationYear":"2014","publicationDate":"2014-10-15","volumeSupText":"Volume 100","articleNumber":"","pageRange":"1-14","trace-token":"AAAAQJyOCwzcGbLeWZFGdHWs3EDsJX6fmtsAiOVF0QSVqpRizOajJSXRiYNOIr66OfZ0z4gXn94UZruxJpEg4kx1Rzad7vtXcVcjj8L9KFiyBYsCum9V5A","authors":{"content":[{"#name":"author-group","$":{"id":"ag0005"},"$$":[{"#name":"author","$":{"id":"au0005"},"$$":[{"#name":"given-name","_":"F."},{"#name":"surname","_":"Robineau"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0010"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0015"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"cr0005","id":"cf0020"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"âŽ"}]},{"#name":"e-address","$":{"id":"em0005","type":"email"},"_":"Fabien.Robineau@unige.ch"}]},{"#name":"author","$":{"id":"au0010"},"$$":[{"#name":"given-name","_":"S.W."},{"#name":"surname","_":"Rieger"},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0025"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0030"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au0015"},"$$":[{"#name":"given-name","_":"C."},{"#name":"surname","_":"Mermoud"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0035"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]}]},{"#name":"author","$":{"id":"au0020"},"$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Pichon"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0040"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0045"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au0025"},"$$":[{"#name":"given-name","_":"Y."},{"#name":"surname","_":"Koush"},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0055"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"af0020","id":"cf0060"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]},{"#name":"cross-ref","$":{"refid":"af0025","id":"cf0065"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"e"}]}]},{"#name":"author","$":{"id":"au0030"},"$$":[{"#name":"given-name","_":"D."},{"#name":"surname","_":"Van De Ville"},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0070"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"af0020","id":"cf0075"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]},{"#name":"cross-ref","$":{"refid":"af0025","id":"cf0080"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"e"}]}]},{"#name":"author","$":{"id":"au0035"},"$$":[{"#name":"given-name","_":"P."},{"#name":"surname","_":"Vuilleumier"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0085"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0090"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0095"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au0040"},"$$":[{"#name":"given-name","_":"F."},{"#name":"surname","_":"Scharnowski"},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0100"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"af0020","id":"cf0105"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]},{"#name":"cross-ref","$":{"refid":"af0025","id":"cf0110"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"e"}]}]},{"#name":"affiliation","$":{"id":"af0005"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","$":{"id":"tn0005"},"_":"Laboratory for Behavioral Neurology and Imaging of Cognition, Department of Neuroscience, University Medical Center, Geneva, Switzerland"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Laboratory for Behavioral Neurology and Imaging of Cognition"},{"#name":"organization","_":"Department of Neuroscience"},{"#name":"organization","_":"University Medical Center"},{"#name":"city","_":"Geneva"},{"#name":"country","_":"Switzerland"}]}]},{"#name":"affiliation","$":{"id":"af0010"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","$":{"id":"tn0010"},"_":"Geneva Neuroscience Center, Geneva, Switzerland"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Geneva Neuroscience Center"},{"#name":"city","_":"Geneva"},{"#name":"country","_":"Switzerland"}]}]},{"#name":"affiliation","$":{"id":"af0015"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","$":{"id":"tn0015"},"_":"Swiss Center for Affective Sciences, Geneva, Switzerland"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Swiss Center for Affective Sciences"},{"#name":"city","_":"Geneva"},{"#name":"country","_":"Switzerland"}]}]},{"#name":"affiliation","$":{"id":"af0020"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","$":{"id":"tn0020"},"_":"Department of Radiology and Medical Informatics, CIBM, University of Geneva, Geneva, Switzerland"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Radiology and Medical Informatics"},{"#name":"organization","_":"CIBM"},{"#name":"organization","_":"University of Geneva"},{"#name":"city","_":"Geneva"},{"#name":"country","_":"Switzerland"}]}]},{"#name":"affiliation","$":{"id":"af0025"},"$$":[{"#name":"label","_":"e"},{"#name":"textfn","$":{"id":"tn0025"},"_":"Institute of Bioengineering, Ecole Polytechnique FÃ©dÃ©rale de Lausanne (EPFL), Lausanne, Switzerland"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Institute of Bioengineering"},{"#name":"organization","_":"Ecole Polytechnique FÃ©dÃ©rale de Lausanne (EPFL)"},{"#name":"city","_":"Lausanne"},{"#name":"state","_":"Switzerland"}]}]},{"#name":"correspondence","$":{"id":"cr0005"},"$$":[{"#name":"label","_":"âŽ"},{"#name":"text","$":{"id":"tx0005"},"_":"Corresponding author at: Laboratory for Behavioral Neurology and Imaging of Cognition, Department of Neuroscience, University Medical Center, 1 rue Michel-Servet, 1211 Geneva, Switzerland."}]}]}],"floats":[],"footnotes":[],"attachments":[]},"lastAuthor":{"content":[{"#name":"author","$":{"id":"au0040"},"$$":[{"#name":"given-name","_":"F."},{"#name":"surname","_":"Scharnowski"},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0100"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"af0020","id":"cf0105"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]},{"#name":"cross-ref","$":{"refid":"af0025","id":"cf0110"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"e"}]}]}],"floats":[],"footnotes":[],"attachments":[]},"title":{"content":[{"#name":"title","$":{"id":"ti0005"},"_":"Self-regulation of inter-hemispheric visual cortex balance through real-time fMRI neurofeedback training"}],"floats":[],"footnotes":[],"attachments":[]},"openArchive":false,"openAccess":false,"document-subtype":"fla","content-family":"serial","contentType":"JL","abstract":{"$$":[{"$$":[{"$":{"id":"st0005"},"#name":"section-title","_":"Abstract"},{"$$":[{"$$":[{"#name":"__text__","_":"Recent advances in neurofeedback based on real-time functional magnetic resonance imaging (fMRI) allow for learning to control spatially localized brain activity in the range of millimeters across the entire brain. Real-time fMRI neurofeedback studies have demonstrated the feasibility of self-regulating activation in specific areas that are involved in a variety of functions, such as perception, motor control, language, and emotional processing. In most of these previous studies, participants trained to control activity within one region of interest (ROI). In the present study, we extended the neurofeedback approach by now training healthy participants to control the interhemispheric balance between their left and right visual cortices. This was accomplished by providing feedback based on the difference in activity between a target visual ROI and the corresponding homologue region in the opposite hemisphere. Eight out of 14 participants learned to control the differential feedback signal over the course of 3 neurofeedback training sessions spread over 3"},{"$":{"sp":"0.25"},"#name":"hsp"},{"#name":"__text__","_":"days, i.e., they produced consistent increases in the visual target ROI relative to the opposite visual cortex. Those who learned to control the differential feedback signal were subsequently also able to exert that control in the absence of neurofeedback. Such learning to voluntarily control the balance between cortical areas of the two hemispheres might offer promising rehabilitation approaches for neurological or psychiatric conditions associated with pathological asymmetries in brain activity patterns, such as hemispatial neglect, dyslexia, or mood disorders."}],"$":{"view":"all","id":"sp0005"},"#name":"simple-para"}],"$":{"view":"all","id":"as0005"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0005","class":"author"},"#name":"abstract"},{"$$":[{"$":{"id":"st0010"},"#name":"section-title","_":"Highlights"},{"$$":[{"$$":[{"$$":[{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0345"},"#name":"para","_":"Neurofeedback allows for learning control over the interhemispheric visual cortex balance."}],"$":{"id":"u0005"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0350"},"#name":"para","_":"Learned interhemispheric control is maintained in the absence of neurofeedback."}],"$":{"id":"u0010"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0355"},"#name":"para","_":"Pathological asymmetries in brain activity might be normalized using neurofeedback."}],"$":{"id":"u0015"},"#name":"list-item"}],"$":{"id":"l0005"},"#name":"list"}],"$":{"view":"all","id":"sp0055"},"#name":"simple-para"}],"$":{"view":"all","id":"as0010"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0010","class":"author-highlights"},"#name":"abstract"}],"$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"#name":"abstracts"},"pdf":{"urlType":"download","url":"/science/article/pii/S1053811914004571/pdfft?md5=23e96227e806d187c3611d3c607154ae&pid=1-s2.0-S1053811914004571-main.pdf"},"iss-first":"","vol-first":"100","isThirdParty":false,"issn-primary-unformatted":"10538119","issn-primary-formatted":"1053-8119"},{"pii":"S1053811914006259","doi":"10.1016/j.neuroimage.2014.07.042","journalTitle":"NeuroImage","publicationYear":"2014","publicationDate":"2014-10-15","volumeSupText":"Volume 100","articleNumber":"","pageRange":"663-675","trace-token":"AAAAQJyOCwzcGbLeWZFGdHWs3EDsJX6fmtsAiOVF0QSVqpRi7qDD8I_RVG8Wb7-JpnJDDOJQxutq9UzELwW6FoOYtWua4w9FKjBpAnZEVuefsHIC9vTGqQ","authors":{"content":[{"#name":"author-group","$":{"id":"ag0005"},"$$":[{"#name":"author","$":{"id":"au0005"},"$$":[{"#name":"given-name","_":"Dave R.M."},{"#name":"surname","_":"Langers"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0020"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0025"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"cr0005","id":"cf0030"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"âŽ"}]},{"#name":"e-address","$":{"id":"em0005","type":"email"},"_":"davey.langers@nottingham.ac.uk"}]},{"#name":"author","$":{"id":"au0010"},"$$":[{"#name":"given-name","_":"Rosa M."},{"#name":"surname","_":"Sanchez-Panchuelo"},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0035"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au0015"},"$$":[{"#name":"given-name","_":"Susan T."},{"#name":"surname","_":"Francis"},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0040"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au0020"},"$$":[{"#name":"given-name","_":"Katrin"},{"#name":"surname","_":"Krumbholz"},{"#name":"cross-ref","$":{"refid":"af0020","id":"cf0045"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]}]},{"#name":"author","$":{"id":"au0025"},"$$":[{"#name":"given-name","_":"Deborah A."},{"#name":"surname","_":"Hall"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0055"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]},{"#name":"affiliation","$":{"id":"af0005"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","$":{"id":"tn0005"},"_":"National Institute for Health Research (NIHR) Nottingham Hearing Biomedical Research Unit, University of Nottingham, Nottingham, UK"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"National Institute for Health Research (NIHR) Nottingham Hearing Biomedical Research Unit"},{"#name":"organization","_":"University of Nottingham"},{"#name":"city","_":"Nottingham"},{"#name":"country","_":"UK"}]}]},{"#name":"affiliation","$":{"id":"af0010"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","$":{"id":"tn0010"},"_":"Otology and Hearing Group, Division of Clinical Neuroscience, School of Medicine, University of Nottingham, Nottingham, UK"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Otology and Hearing Group"},{"#name":"organization","_":"Division of Clinical Neuroscience"},{"#name":"organization","_":"School of Medicine"},{"#name":"organization","_":"University of Nottingham"},{"#name":"city","_":"Nottingham"},{"#name":"country","_":"UK"}]}]},{"#name":"affiliation","$":{"id":"af0015"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","$":{"id":"tn0015"},"_":"Sir Peter Mansfield Magnetic Resonance Centre, School of Physics and Astronomy, University of Nottingham, Nottingham, UK"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Sir Peter Mansfield Magnetic Resonance Centre"},{"#name":"organization","_":"School of Physics and Astronomy"},{"#name":"organization","_":"University of Nottingham"},{"#name":"city","_":"Nottingham"},{"#name":"country","_":"UK"}]}]},{"#name":"affiliation","$":{"id":"af0020"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","$":{"id":"tn0020"},"_":"MRC Institute for Hearing Research, Nottingham, UK"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"MRC Institute for Hearing Research"},{"#name":"city","_":"Nottingham"},{"#name":"country","_":"UK"}]}]},{"#name":"correspondence","$":{"id":"cr0005"},"$$":[{"#name":"label","_":"âŽ"},{"#name":"text","$":{"id":"tx0005"},"$$":[{"#name":"__text__","_":"Corresponding author at: NIHR Nottingham Hearing Biomedical Research Unit, Ropewalk House, 113 The Ropewalk, Nottingham NG1 5DU, UK. Fax: +"},{"#name":"hsp","$":{"sp":"0.10"}},{"#name":"__text__","_":"44 115 82 32615."}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"lastAuthor":{"content":[{"#name":"author","$":{"id":"au0025"},"$$":[{"#name":"given-name","_":"Deborah A."},{"#name":"surname","_":"Hall"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0055"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]}],"floats":[],"footnotes":[],"attachments":[]},"title":{"content":[{"#name":"title","$":{"id":"ti0005"},"_":"Neuroimaging paradigms for tonotopic mapping (II): The influence of acquisition protocol"}],"floats":[],"footnotes":[],"attachments":[]},"openArchive":false,"openAccess":true,"document-subtype":"fla","content-family":"serial","contentType":"JL","abstract":{"$$":[{"$$":[{"$":{"id":"st0005"},"#name":"section-title","_":"Abstract"},{"$$":[{"$$":[{"#name":"__text__","_":"Numerous studies on the tonotopic organisation of auditory cortex in humans have employed a wide range of neuroimaging protocols to assess cortical frequency tuning. In the present functional magnetic resonance imaging (fMRI) study, we made a systematic comparison between acquisition protocols with variable levels of interference from acoustic scanner noise. Using sweep stimuli to evoke travelling waves of activation, we measured sound-evoked response signals using "},{"#name":"italic","_":"sparse"},{"#name":"__text__","_":", "},{"#name":"italic","_":"clustered"},{"#name":"__text__","_":", and "},{"#name":"italic","_":"continuous"},{"#name":"__text__","_":" imaging protocols that were characterised by inter-scan intervals of 8.8, 2.2, or 0.0"},{"$":{"sp":"0.25"},"#name":"hsp"},{"#name":"__text__","_":"s, respectively. With regard to sensitivity to sound-evoked activation, the sparse and clustered protocols performed similarly, and both detected more activation than the continuous method. Qualitatively, tonotopic maps in activated areas proved highly similar, in the sense that the overall pattern of tonotopic gradients was reproducible across all three protocols. However, quantitatively, we observed substantial reductions in response amplitudes to moderately low stimulus frequencies that coincided with regions of strong energy in the scanner noise spectrum for the clustered and continuous protocols compared to the sparse protocol. At the same time, extreme frequencies became over-represented for these two protocols, and high best frequencies became relatively more abundant. Our results indicate that although all three scanning protocols are suitable to determine the layout of tonotopic fields, an exact quantitative assessment of the representation of various sound frequencies is substantially confounded by the presence of scanner noise. In addition, we noticed anomalous signal dynamics in response to our travelling wave paradigm that suggest that the assessment of frequency-dependent tuning is non-trivially influenced by time-dependent (hemo)dynamics when using sweep stimuli."}],"$":{"view":"all","id":"sp0005"},"#name":"simple-para"}],"$":{"view":"all","id":"as0005"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0005","class":"author"},"#name":"abstract"},{"$$":[{"$":{"id":"st0010"},"#name":"section-title","_":"Highlights"},{"$$":[{"$$":[{"$$":[{"$$":[{"#name":"label","_":"â€¢"},{"$$":[{"#name":"__text__","_":"We compare tonotopic mapping protocols using "},{"#name":"italic","_":"sparse"},{"#name":"__text__","_":", "},{"#name":"italic","_":"clustered"},{"#name":"__text__","_":", or "},{"#name":"italic","_":"continuous"},{"#name":"__text__","_":" imaging."}],"$":{"view":"all","id":"p0235"},"#name":"para"}],"$":{"id":"u0005"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0240"},"#name":"para","_":"Travelling wave stimuli and Fourier analysis were used to assess frequency tuning."}],"$":{"id":"u0010"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0245"},"#name":"para","_":"All three imaging protocols resulted in qualitatively similar tonotopic maps."}],"$":{"id":"u0015"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0250"},"#name":"para","_":"Frequency-specific interactions and response reductions due to scanner noise occurred."}],"$":{"id":"u0020"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0255"},"#name":"para","_":"Anomalous signal behaviour suggests that hemodynamics confound frequency tuning."}],"$":{"id":"u0025"},"#name":"list-item"}],"$":{"id":"l0005"},"#name":"list"}],"$":{"view":"all","id":"sp0050"},"#name":"simple-para"}],"$":{"view":"all","id":"as0010"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0010","class":"author-highlights"},"#name":"abstract"}],"$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"#name":"abstracts"},"pdf":{"urlType":"download","url":"/science/article/pii/S1053811914006259/pdfft?md5=455f9cbe41c796a8c1baca036ba537ef&pid=1-s2.0-S1053811914006259-main.pdf"},"iss-first":"","vol-first":"100","isThirdParty":false,"issn-primary-unformatted":"10538119","issn-primary-formatted":"1053-8119"},{"pii":"S1053811914004649","doi":"10.1016/j.neuroimage.2014.05.079","journalTitle":"NeuroImage","publicationYear":"2014","publicationDate":"2014-10-15","volumeSupText":"Volume 100","articleNumber":"","pageRange":"51-59","trace-token":"AAAAQJyOCwzcGbLeWZFGdHWs3EDsJX6fmtsAiOVF0QSVqpRiNs1r7j5xuxGJG3gQBkb0nd1EoN9GHpAqocVYcAMvnaUXy4LDPi_rtg10_T_Sjq0ywHRHfw","authors":{"content":[{"#name":"author-group","$":{"id":"ag0005"},"$$":[{"#name":"author","$":{"id":"au0005"},"$$":[{"#name":"given-name","_":"Andrew T."},{"#name":"surname","_":"Curtis"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0045"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]},{"#name":"author","$":{"id":"au0010"},"$$":[{"#name":"given-name","_":"R. Matthew"},{"#name":"surname","_":"Hutchison"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0055"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0015","id":"cf0060"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au0015","orcid":"0000-0002-7916-0263"},"$$":[{"#name":"given-name","_":"Ravi S."},{"#name":"surname","_":"Menon"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0065"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0070"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"cr0005","id":"cf0075"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"âŽ"}]},{"#name":"e-address","$":{"id":"em0005","type":"email"},"_":"rmenon@robarts.ca"}]},{"#name":"affiliation","$":{"id":"af0005"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","$":{"id":"tn0005"},"_":"Centre for Functional and Metabolic Mapping, Robarts Research Institute, The University of Western Ontario, 100 Perth Drive, London, Ontario, N6A 5K8, Canada"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Centre for Functional and Metabolic Mapping"},{"#name":"organization","_":"Robarts Research Institute"},{"#name":"organization","_":"The University of Western Ontario"},{"#name":"address-line","_":"100 Perth Drive"},{"#name":"city","_":"London"},{"#name":"state","_":"Ontario"},{"#name":"postal-code","_":"N6A 5K8"},{"#name":"country","_":"Canada"}]}]},{"#name":"affiliation","$":{"id":"af0010"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","$":{"id":"tn0010"},"_":"Department of Medical Biophysics, The University of Western Ontario, London, Ontario, N6A 5C1, Canada"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Medical Biophysics"},{"#name":"organization","_":"The University of Western Ontario"},{"#name":"city","_":"London"},{"#name":"state","_":"Ontario"},{"#name":"postal-code","_":"N6A 5C1"},{"#name":"country","_":"Canada"}]}]},{"#name":"affiliation","$":{"id":"af0015"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","$":{"id":"tn0015"},"_":"Center for Brain Science, Harvard University, Cambridge, MA 02138, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Center for Brain Science"},{"#name":"organization","_":"Harvard University"},{"#name":"city","_":"Cambridge"},{"#name":"state","_":"MA"},{"#name":"postal-code","_":"02138"},{"#name":"country","_":"USA"}]}]},{"#name":"correspondence","$":{"id":"cr0005"},"$$":[{"#name":"label","_":"âŽ"},{"#name":"text","$":{"id":"tx0005"},"$$":[{"#name":"__text__","_":"Corresponding author at: Robarts Research Institute, The University of Western Ontario, 100 Perth Drive, London, Ontario, N6A 5K8, Canada. Fax: +"},{"#name":"hsp","$":{"sp":"0.10"}},{"#name":"__text__","_":"1 519 931 5260."}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"lastAuthor":{"content":[{"#name":"author","$":{"id":"au0015","orcid":"0000-0002-7916-0263"},"$$":[{"#name":"given-name","_":"Ravi S."},{"#name":"surname","_":"Menon"},{"#name":"cross-ref","$":{"refid":"af0005","id":"cf0065"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"af0010","id":"cf0070"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"cr0005","id":"cf0075"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"âŽ"}]},{"#name":"e-address","$":{"id":"em0005","type":"email"},"_":"rmenon@robarts.ca"}]}],"floats":[],"footnotes":[],"attachments":[]},"title":{"content":[{"#name":"title","$":{"id":"ti0005"},"_":"Phase based venous suppression in resting-state BOLD GE-fMRI"}],"floats":[],"footnotes":[],"attachments":[]},"openArchive":false,"openAccess":false,"document-subtype":"fla","content-family":"serial","contentType":"JL","abstract":{"$$":[{"$$":[{"$":{"id":"st0005"},"#name":"section-title","_":"Abstract"},{"$$":[{"$":{"view":"all","id":"sp0005"},"#name":"simple-para","_":"Resting-state functional MRI (RS-fMRI) is a widely used method for inferring connectivity between brain regions or nodes. As with task-based fMRI, the spatial specificity of the connectivity maps can be distorted by the strong biasing effect of the BOLD signal in macroscopic veins. In RS-fMRI this effect is exacerbated by the temporal coherences of physiological origin between large veins that are widely distributed in the brain. In gradient echo based EPI, used for the vast majority of RS-fMRI, macroscopic veins that carry BOLD-related changes exhibit a strong phase response. This allows for post-processing identification and removal of venous signals using a phase regressor technique. Here, we employ this approach to suppress macrovascular venous contributions in high-field whole-brain RS-fMRI data sets, resulting in significant changes to both the spatial localization of the networks and the correlations between the network nodes. These effects were observed at both the individual and group analysis level, suggesting that venous contamination is a confounding factor for RS-fMRI studies even at relatively low image resolutions. Suppression of the macrovascular signal using the phase regression approach may therefore help to better identify, delineate, and interpret the true structure of large-scale brain networks."}],"$":{"view":"all","id":"as0005"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0005","class":"author"},"#name":"abstract"},{"$$":[{"$":{"id":"st0010"},"#name":"section-title","_":"Highlights"},{"$$":[{"$$":[{"$$":[{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0200"},"#name":"para","_":"A phase regressor is employed to identify macrovascular components of RS-fMRI data."}],"$":{"id":"u0005"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0205"},"#name":"para","_":"Phase regressor is modified to estimate the phase and magnitude signal variances."}],"$":{"id":"u0010"},"#name":"list-item"},{"$$":[{"#name":"label","_":"â€¢"},{"$":{"view":"all","id":"p0210"},"#name":"para","_":"Removal of macrovascular signal changes measured RS correlations and connectivity maps."}],"$":{"id":"u0015"},"#name":"list-item"}],"$":{"id":"l0005"},"#name":"list"}],"$":{"view":"all","id":"sp0050"},"#name":"simple-para"}],"$":{"view":"all","id":"as0010"},"#name":"abstract-sec"}],"$":{"view":"all","id":"ab0010","class":"author-highlights"},"#name":"abstract"}],"$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"#name":"abstracts"},"pdf":{"urlType":"download","url":"/science/article/pii/S1053811914004649/pdfft?md5=9b9021b87ca760cb13256268a65d8d59&pid=1-s2.0-S1053811914004649-main.pdf"},"iss-first":"","vol-first":"100","isThirdParty":false,"issn-primary-unformatted":"10538119","issn-primary-formatted":"1053-8119"}]},"references":{"content":[{"#name":"bibliography","$":{"xmlns:ce":true,"xmlns:aep":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:sa":true,"xmlns:ja":true,"xmlns":true,"id":"bi0005","view":"all"},"$$":[{"#name":"section-title","$":{"id":"st0175"},"_":"References"},{"#name":"bibliography-sec","$":{"id":"bs0005","view":"all"},"$$":[{"#name":"bib-reference","$":{"id":"bb0005"},"$$":[{"#name":"label","_":"Aldous, 1985"},{"#name":"reference","$":{"id":"rf0005","refId":"1"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"D.J."},{"#name":"surname","_":"Aldous"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Exchangeability and related topics"}]}]},{"#name":"host","$$":[{"#name":"book","$$":[{"#name":"book-series","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Ã‰cole d'Ã‰tÃ© de ProbabilitÃ©s de Saint-Flour XIII â€” 1983 (Lecture Notes in Mathematics)"}]},{"#name":"volume-nr","_":"1117"}]}]},{"#name":"date","_":"1985"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"1"},{"#name":"last-page","_":"198"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0010"},"$$":[{"#name":"label","_":"Alexander-Bloch et al., 2010"},{"#name":"reference","$":{"id":"rf0010","refId":"2"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"A.F."},{"#name":"surname","_":"Alexander-Bloch"}]},{"#name":"author","$$":[{"#name":"given-name","_":"N."},{"#name":"surname","_":"Gogtay"}]},{"#name":"author","$$":[{"#name":"given-name","_":"D."},{"#name":"surname","_":"Meunier"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R."},{"#name":"surname","_":"Birn"}]},{"#name":"author","$$":[{"#name":"given-name","_":"L."},{"#name":"surname","_":"Clasen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"F."},{"#name":"surname","_":"Lalonde"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R."},{"#name":"surname","_":"Lenroot"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J."},{"#name":"surname","_":"Giedd"}]},{"#name":"author","$$":[{"#name":"given-name","_":"E.T."},{"#name":"surname","_":"Bullmore"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Disrupted modularity and local connectivity of brain functional networks in childhood-onset schizophrenia"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Front. Syst. Neurosci."}]},{"#name":"volume-nr","_":"4"}]},{"#name":"date","_":"2010"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0015"},"$$":[{"#name":"label","_":"Andersen et al., 2012a"},{"#name":"reference","$":{"id":"rf0015","refId":"3"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"K.W."},{"#name":"surname","_":"Andersen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"K.H."},{"#name":"surname","_":"Madsen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"H."},{"#name":"surname","_":"Siebner"}]},{"#name":"author","$$":[{"#name":"given-name","_":"L.K."},{"#name":"surname","_":"Hansen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"MÃ¸rup"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Identification of functional clusters in the striatum using infinite relational modeling"}]}]},{"#name":"host","$$":[{"#name":"edited-book","$$":[{"#name":"editors","$$":[{"#name":"editor","$$":[{"#name":"given-name","_":"G."},{"#name":"surname","_":"Langs"}]},{"#name":"editor","$$":[{"#name":"given-name","_":"I."},{"#name":"surname","_":"Rish"}]},{"#name":"editor","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"Grosse-Wentrup"}]},{"#name":"editor","$$":[{"#name":"given-name","_":"B."},{"#name":"surname","_":"Murphy"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Machine Learning and Interpretation in Neuroimaging"}]},{"#name":"book-series","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Lecture Notes in Computer Science"}]}]}]},{"#name":"date","_":"2012"},{"#name":"publisher","$$":[{"#name":"name","_":"Springer"},{"#name":"location","_":"Berlin Heidelberg"}]}]},{"#name":"pages","$$":[{"#name":"first-page","_":"226"},{"#name":"last-page","_":"233"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0020"},"$$":[{"#name":"label","_":"Andersen et al., 2012b"},{"#name":"reference","$":{"id":"rf0020","refId":"4"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"K.W."},{"#name":"surname","_":"Andersen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"MÃ¸rup"}]},{"#name":"author","$$":[{"#name":"given-name","_":"H."},{"#name":"surname","_":"Siebner"}]},{"#name":"author","$$":[{"#name":"given-name","_":"K.H."},{"#name":"surname","_":"Madsen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"L.K."},{"#name":"surname","_":"Hansen"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Identifying modular relations in complex brain networks"}]}]},{"#name":"host","$$":[{"#name":"edited-book","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"2012 IEEE International Workshop on Machine Learning for Signal Processing"}]},{"#name":"date","_":"2012"},{"#name":"publisher","$$":[{"#name":"name","_":"IEEE"}]}]},{"#name":"pages","$$":[{"#name":"first-page","_":"1"},{"#name":"last-page","_":"6"}]}]},{"#name":"comment","_":"(Sep.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0025"},"$$":[{"#name":"label","_":"Ashburner and Friston, 2005"},{"#name":"reference","$":{"id":"rf0025","refId":"5"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"J."},{"#name":"surname","_":"Ashburner"}]},{"#name":"author","$$":[{"#name":"given-name","_":"K.J."},{"#name":"surname","_":"Friston"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Unified segmentation"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"NeuroImage"}]},{"#name":"volume-nr","_":"26"}]},{"#name":"issue-nr","_":"3"},{"#name":"date","_":"2005"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"839"},{"#name":"last-page","_":"851"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0030"},"$$":[{"#name":"label","_":"BarabÃ¡si, 2003"},{"#name":"reference","$":{"id":"rf0030","refId":"6"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"A.L."},{"#name":"surname","_":"BarabÃ¡si"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Linked: How Everything is Connected to Everything Else and What It Means for Business, Science, and Everyday Life"}]}]},{"#name":"host","$$":[{"#name":"book","$$":[{"#name":"date","_":"2003"},{"#name":"publisher","$$":[{"#name":"name","_":"Plume Editors"}]}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0035"},"$$":[{"#name":"label","_":"Bassett et al., 2011"},{"#name":"reference","$":{"id":"rf0035","refId":"7"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"D.S."},{"#name":"surname","_":"Bassett"}]},{"#name":"author","$$":[{"#name":"given-name","_":"N.F."},{"#name":"surname","_":"Wymbs"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M.a."},{"#name":"surname","_":"Porter"}]},{"#name":"author","$$":[{"#name":"given-name","_":"P.J."},{"#name":"surname","_":"Mucha"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J.M."},{"#name":"surname","_":"Carlson"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.T."},{"#name":"surname","_":"Grafton"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Dynamic reconfiguration of human brain networks during learning"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Proc. Natl. Acad. Sci. U. S. A."}]},{"#name":"volume-nr","_":"108"}]},{"#name":"issue-nr","_":"18"},{"#name":"date","_":"2011"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"7641"},{"#name":"last-page","_":"7646"}]}]},{"#name":"comment","_":"(May)"}]}]},{"#name":"bib-reference","$":{"id":"bb0040"},"$$":[{"#name":"label","_":"Birn et al., 2006"},{"#name":"reference","$":{"id":"rf0040","refId":"8"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"R.M."},{"#name":"surname","_":"Birn"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J.B."},{"#name":"surname","_":"Diamond"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M.A."},{"#name":"surname","_":"Smith"}]},{"#name":"author","$$":[{"#name":"given-name","_":"P.A."},{"#name":"surname","_":"Bandettini"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Separating respiratory-variation-related fluctuations from neuronal-activity-related fluctuations in fMRI"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"NeuroImage"}]},{"#name":"volume-nr","_":"31"}]},{"#name":"issue-nr","_":"4"},{"#name":"date","_":"2006"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"1536"},{"#name":"last-page","_":"1548"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0045"},"$$":[{"#name":"label","_":"Biswal et al., 2010"},{"#name":"reference","$":{"id":"rf0045","refId":"9"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"B.B."},{"#name":"surname","_":"Biswal"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"Mennes"}]},{"#name":"author","$$":[{"#name":"given-name","_":"X.N."},{"#name":"surname","_":"Zuo"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Gohel"}]},{"#name":"author","$$":[{"#name":"given-name","_":"C."},{"#name":"surname","_":"Kelly"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.M."},{"#name":"surname","_":"Smith"}]},{"#name":"author","$$":[{"#name":"given-name","_":"C.F."},{"#name":"surname","_":"Beckmann"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J.S."},{"#name":"surname","_":"Adelstein"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R.L."},{"#name":"surname","_":"Buckner"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Colcombe"}]},{"#name":"author","$$":[{"#name":"given-name","_":"A.M."},{"#name":"surname","_":"Dogonowski"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"Ernst"}]},{"#name":"author","$$":[{"#name":"given-name","_":"D."},{"#name":"surname","_":"Fair"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"Hampson"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M.J."},{"#name":"surname","_":"Hoptman"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J.S."},{"#name":"surname","_":"Hyde"}]},{"#name":"author","$$":[{"#name":"given-name","_":"V.J."},{"#name":"surname","_":"Kiviniemi"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R."},{"#name":"surname","_":"KÃ¶tter"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.J."},{"#name":"surname","_":"Li"}]},{"#name":"author","$$":[{"#name":"given-name","_":"C.P."},{"#name":"surname","_":"Lin"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M.J."},{"#name":"surname","_":"Lowe"}]},{"#name":"author","$$":[{"#name":"given-name","_":"C."},{"#name":"surname","_":"Mackay"}]},{"#name":"author","$$":[{"#name":"given-name","_":"D.J."},{"#name":"surname","_":"Madden"}]},{"#name":"author","$$":[{"#name":"given-name","_":"K.H."},{"#name":"surname","_":"Madsen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"D.S."},{"#name":"surname","_":"Margulies"}]},{"#name":"author","$$":[{"#name":"given-name","_":"H.S."},{"#name":"surname","_":"Mayberg"}]},{"#name":"author","$$":[{"#name":"given-name","_":"K."},{"#name":"surname","_":"McMahon"}]},{"#name":"author","$$":[{"#name":"given-name","_":"C.S."},{"#name":"surname","_":"Monk"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.H."},{"#name":"surname","_":"Mostofsky"}]},{"#name":"author","$$":[{"#name":"given-name","_":"B.J."},{"#name":"surname","_":"Nagel"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J.J."},{"#name":"surname","_":"Pekar"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.J."},{"#name":"surname","_":"Peltier"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.E."},{"#name":"surname","_":"Petersen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"V."},{"#name":"surname","_":"Riedl"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.a.R.B."},{"#name":"surname","_":"Rombouts"}]},{"#name":"author","$$":[{"#name":"given-name","_":"B."},{"#name":"surname","_":"Rypma"}]},{"#name":"author","$$":[{"#name":"given-name","_":"B.L."},{"#name":"surname","_":"Schlaggar"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Schmidt"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R.D."},{"#name":"surname","_":"Seidler"}]},{"#name":"author","$$":[{"#name":"given-name","_":"G.J."},{"#name":"surname","_":"Siegle"}]},{"#name":"author","$$":[{"#name":"given-name","_":"C."},{"#name":"surname","_":"Sorg"}]},{"#name":"author","$$":[{"#name":"given-name","_":"G.J."},{"#name":"surname","_":"Teng"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J."},{"#name":"surname","_":"Veijola"}]},{"#name":"author","$$":[{"#name":"given-name","_":"A."},{"#name":"surname","_":"Villringer"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"Walter"}]},{"#name":"author","$$":[{"#name":"given-name","_":"L."},{"#name":"surname","_":"Wang"}]},{"#name":"author","$$":[{"#name":"given-name","_":"X.C."},{"#name":"surname","_":"Weng"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Whitfield-Gabrieli"}]},{"#name":"author","$$":[{"#name":"given-name","_":"P."},{"#name":"surname","_":"Williamson"}]},{"#name":"author","$$":[{"#name":"given-name","_":"C."},{"#name":"surname","_":"Windischberger"}]},{"#name":"author","$$":[{"#name":"given-name","_":"Y.F."},{"#name":"surname","_":"Zang"}]},{"#name":"author","$$":[{"#name":"given-name","_":"H.Y."},{"#name":"surname","_":"Zhang"}]},{"#name":"author","$$":[{"#name":"given-name","_":"F.X."},{"#name":"surname","_":"Castellanos"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M.P."},{"#name":"surname","_":"Milham"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Toward discovery science of human brain function"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Proc. Natl. Acad. Sci. U. S. A."}]},{"#name":"volume-nr","_":"107"}]},{"#name":"issue-nr","_":"10"},{"#name":"date","_":"2010"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"4734"},{"#name":"last-page","_":"4739"}]}]},{"#name":"comment","_":"(Mar.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0050"},"$$":[{"#name":"label","_":"Blondel et al., 2008"},{"#name":"reference","$":{"id":"rf0050","refId":"10"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"V.D."},{"#name":"surname","_":"Blondel"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J.L."},{"#name":"surname","_":"Guillaume"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R."},{"#name":"surname","_":"Lambiotte"}]},{"#name":"author","$$":[{"#name":"given-name","_":"E."},{"#name":"surname","_":"Lefebvre"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Fast unfolding of communities in large networks"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"J. Stat. Mech. Theory Exp."}]},{"#name":"volume-nr","_":"2008"}]},{"#name":"date","_":"2008"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"P10008"}]}]},{"#name":"comment","_":"(Oct.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0055"},"$$":[{"#name":"label","_":"Bullmore and Bassett, 2011"},{"#name":"reference","$":{"id":"rf0055","refId":"11"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"E.T."},{"#name":"surname","_":"Bullmore"}]},{"#name":"author","$$":[{"#name":"given-name","_":"D.S."},{"#name":"surname","_":"Bassett"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Brain graphs: graphical models of the human brain connectome"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Annu. Rev. Clin. Psychol."}]},{"#name":"volume-nr","_":"7"}]},{"#name":"date","_":"2011"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"113"},{"#name":"last-page","_":"140"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0060"},"$$":[{"#name":"label","_":"Craddock et al., 2012"},{"#name":"reference","$":{"id":"rf0060","refId":"12"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"R.C."},{"#name":"surname","_":"Craddock"}]},{"#name":"author","$$":[{"#name":"given-name","_":"G."},{"#name":"surname","_":"James"}]},{"#name":"author","$$":[{"#name":"given-name","_":"P.E."},{"#name":"surname","_":"Holtzheimer"}]},{"#name":"author","$$":[{"#name":"given-name","_":"X.P."},{"#name":"surname","_":"Hu"}]},{"#name":"author","$$":[{"#name":"given-name","_":"H.S."},{"#name":"surname","_":"Mayberg"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"A whole brain fMRI atlas generated via spatially constrained spectral clustering"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Hum. Brain Mapp."}]},{"#name":"volume-nr","_":"33"}]},{"#name":"issue-nr","_":"8"},{"#name":"date","_":"2012"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"1914"},{"#name":"last-page","_":"1928"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0065"},"$$":[{"#name":"label","_":"Dagli et al., 1999"},{"#name":"reference","$":{"id":"rf0065","refId":"13"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M.S."},{"#name":"surname","_":"Dagli"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J.E."},{"#name":"surname","_":"Ingeholm"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J.V."},{"#name":"surname","_":"Haxby"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Localization of cardiac-induced signal change in fMRI"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Neuroimage"}]},{"#name":"volume-nr","_":"9"}]},{"#name":"date","_":"1999"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"407"},{"#name":"last-page","_":"415"}]}]},{"#name":"comment","_":"(Apr.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0070"},"$$":[{"#name":"label","_":"Dahl, 2005"},{"#name":"reference","$":{"id":"rf0070","refId":"14"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"D.B."},{"#name":"surname","_":"Dahl"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Sequentially-Allocated Merge-Split Sampler for Conjugate and Nonconjugate Dirichlet Process Mixture Models"}]}]},{"#name":"host","$$":[{"#name":"edited-book","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Technical Report"}]},{"#name":"date","_":"2005"},{"#name":"publisher","$$":[{"#name":"name","_":"Department of Statistics, Texas A&M University"}]}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0075"},"$$":[{"#name":"label","_":"Damoiseaux et al., 2006"},{"#name":"reference","$":{"id":"rf0075","refId":"15"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"J.S."},{"#name":"surname","_":"Damoiseaux"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Rombouts"}]},{"#name":"author","$$":[{"#name":"given-name","_":"F."},{"#name":"surname","_":"Barkhof"}]},{"#name":"author","$$":[{"#name":"given-name","_":"P."},{"#name":"surname","_":"Scheltens"}]},{"#name":"author","$$":[{"#name":"given-name","_":"C.J."},{"#name":"surname","_":"Stam"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.M."},{"#name":"surname","_":"Smith"}]},{"#name":"author","$$":[{"#name":"given-name","_":"C.F."},{"#name":"surname","_":"Beckmann"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Consistent resting-state networks across healthy subjects"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Proc. Natl. Acad. Sci. U. S. A."}]},{"#name":"volume-nr","_":"103"}]},{"#name":"issue-nr","_":"37"},{"#name":"date","_":"2006"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"13848"},{"#name":"last-page","_":"13853"}]}]},{"#name":"comment","_":"(Sep.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0080"},"$$":[{"#name":"label","_":"Fortunato, 2010"},{"#name":"reference","$":{"id":"rf0080","refId":"16"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Fortunato"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Community detection in graphs"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Phys. Rep."}]},{"#name":"volume-nr","_":"486"}]},{"#name":"issue-nr","_":"3-5"},{"#name":"date","_":"2010"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"75"},{"#name":"last-page","_":"174"}]}]},{"#name":"comment","_":"(arXiv:arXiv:0906.0612v2)"}]}]},{"#name":"bib-reference","$":{"id":"bb0085"},"$$":[{"#name":"label","_":"Fortunato and BarthÃ©lemy, 2007"},{"#name":"reference","$":{"id":"rf0085","refId":"17"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Fortunato"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"BarthÃ©lemy"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Resolution limit in community detection"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Proc. Natl. Acad. Sci. U. S. A."}]},{"#name":"volume-nr","_":"104"}]},{"#name":"issue-nr","_":"1"},{"#name":"date","_":"2007"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"36"},{"#name":"last-page","_":"41"}]}]},{"#name":"comment","_":"(Jan.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0090"},"$$":[{"#name":"label","_":"Fox et al., 2005"},{"#name":"reference","$":{"id":"rf0090","refId":"18"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"Fox"}]},{"#name":"author","$$":[{"#name":"given-name","_":"A."},{"#name":"surname","_":"Snyder"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J."},{"#name":"surname","_":"Vincent"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"The human brain is intrinsically organized into dynamic, anticorrelated functional networks"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Proc. Natl. Acad. Sci."}]},{"#name":"volume-nr","_":"102"}]},{"#name":"issue-nr","_":"27"},{"#name":"date","_":"2005"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"9673"},{"#name":"last-page","_":"9678"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0095"},"$$":[{"#name":"label","_":"Friston et al., 1996"},{"#name":"reference","$":{"id":"rf0095","refId":"19"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"K.J."},{"#name":"surname","_":"Friston"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Williams"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R."},{"#name":"surname","_":"Howard"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R.S."},{"#name":"surname","_":"Frackowiak"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R."},{"#name":"surname","_":"Turner"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Movement-related effects in fMRI time-series"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Magn. Reson. Med."}]},{"#name":"volume-nr","_":"35"}]},{"#name":"issue-nr","_":"3"},{"#name":"date","_":"1996"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"346"},{"#name":"last-page","_":"355"}]}]},{"#name":"comment","_":"(Mar.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0100"},"$$":[{"#name":"label","_":"Glover et al., 2000"},{"#name":"reference","$":{"id":"rf0100","refId":"20"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"G.H."},{"#name":"surname","_":"Glover"}]},{"#name":"author","$$":[{"#name":"given-name","_":"T.Q."},{"#name":"surname","_":"Li"}]},{"#name":"author","$$":[{"#name":"given-name","_":"D."},{"#name":"surname","_":"Ress"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Image-based method for retrospective correction of physiological motion effects in fMRI: RETROICOR"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Magn. Reson. Med."}]},{"#name":"volume-nr","_":"44"}]},{"#name":"issue-nr","_":"1"},{"#name":"date","_":"2000"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"162"},{"#name":"last-page","_":"167"}]}]},{"#name":"comment","_":"(Jul.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0105"},"$$":[{"#name":"label","_":"Goutte et al., 1999"},{"#name":"reference","$":{"id":"rf0105","refId":"21"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"C."},{"#name":"surname","_":"Goutte"}]},{"#name":"author","$$":[{"#name":"given-name","_":"P."},{"#name":"surname","_":"Toft"}]},{"#name":"author","$$":[{"#name":"given-name","_":"E."},{"#name":"surname","_":"Rostrup"}]},{"#name":"author","$$":[{"#name":"given-name","_":"F."},{"#name":"surname","_":"Nielsen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"L.K."},{"#name":"surname","_":"Hansen"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"On clustering fMRI time series"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"NeuroImage"}]},{"#name":"volume-nr","_":"9"}]},{"#name":"issue-nr","_":"3"},{"#name":"date","_":"1999"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"298"},{"#name":"last-page","_":"310"}]}]},{"#name":"comment","_":"(Mar.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0110"},"$$":[{"#name":"label","_":"Hagmann et al., 2008"},{"#name":"reference","$":{"id":"rf0110","refId":"22"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"P."},{"#name":"surname","_":"Hagmann"}]},{"#name":"author","$$":[{"#name":"given-name","_":"L."},{"#name":"surname","_":"Cammoun"}]},{"#name":"author","$$":[{"#name":"given-name","_":"X."},{"#name":"surname","_":"Gigandet"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R."},{"#name":"surname","_":"Meuli"}]},{"#name":"author","$$":[{"#name":"given-name","_":"C.J."},{"#name":"surname","_":"Honey"}]},{"#name":"author","$$":[{"#name":"given-name","_":"V.J."},{"#name":"surname","_":"Wedeen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"O."},{"#name":"surname","_":"Sporns"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Mapping the structural core of human cerebral cortex"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"PLoS Biol."}]},{"#name":"volume-nr","_":"6"}]},{"#name":"issue-nr","_":"7"},{"#name":"date","_":"2008"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"e159"}]}]},{"#name":"comment","_":"(Jul.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0115"},"$$":[{"#name":"label","_":"Herlau et al., 2012"},{"#name":"reference","$":{"id":"rf0115","refId":"23"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"T."},{"#name":"surname","_":"Herlau"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"MÃ¸rup"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M.N."},{"#name":"surname","_":"Schmidt"}]},{"#name":"author","$$":[{"#name":"given-name","_":"L.K."},{"#name":"surname","_":"Hansen"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Modelling dense relational data, in: Machine Learning for Signal Processing (MLSP)"}]}]},{"#name":"host","$$":[{"#name":"edited-book","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"2012 IEEE International Workshop on"}]},{"#name":"date","_":"2012"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"1"},{"#name":"last-page","_":"6"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0120"},"$$":[{"#name":"label","_":"Jain and Neal, 2004"},{"#name":"reference","$":{"id":"rf0120","refId":"24"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Jain"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R.M."},{"#name":"surname","_":"Neal"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"A split-merge Markov chain Monte Carlo procedure for the Dirichlet process mixture model"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"J. Comput. Graph. Stat."}]},{"#name":"volume-nr","_":"13"}]},{"#name":"issue-nr","_":"1"},{"#name":"date","_":"2004"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"158"},{"#name":"last-page","_":"182"}]}]},{"#name":"comment","_":"(Mar.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0125"},"$$":[{"#name":"label","_":"Kemp et al., 1999"},{"#name":"reference","$":{"id":"rf0125","refId":"25"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"C."},{"#name":"surname","_":"Kemp"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J."},{"#name":"surname","_":"Tenenbaum"}]},{"#name":"author","$$":[{"#name":"given-name","_":"T."},{"#name":"surname","_":"Griffiths"}]},{"#name":"author","$$":[{"#name":"given-name","_":"T."},{"#name":"surname","_":"Yamada"}]},{"#name":"author","$$":[{"#name":"given-name","_":"N."},{"#name":"surname","_":"Ueda"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Learning systems of concepts with an infinite relational model"}]}]},{"#name":"host","$$":[{"#name":"edited-book","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Proceedings of the 21th National Conference on Artificial Intelligence (AAAI)"}]},{"#name":"date","_":"1999"},{"#name":"publisher","$$":[{"#name":"name","_":"AAAI Press; MIT Press"},{"#name":"location","_":"Menlo Park, CA; Cambridge, MA; London"}]}]},{"#name":"pages","$$":[{"#name":"first-page","_":"381"},{"#name":"last-page","_":"388"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0130"},"$$":[{"#name":"label","_":"Lehmann and Hansen, 2007"},{"#name":"reference","$":{"id":"rf0130","refId":"26"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Lehmann"}]},{"#name":"author","$$":[{"#name":"given-name","_":"L.K."},{"#name":"surname","_":"Hansen"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Deterministic modularity optimization"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Eur. Phys. J."}]},{"#name":"volume-nr","_":"60"}]},{"#name":"issue-nr","_":"1"},{"#name":"date","_":"2007"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"83"},{"#name":"last-page","_":"88"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0135"},"$$":[{"#name":"label","_":"Lund et al., 2006"},{"#name":"reference","$":{"id":"rf0135","refId":"27"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"T.E."},{"#name":"surname","_":"Lund"}]},{"#name":"author","$$":[{"#name":"given-name","_":"K.H."},{"#name":"surname","_":"Madsen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"K."},{"#name":"surname","_":"Sidaros"}]},{"#name":"author","$$":[{"#name":"given-name","_":"W.L."},{"#name":"surname","_":"Luo"}]},{"#name":"author","$$":[{"#name":"given-name","_":"T.E."},{"#name":"surname","_":"Nichols"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Non-white noise in fMRI: does modelling have an impact?"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"NeuroImage"}]},{"#name":"volume-nr","_":"29"}]},{"#name":"issue-nr","_":"1"},{"#name":"date","_":"2006"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"54"},{"#name":"last-page","_":"66"}]}]},{"#name":"comment","_":"(Jan.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0140"},"$$":[{"#name":"label","_":"McKeown et al., 1998"},{"#name":"reference","$":{"id":"rf0140","refId":"28"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M.J."},{"#name":"surname","_":"McKeown"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Makeig"}]},{"#name":"author","$$":[{"#name":"given-name","_":"G.G."},{"#name":"surname","_":"Brown"}]},{"#name":"author","$$":[{"#name":"given-name","_":"T.P."},{"#name":"surname","_":"Jung"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.S."},{"#name":"surname","_":"Kindermann"}]},{"#name":"author","$$":[{"#name":"given-name","_":"A.J."},{"#name":"surname","_":"Bell"}]},{"#name":"author","$$":[{"#name":"given-name","_":"T.J."},{"#name":"surname","_":"Sejnowski"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Analysis of fMRI data by blind separation into independent spatial components"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Hum. Brain Mapp."}]},{"#name":"volume-nr","_":"6"}]},{"#name":"issue-nr","_":"3"},{"#name":"date","_":"1998"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"160"},{"#name":"last-page","_":"188"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0145"},"$$":[{"#name":"label","_":"McKeown et al., 2003"},{"#name":"reference","$":{"id":"rf0145","refId":"29"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M.J."},{"#name":"surname","_":"McKeown"}]},{"#name":"author","$$":[{"#name":"given-name","_":"L.K."},{"#name":"surname","_":"Hansen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"T.J."},{"#name":"surname","_":"Sejnowski"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Independent component analysis of functional MRI: what is signal and what is noise?"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Curr. Opin. Neurobiol."}]},{"#name":"volume-nr","_":"13"}]},{"#name":"issue-nr","_":"5"},{"#name":"date","_":"2003"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"620"},{"#name":"last-page","_":"629"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0150"},"$$":[{"#name":"label","_":"Meunier et al., 2009"},{"#name":"reference","$":{"id":"rf0150","refId":"30"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"D."},{"#name":"surname","_":"Meunier"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Achard"}]},{"#name":"author","$$":[{"#name":"given-name","_":"A."},{"#name":"surname","_":"Morcom"}]},{"#name":"author","$$":[{"#name":"given-name","_":"E."},{"#name":"surname","_":"Bullmore"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Age-related changes in modular organization of human brain functional networks"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"NeuroImage"}]},{"#name":"volume-nr","_":"44"}]},{"#name":"issue-nr","_":"3"},{"#name":"date","_":"2009"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"715"},{"#name":"last-page","_":"723"}]}]},{"#name":"comment","_":"(Feb.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0155"},"$$":[{"#name":"label","_":"MÃ¸rup and Schmidt, 2012"},{"#name":"reference","$":{"id":"rf0155","refId":"31"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"MÃ¸rup"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M.N."},{"#name":"surname","_":"Schmidt"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Bayesian community detection"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Neural Comput."}]},{"#name":"volume-nr","_":"24"}]},{"#name":"issue-nr","_":"9"},{"#name":"date","_":"2012"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"2434"},{"#name":"last-page","_":"2456"}]}]},{"#name":"comment","_":"(Sep.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0160"},"$$":[{"#name":"label","_":"MÃ¸rup et al., 2010"},{"#name":"reference","$":{"id":"rf0160","refId":"32"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"MÃ¸rup"}]},{"#name":"author","$$":[{"#name":"given-name","_":"K.H."},{"#name":"surname","_":"Madsen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"A.M."},{"#name":"surname","_":"Dogonowski"}]},{"#name":"author","$$":[{"#name":"given-name","_":"H."},{"#name":"surname","_":"Siebner"}]},{"#name":"author","$$":[{"#name":"given-name","_":"L.K."},{"#name":"surname","_":"Hansen"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Infinite relational modeling of functional connectivity in resting state fMRI"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Adv. Neural Inf. Process. Syst."}]},{"#name":"volume-nr","_":"23"}]},{"#name":"date","_":"2010"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"1750"},{"#name":"last-page","_":"1758"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0165"},"$$":[{"#name":"label","_":"Newman, 2006"},{"#name":"reference","$":{"id":"rf0165","refId":"33"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M.E.J."},{"#name":"surname","_":"Newman"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Modularity and community structure in networks"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Proc. Natl. Acad. Sci. U. S. A."}]},{"#name":"volume-nr","_":"103"}]},{"#name":"issue-nr","_":"23"},{"#name":"date","_":"2006"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"8577"},{"#name":"last-page","_":"8582"}]}]},{"#name":"comment","_":"(Jun.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0170"},"$$":[{"#name":"label","_":"Nowicki and Snijders, 2001"},{"#name":"reference","$":{"id":"rf0170","refId":"34"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"K."},{"#name":"surname","_":"Nowicki"}]},{"#name":"author","$$":[{"#name":"given-name","_":"T."},{"#name":"surname","_":"Snijders"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Estimation and prediction for stochastic blockstructures"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"J. Am. Stat. Assoc."}]},{"#name":"volume-nr","_":"96"}]},{"#name":"issue-nr","_":"455"},{"#name":"date","_":"2001"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"1077"},{"#name":"last-page","_":"1087"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0175"},"$$":[{"#name":"label","_":"Oldfield, 1971"},{"#name":"reference","$":{"id":"rf0175","refId":"35"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"R.C."},{"#name":"surname","_":"Oldfield"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"The assessment and analysis of handedness: the Edinburgh inventory"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Neuropsychologia"}]},{"#name":"volume-nr","_":"9"}]},{"#name":"issue-nr","_":"1"},{"#name":"date","_":"1971"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"97"},{"#name":"last-page","_":"113"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0180"},"$$":[{"#name":"label","_":"Power et al., 2011"},{"#name":"reference","$":{"id":"rf0180","refId":"36"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"J.D."},{"#name":"surname","_":"Power"}]},{"#name":"author","$$":[{"#name":"given-name","_":"A.L."},{"#name":"surname","_":"Cohen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.M."},{"#name":"surname","_":"Nelson"}]},{"#name":"author","$$":[{"#name":"given-name","_":"G.S."},{"#name":"surname","_":"Wig"}]},{"#name":"author","$$":[{"#name":"given-name","_":"K.A."},{"#name":"surname","_":"Barnes"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J.A."},{"#name":"surname","_":"Church"}]},{"#name":"author","$$":[{"#name":"given-name","_":"A.C."},{"#name":"surname","_":"Vogel"}]},{"#name":"author","$$":[{"#name":"given-name","_":"T.O."},{"#name":"surname","_":"Laumann"}]},{"#name":"author","$$":[{"#name":"given-name","_":"F.M."},{"#name":"surname","_":"Miezin"}]},{"#name":"author","$$":[{"#name":"given-name","_":"B.L."},{"#name":"surname","_":"Schlaggar"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.E."},{"#name":"surname","_":"Petersen"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Functional network organization of the human brain"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Neuron"}]},{"#name":"volume-nr","_":"72"}]},{"#name":"issue-nr","_":"4"},{"#name":"date","_":"2011"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"665"},{"#name":"last-page","_":"678"}]}]},{"#name":"comment","_":"(Nov.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0185"},"$$":[{"#name":"label","_":"Power et al., 2014"},{"#name":"reference","$":{"id":"rf0185","refId":"37"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"J.D."},{"#name":"surname","_":"Power"}]},{"#name":"author","$$":[{"#name":"given-name","_":"A."},{"#name":"surname","_":"Mitra"}]},{"#name":"author","$$":[{"#name":"given-name","_":"T.O."},{"#name":"surname","_":"Laumann"}]},{"#name":"author","$$":[{"#name":"given-name","_":"A.Z."},{"#name":"surname","_":"Snyder"}]},{"#name":"author","$$":[{"#name":"given-name","_":"B.L."},{"#name":"surname","_":"Schlaggar"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.E."},{"#name":"surname","_":"Petersen"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Methods to detect, characterize, and remove motion artifact in resting state fMRI"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"NeuroImage"}]},{"#name":"volume-nr","_":"84"}]},{"#name":"date","_":"2014"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"320"},{"#name":"last-page","_":"341"}]}]},{"#name":"comment","_":"(Jan.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0190"},"$$":[{"#name":"label","_":"Rosvall and Bergstrom, 2008"},{"#name":"reference","$":{"id":"rf0190","refId":"38"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"Rosvall"}]},{"#name":"author","$$":[{"#name":"given-name","_":"C."},{"#name":"surname","_":"Bergstrom"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Maps of random walks on complex networks reveal community structure"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Proc. Natl. Acad. Sci."}]},{"#name":"volume-nr","_":"105"}]},{"#name":"issue-nr","_":"4"},{"#name":"date","_":"2008"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"1118"},{"#name":"last-page","_":"1123"}]}]},{"#name":"comment","_":"(Sep. arXiv:0709.4500)"}]}]},{"#name":"bib-reference","$":{"id":"bb0195"},"$$":[{"#name":"label","_":"Rubinov and Sporns, 2010"},{"#name":"reference","$":{"id":"rf0195","refId":"39"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"Rubinov"}]},{"#name":"author","$$":[{"#name":"given-name","_":"O."},{"#name":"surname","_":"Sporns"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Complex network measures of brain connectivity: uses and interpretations"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"NeuroImage"}]},{"#name":"volume-nr","_":"52"}]},{"#name":"issue-nr","_":"3"},{"#name":"date","_":"2010"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"1059"},{"#name":"last-page","_":"1069"}]}]},{"#name":"comment","_":"(Sep.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0200"},"$$":[{"#name":"label","_":"Schmidt and MÃ¸rup, 2013"},{"#name":"reference","$":{"id":"rf0200","refId":"40"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M.N."},{"#name":"surname","_":"Schmidt"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"MÃ¸rup"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Nonparametric Bayesian modeling of complex networks"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"IEEE Signal Process. Mag."}]},{"#name":"volume-nr","_":"30"}]},{"#name":"issue-nr","_":"3"},{"#name":"date","_":"2013"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"110"},{"#name":"last-page","_":"128"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0205"},"$$":[{"#name":"label","_":"Smith et al., 1999"},{"#name":"reference","$":{"id":"rf0205","refId":"41"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"A.M."},{"#name":"surname","_":"Smith"}]},{"#name":"author","$$":[{"#name":"given-name","_":"B.K."},{"#name":"surname","_":"Lewis"}]},{"#name":"author","$$":[{"#name":"given-name","_":"U.E."},{"#name":"surname","_":"Ruttimann"}]},{"#name":"author","$$":[{"#name":"given-name","_":"F.Q."},{"#name":"surname","_":"Ye"}]},{"#name":"author","$$":[{"#name":"given-name","_":"T.M."},{"#name":"surname","_":"Sinnwell"}]},{"#name":"author","$$":[{"#name":"given-name","_":"Y."},{"#name":"surname","_":"Yang"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J.H."},{"#name":"surname","_":"Duyn"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J.A."},{"#name":"surname","_":"Frank"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Investigation of low frequency drift in fMRI signal"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Neuroimage"}]},{"#name":"volume-nr","_":"9"}]},{"#name":"date","_":"1999"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"526"},{"#name":"last-page","_":"533"}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0210"},"$$":[{"#name":"label","_":"Sporns, 2011"},{"#name":"reference","$":{"id":"rf0210","refId":"42"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"O."},{"#name":"surname","_":"Sporns"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"The human connectome: a complex network"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Ann. N. Y. Acad. Sci."}]},{"#name":"volume-nr","_":"1224"}]},{"#name":"issue-nr","_":"1"},{"#name":"date","_":"2011"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"109"},{"#name":"last-page","_":"125"}]}]},{"#name":"comment","_":"(Apr.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0215"},"$$":[{"#name":"label","_":"Sporns, 2013"},{"#name":"reference","$":{"id":"rf0215","refId":"43"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"O."},{"#name":"surname","_":"Sporns"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Network attributes for segregation and integration in the human brain"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Curr. Opin. Neurobiol."}]},{"#name":"volume-nr","_":"23"}]},{"#name":"issue-nr","_":"2"},{"#name":"date","_":"2013"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"162"},{"#name":"last-page","_":"171"}]}]},{"#name":"comment","_":"(Jan.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0220"},"$$":[{"#name":"label","_":"Stanley et al., 2013"},{"#name":"reference","$":{"id":"rf0220","refId":"44"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M.L."},{"#name":"surname","_":"Stanley"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M.N."},{"#name":"surname","_":"Moussa"}]},{"#name":"author","$$":[{"#name":"given-name","_":"B.M."},{"#name":"surname","_":"Paolini"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R.G."},{"#name":"surname","_":"Lyday"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J.H."},{"#name":"surname","_":"Burdette"}]},{"#name":"author","$$":[{"#name":"given-name","_":"P.J."},{"#name":"surname","_":"Laurienti"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Defining nodes in complex brain networks"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Front. Comput. Neurosci."}]},{"#name":"volume-nr","_":"7"}]},{"#name":"issue-nr","_":"November"},{"#name":"date","_":"2013"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"169"}]}]},{"#name":"comment","_":"(Jan.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0225"},"$$":[{"#name":"label","_":"Stevens et al., 2012"},{"#name":"reference","$":{"id":"rf0225","refId":"45"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"A.A."},{"#name":"surname","_":"Stevens"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S.C."},{"#name":"surname","_":"Tappon"}]},{"#name":"author","$$":[{"#name":"given-name","_":"A."},{"#name":"surname","_":"Garg"}]},{"#name":"author","$$":[{"#name":"given-name","_":"D.A."},{"#name":"surname","_":"Fair"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Functional brain network modularity captures inter- and intra-individual variation in working memory capacity"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"PLoS One"}]},{"#name":"volume-nr","_":"7"}]},{"#name":"issue-nr","_":"1"},{"#name":"date","_":"2012"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"e30468"}]}]},{"#name":"comment","_":"(Jan.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0230"},"$$":[{"#name":"label","_":"Strother et al., 2002"},{"#name":"reference","$":{"id":"rf0230","refId":"46"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"S.C."},{"#name":"surname","_":"Strother"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J."},{"#name":"surname","_":"Anderson"}]},{"#name":"author","$$":[{"#name":"given-name","_":"L.K."},{"#name":"surname","_":"Hansen"}]},{"#name":"author","$$":[{"#name":"given-name","_":"U."},{"#name":"surname","_":"Kjems"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R."},{"#name":"surname","_":"Kustra"}]},{"#name":"author","$$":[{"#name":"given-name","_":"J."},{"#name":"surname","_":"Sidtis"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Frutiger"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"Muley"}]},{"#name":"author","$$":[{"#name":"given-name","_":"S."},{"#name":"surname","_":"LaConte"}]},{"#name":"author","$$":[{"#name":"given-name","_":"D."},{"#name":"surname","_":"Rottenberg"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"The quantitative evaluation of functional neuroimaging experiments: the NPAIRS data analysis framework"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"NeuroImage"}]},{"#name":"volume-nr","_":"15"}]},{"#name":"issue-nr","_":"4"},{"#name":"date","_":"2002"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"747"},{"#name":"last-page","_":"771"}]}]},{"#name":"comment","_":"(Apr.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0235"},"$$":[{"#name":"label","_":"Tzourio-Mazoyer et al., 2002"},{"#name":"reference","$":{"id":"rf0235","refId":"47"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"N."},{"#name":"surname","_":"Tzourio-Mazoyer"}]},{"#name":"author","$$":[{"#name":"given-name","_":"B."},{"#name":"surname","_":"Landeau"}]},{"#name":"author","$$":[{"#name":"given-name","_":"D."},{"#name":"surname","_":"Papathanassiou"}]},{"#name":"author","$$":[{"#name":"given-name","_":"F."},{"#name":"surname","_":"Crivello"}]},{"#name":"author","$$":[{"#name":"given-name","_":"O."},{"#name":"surname","_":"Etard"}]},{"#name":"author","$$":[{"#name":"given-name","_":"N."},{"#name":"surname","_":"Delcroix"}]},{"#name":"author","$$":[{"#name":"given-name","_":"B."},{"#name":"surname","_":"Mazoyer"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"Joliot"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Automated anatomical labeling of activations in SPM using a macroscopic anatomical parcellation of the MNI MRI single-subject brain"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Neuroimage"}]},{"#name":"volume-nr","_":"15"}]},{"#name":"issue-nr","_":"4"},{"#name":"date","_":"2002"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"273"},{"#name":"last-page","_":"289"}]}]},{"#name":"comment","_":"(Jan.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0240"},"$$":[{"#name":"label","_":"van den Heuvel and Sporns, 2011"},{"#name":"reference","$":{"id":"rf0240","refId":"48"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M.P."},{"#name":"surname","_":"van den Heuvel"}]},{"#name":"author","$$":[{"#name":"given-name","_":"O."},{"#name":"surname","_":"Sporns"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Rich-club organization of the human connectome"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"J. Neurosci."}]},{"#name":"volume-nr","_":"31"}]},{"#name":"issue-nr","_":"44"},{"#name":"date","_":"2011"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"15775"},{"#name":"last-page","_":"15786"}]}]},{"#name":"comment","_":"(Nov.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0245"},"$$":[{"#name":"label","_":"van den Heuvel et al., 2008"},{"#name":"reference","$":{"id":"rf0245","refId":"49"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"M.P."},{"#name":"surname","_":"van den Heuvel"}]},{"#name":"author","$$":[{"#name":"given-name","_":"C.J."},{"#name":"surname","_":"Stam"}]},{"#name":"author","$$":[{"#name":"given-name","_":"M."},{"#name":"surname","_":"Boersma"}]},{"#name":"author","$$":[{"#name":"given-name","_":"H.E."},{"#name":"surname","_":"Hulshoff Pol"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Small-world and scale-free organization of voxel-based resting-state functional connectivity in the human brain"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"NeuroImage"}]},{"#name":"volume-nr","_":"43"}]},{"#name":"issue-nr","_":"3"},{"#name":"date","_":"2008"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"528"},{"#name":"last-page","_":"539"}]}]},{"#name":"comment","_":"(Nov.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0250"},"$$":[{"#name":"label","_":"Varoquaux and Craddock, 2013"},{"#name":"reference","$":{"id":"rf0250","refId":"50"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"G."},{"#name":"surname","_":"Varoquaux"}]},{"#name":"author","$$":[{"#name":"given-name","_":"R.C."},{"#name":"surname","_":"Craddock"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Learning and comparing functional connectomes across subjects"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"NeuroImage"}]},{"#name":"volume-nr","_":"80"}]},{"#name":"date","_":"2013"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"405"},{"#name":"last-page","_":"415"}]}]},{"#name":"comment","_":"(Oct.)"}]}]},{"#name":"bib-reference","$":{"id":"bb0255"},"$$":[{"#name":"label","_":"Xu et al., 2006"},{"#name":"reference","$":{"id":"rf0255","refId":"51"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"Z."},{"#name":"surname","_":"Xu"}]},{"#name":"author","$$":[{"#name":"given-name","_":"V."},{"#name":"surname","_":"Tresp"}]},{"#name":"author","$$":[{"#name":"given-name","_":"K."},{"#name":"surname","_":"Yu"}]},{"#name":"author","$$":[{"#name":"given-name","_":"H."},{"#name":"surname","_":"Kriegel"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"Infinite hidden relational models"}]}]},{"#name":"host","$$":[{"#name":"edited-book","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Proceedings of the 22nd International Conference on Uncertainty in Artificial Intelligence (UAI)"}]},{"#name":"date","_":"2006"},{"#name":"publisher","$$":[{"#name":"name","_":"Citeseer"}]}]}]}]}]},{"#name":"bib-reference","$":{"id":"bb0260"},"$$":[{"#name":"label","_":"Zalesky et al., 2012"},{"#name":"reference","$":{"id":"rf0260","refId":"52"},"$$":[{"#name":"contribution","$":{"langtype":"en"},"$$":[{"#name":"authors","$$":[{"#name":"author","$$":[{"#name":"given-name","_":"A."},{"#name":"surname","_":"Zalesky"}]},{"#name":"author","$$":[{"#name":"given-name","_":"A."},{"#name":"surname","_":"Fornito"}]},{"#name":"author","$$":[{"#name":"given-name","_":"E."},{"#name":"surname","_":"Bullmore"}]}]},{"#name":"title","$$":[{"#name":"maintitle","_":"On the use of correlation as a measure of network connectivity"}]}]},{"#name":"host","$$":[{"#name":"issue","$$":[{"#name":"series","$$":[{"#name":"title","$$":[{"#name":"maintitle","_":"Neuroimage"}]},{"#name":"volume-nr","_":"60"}]},{"#name":"date","_":"2012"}]},{"#name":"pages","$$":[{"#name":"first-page","_":"2096"},{"#name":"last-page","_":"2106"}]}]}]}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"referenceLinks":{"external":[{"refId":3,"scopusHubEid":"2-s2.0-84870018262","crossRefDoi":"10.1007/978-3-642-34713-9_29"},{"refId":4,"crossRefDoi":"10.1109/MLSP.2012.6349739"},{"refId":5,"scopusHubEid":"2-s2.0-20444501009"},{"refId":7,"scopusHubEid":"2-s2.0-79955467238","crossRefDoi":"10.1073/pnas.1018985108"},{"refId":8,"scopusHubEid":"2-s2.0-33746852686"},{"refId":9,"scopusHubEid":"2-s2.0-77649218601","crossRefDoi":"10.1073/pnas.0911855107"},{"refId":10,"scopusHubEid":"2-s2.0-56349094785"},{"refId":11,"scopusHubEid":"2-s2.0-79953664777","crossRefDoi":"10.1146/annurev-clinpsy-040510-143934"},{"refId":12,"scopusHubEid":"2-s2.0-84863882574","crossRefDoi":"10.1002/hbm.21333"},{"refId":13,"scopusHubEid":"2-s2.0-0032777187"},{"refId":15,"scopusHubEid":"2-s2.0-33748796093","crossRefDoi":"10.1073/pnas.0601417103"},{"refId":16,"scopusHubEid":"2-s2.0-74049087026"},{"refId":17,"scopusHubEid":"2-s2.0-33846126275","crossRefDoi":"10.1073/pnas.0605965104"},{"refId":18,"scopusHubEid":"2-s2.0-22144469412","crossRefDoi":"10.1073/pnas.0504136102"},{"refId":19,"scopusHubEid":"2-s2.0-0030032333","crossRefDoi":"10.1002/mrm.1910350312"},{"refId":20,"scopusHubEid":"2-s2.0-0033934775"},{"refId":21,"scopusHubEid":"2-s2.0-0032796478"},{"refId":22,"crossRefDoi":"10.1371/journal.pbio.0060159"},{"refId":24,"scopusHubEid":"2-s2.0-1842486852"},{"refId":26,"scopusHubEid":"2-s2.0-37449015651","crossRefDoi":"10.1140/epjb/e2007-00313-2"},{"refId":27,"scopusHubEid":"2-s2.0-29244477177"},{"refId":28,"scopusHubEid":"2-s2.0-0031861367"},{"refId":29,"scopusHubEid":"2-s2.0-0242404415"},{"refId":30,"scopusHubEid":"2-s2.0-57649177558"},{"refId":31,"scopusHubEid":"2-s2.0-84870691266"},{"refId":33,"scopusHubEid":"2-s2.0-33745012299","crossRefDoi":"10.1073/pnas.0601602103"},{"refId":34,"scopusHubEid":"2-s2.0-0442296603"},{"refId":35,"scopusHubEid":"2-s2.0-0015021890"},{"refId":36,"scopusHubEid":"2-s2.0-81355153871"},{"refId":37,"scopusHubEid":"2-s2.0-84884227262"},{"refId":38,"scopusHubEid":"2-s2.0-39549086558","crossRefDoi":"10.1073/pnas.0706851105"},{"refId":39,"scopusHubEid":"2-s2.0-77954385460"},{"refId":40,"scopusHubEid":"2-s2.0-85032752062"},{"refId":41,"scopusHubEid":"2-s2.0-0032789990"},{"refId":42,"scopusHubEid":"2-s2.0-79954516559","crossRefDoi":"10.1111/j.1749-6632.2010.05888.x"},{"refId":43,"scopusHubEid":"2-s2.0-84875863684"},{"refId":44,"scopusHubEid":"2-s2.0-85103752415"},{"refId":45,"scopusHubEid":"2-s2.0-84856030194","crossRefDoi":"10.1371/journal.pone.0030468"},{"refId":46,"scopusHubEid":"2-s2.0-0036338502"},{"refId":48,"scopusHubEid":"2-s2.0-80155148207"},{"refId":49,"scopusHubEid":"2-s2.0-54149095921"},{"refId":50,"scopusHubEid":"2-s2.0-84880327399"},{"refId":52,"scopusHubEid":"2-s2.0-84858138977"}],"internal":[{"refId":5,"pii":"S1053811905001102","filesize":"436KB","pdf":{"urlType":"download","url":"/science/article/pii/S1053811905001102/pdfft?md5=1de64d4cb10143324a12856cf1af3504&pid=1-s2.0-S1053811905001102-main.pdf"}},{"refId":8,"pii":"S1053811906001248","filesize":"3MB","pdf":{"urlType":"download","url":"/science/article/pii/S1053811906001248/pdfft?md5=1de09e02d8701ed9e0c7df4a443adbd7&pid=1-s2.0-S1053811906001248-main.pdf"}},{"refId":13,"pii":"S1053811998904244","filesize":"623KB","pdf":{"urlType":"download","url":"/science/article/pii/S1053811998904244/pdf?md5=3156f125d77e3eea6110e33e8da9fe31&pid=1-s2.0-S1053811998904244-main.pdf"}},{"refId":16,"pii":"S0370157309002841","filesize":"14MB","pdf":{"urlType":"download","url":"/science/article/pii/S0370157309002841/pdfft?md5=73579bd7359e961d3fdf5788ddd437e0&pid=1-s2.0-S0370157309002841-main.pdf"}},{"refId":21,"pii":"S1053811998903913","filesize":"368KB","pdf":{"urlType":"download","url":"/science/article/pii/S1053811998903913/pdf?md5=f2bd3bacdc2357425428e941b13cb843&pid=1-s2.0-S1053811998903913-main.pdf"}},{"refId":27,"pii":"S105381190500501X","filesize":"1MB","pdf":{"urlType":"download","url":"/science/article/pii/S105381190500501X/pdfft?md5=02d9be86e3de6397ec804ed31757025a&pid=1-s2.0-S105381190500501X-main.pdf"}},{"refId":29,"pii":"S0959438803001338","filesize":"262KB","pdf":{"urlType":"download","url":"/science/article/pii/S0959438803001338/pdfft?md5=0c9afb7ed59b9f8e43a99dddcfcd8d73&pid=1-s2.0-S0959438803001338-main.pdf"}},{"refId":30,"pii":"S1053811908011026","filesize":"1MB","pdf":{"urlType":"download","url":"/science/article/pii/S1053811908011026/pdfft?md5=88040029c95f241c97af009ec9d84c8e&pid=1-s2.0-S1053811908011026-main.pdf"}},{"refId":35,"pii":"0028393271900674","filesize":"1MB","pdf":{"urlType":"download","url":"/science/article/pii/0028393271900674/pdf?md5=464206d3307518b893eb87c7559ca60b&pid=1-s2.0-0028393271900674-main.pdf"}},{"refId":36,"pii":"S0896627311007926","filesize":"2MB","pdf":{"urlType":"download","url":"/science/article/pii/S0896627311007926/pdfft?md5=1d089608bf2b1c00696945bb9c4f5469&pid=1-s2.0-S0896627311007926-main.pdf"}},{"refId":37,"pii":"S1053811913009117","filesize":"5MB","pdf":{"urlType":"download","url":"/science/article/pii/S1053811913009117/pdfft?md5=17fff16b2e8a5dfdedfa31aafc690fad&pid=1-s2.0-S1053811913009117-main.pdf"}},{"refId":39,"pii":"S105381190901074X","filesize":"1MB","pdf":{"urlType":"download","url":"/science/article/pii/S105381190901074X/pdfft?md5=e4952533369cd4b6117061057eb47c1d&pid=1-s2.0-S105381190901074X-main.pdf"}},{"refId":41,"pii":"S1053811999904354","filesize":"250KB","pdf":{"urlType":"download","url":"/science/article/pii/S1053811999904354/pdf?md5=735b49c0f8daee8f797901f00e5ec558&pid=1-s2.0-S1053811999904354-main.pdf"}},{"refId":43,"pii":"S0959438812001894","filesize":"3MB","pdf":{"urlType":"download","url":"/science/article/pii/S0959438812001894/pdfft?md5=20c5d5dc5b8cb2e4eb30e5f093956ecf&pid=1-s2.0-S0959438812001894-main.pdf"}},{"refId":46,"pii":"S1053811901910341","filesize":"406KB","pdf":{"urlType":"download","url":"/science/article/pii/S1053811901910341/pdf?md5=099047c9df8e671fcdb08fc20a42426d&pid=1-s2.0-S1053811901910341-main.pdf"}},{"refId":47,"pii":"S1053811901909784","filesize":"1MB","pdf":{"urlType":"download","url":"/science/article/pii/S1053811901909784/pdf?md5=b92cc2502df02189cd08b79f677bc81f&pid=1-s2.0-S1053811901909784-main.pdf"}},{"refId":49,"pii":"S1053811908009130","filesize":"1MB","pdf":{"urlType":"download","url":"/science/article/pii/S1053811908009130/pdfft?md5=e2e1d37ffeb39a50371f90d98c450d27&pid=1-s2.0-S1053811908009130-main.pdf"}},{"refId":50,"pii":"S1053811913003340","filesize":"2MB","pdf":{"urlType":"download","url":"/science/article/pii/S1053811913003340/pdfft?md5=ca505e057f83f3874e0304b5236bad90&pid=1-s2.0-S1053811913003340-main.pdf"}},{"refId":52,"pii":"S1053811912001784","filesize":"1MB","pdf":{"urlType":"download","url":"/science/article/pii/S1053811912001784/pdfft?md5=c365f91e34548c4603e75417c6bb24e5&pid=1-s2.0-S1053811912001784-main.pdf"}}]},"refersTo":{},"referredToBy":{},"relatedContent":{"isModal":false,"isOpenSpecialIssueArticles":false,"isOpenRecommendations":true,"isOpenReaxys":true,"citingArticles":[false,false,false,false,false,false],"recommendations":[false,false,false,false,false,false]},"seamlessAccess":{},"signInFromEmail":{"isOpen":false},"specialIssueArticles":{},"substances":[],"supplementaryFilesData":[],"tableOfContents":{"outlineTitle":"Outline","outline":[{"#name":"entry","$":{"id":"ab0005","type":"abstract","class":"author","depth":"1"},"$$":[{"#name":"title","$":{"id":"st0005"},"_":"Abstract"}]},{"#name":"entry","$":{"id":"ab0010","type":"abstract","class":"author-highlights","depth":"1"},"$$":[{"#name":"title","$":{"id":"st0010"},"_":"Highlights"}]},{"#name":"entry","$":{"id":"ks0005","type":"keywords","class":"keyword","depth":"1"},"$$":[{"#name":"title","$":{"id":"st0015"},"_":"Keywords"}]},{"#name":"entry","$":{"id":"s0005","type":"sections","depth":"1"},"$$":[{"#name":"title","$":{"id":"st0020"},"_":"Introduction"}]},{"#name":"entry","$":{"id":"s0010","type":"sections","depth":"1"},"$$":[{"#name":"title","$":{"id":"st0025"},"_":"Methods"},{"#name":"entry","$":{"id":"s0015","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0030"},"_":"Copenhagen data"}]},{"#name":"entry","$":{"id":"s0020","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0035"},"_":"Beijing and Leipzig data"}]},{"#name":"entry","$":{"id":"s0025","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0040"},"_":"Graph construction"}]},{"#name":"entry","$":{"id":"s0030","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0045"},"_":"The models"},{"#name":"entry","$":{"id":"s0035","depth":"3"},"$$":[{"#name":"title","$":{"id":"st0050"},"_":"The Infinite Relational Model"}]},{"#name":"entry","$":{"id":"s0040","depth":"3"},"$$":[{"#name":"title","$":{"id":"st0055"},"_":"Infinite Diagonal Model"}]},{"#name":"entry","$":{"id":"s0045","depth":"3"},"$$":[{"#name":"title","$":{"id":"st0060"},"_":"Bayesian Community Detection"}]}]},{"#name":"entry","$":{"id":"s0050","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0065"},"_":"ExampleÂ 1"}]},{"#name":"entry","$":{"id":"s0055","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0070"},"_":"IRM and IDM model inference"}]},{"#name":"entry","$":{"id":"s0060","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0075"},"_":"BCD model inference"}]},{"#name":"entry","$":{"id":"s0065","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0080"},"_":"ExampleÂ 2"}]},{"#name":"entry","$":{"id":"s0070","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0085"},"_":"NPAIRS evaluation criteria"}]},{"#name":"entry","$":{"id":"s0075","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0090"},"_":"Experiments"},{"#name":"entry","$":{"id":"s0080","depth":"3"},"$$":[{"#name":"title","$":{"id":"st0095"},"_":"Estimated clusters"}]},{"#name":"entry","$":{"id":"s0085","depth":"3"},"$$":[{"#name":"title","$":{"id":"st0100"},"_":"Predictability and reproducibility"}]},{"#name":"entry","$":{"id":"s0090","depth":"3"},"$$":[{"#name":"title","$":{"id":"st0105"},"_":"Predictability and reproducibility for various link densities"}]}]}]},{"#name":"entry","$":{"id":"s0095","type":"sections","depth":"1"},"$$":[{"#name":"title","$":{"id":"st0110"},"_":"Results"},{"#name":"entry","$":{"id":"s0100","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0115"},"_":"Estimated clusters"}]},{"#name":"entry","$":{"id":"s0105","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0120"},"_":"Predictability and reproducibility"}]},{"#name":"entry","$":{"id":"s0110","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0125"},"_":"Predictability and reproducibility for various link densities"}]}]},{"#name":"entry","$":{"id":"s0115","type":"sections","depth":"1"},"$$":[{"#name":"title","$":{"id":"st0130"},"_":"Discussion and conclusion"},{"#name":"entry","$":{"id":"s0120","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0135"},"_":"Toolbox"}]}]},{"#name":"entry","$":{"id":"ac0005","type":"acknowledgment","depth":"1"},"$$":[{"#name":"title","$":{"id":"st0140"},"_":"Acknowledgment"}]},{"#name":"entry","$":{"id":"s0125","type":"appendices","depth":"1"},"$$":[{"#name":"label","_":"Appendix A"},{"#name":"title","$":{"id":"st0145"},"_":"FCON1000 data"}]},{"#name":"entry","$":{"id":"s0130","type":"appendices","depth":"1"},"$$":[{"#name":"label","_":"Appendix B"},{"#name":"title","$":{"id":"st0150"},"_":"Inference"},{"#name":"entry","$":{"id":"s0135","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0155"},"_":"IRM"}]},{"#name":"entry","$":{"id":"s0140","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0160"},"_":"IDM"}]},{"#name":"entry","$":{"id":"s0145","depth":"2"},"$$":[{"#name":"title","$":{"id":"st0165"},"_":"BCD"}]}]},{"#name":"entry","$":{"id":"s0150","type":"appendices","depth":"1"},"$$":[{"#name":"label","_":"Appendix C"},{"#name":"title","$":{"id":"st0170"},"_":"Cluster labels"}]},{"#name":"entry","$":{"id":"bi0005","type":"bibliography","depth":"1"},"$$":[{"#name":"title","$":{"id":"st0175"},"_":"References"}]}],"figures":[],"tables":[{"#name":"entry","$":{"id":"t0010","type":"sections","local-type":"table"}},{"#name":"entry","$":{"id":"t0015","type":"sections","local-type":"table"}},{"#name":"entry","$":{"id":"t0005","type":"sections","local-type":"table"},"$$":[{"#name":"label","_":"TableÂ 1"},{"#name":"caption","$":{"truncated":"true"},"_":"The mean (std) of normalized mutual information (NMI) between the clustering of 10 runs within and between method along with the number of clusters (D) estimated with each of the three methods IRM, BC..."}]},{"#name":"entry","$":{"id":"t0020","type":"appendices","local-type":"table"}},{"#name":"entry","$":{"id":"t0025","type":"appendices","local-type":"table"}},{"#name":"entry","$":{"id":"t0030","type":"appendices","local-type":"table"}}],"extras":[],"attachments":[],"showEntitledTocLinks":false},"tail":{},"transientError":{"isOpen":false},"userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36","workspace":{"isOpen":false},"viewConfig":{"articleFeature":{"rightsAndContentLink":true},"pathPrefix":""},"virtualSpecialIssue":{"showVirtualSpecialIssueLink":false}}</script>
<script src='https://assets.adobedtm.com/4a848ae9611a/032db4f73473/launch-a6263b31083f.min.js' type="3855cf66cf87175080abf308-text/javascript"></script>
<script nomodule src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/73/js/core-js/3.20.2/core-js.es.minified.js" type="3855cf66cf87175080abf308-text/javascript"></script>
<script src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/51/js/react/16.12.0/react.production.min.js" type="3855cf66cf87175080abf308-text/javascript"></script>
<script src="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/51/js/react-dom/16.12.0/react-dom.production.min.js" type="3855cf66cf87175080abf308-text/javascript"></script>
<script src='https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/f428ac6e436879961ed6bb261fe095ecf4ea09e0/arp.js' async type="3855cf66cf87175080abf308-text/javascript"></script>
<script type="3855cf66cf87175080abf308-text/javascript">
    const pendoData = {"visitor":{"pageName":"SD:product:journal:article","pageType":"CP-CA","pageProduct":"SD","pageLanguage":"en","pageEnvironment":"prod","accessType":"ae:ANON_GUEST","countryCode":"KR"},"account":{"id":"ae:228598","name":"ae:ScienceDirect Guests"},"events":{}};;
    pendoData.events = {
      ready: function () {
        pendo.addAltText();
      },
    };
    function runPendo(data, options) {
  const {
    firstDelay,
    maxRetries,
    urlPrefix,
    urlSuffix,
    apiKey
  } = options;
  (function (apiKey) {
    (function (p, e, n, d, o) {
      var v, w, x, y, z;
      o = p[d] = p[d] || {};
      o._q = [];
      v = ['initialize', 'identify', 'updateOptions', 'pageLoad'];
      for (w = 0, x = v.length; w < x; ++w) (function (m) {
        o[m] = o[m] || function () {
          o._q[m === v[0] ? 'unshift' : 'push']([m].concat([].slice.call(arguments, 0)));
        };
      })(v[w]);
      y = e.createElement(n);
      y.async = !0;
      y.src = urlPrefix + apiKey + urlSuffix;
      z = e.getElementsByTagName(n)[0];
      z.parentNode.insertBefore(y, z);
    })(window, document, 'script', 'pendo');
    pendo.addAltText = function () {
      var target = document.querySelector('body');
      var observer = new MutationObserver(function (mutations) {
        mutations.forEach(function (mutation) {
          if (mutation?.addedNodes?.length) {
            if (mutation.addedNodes[0]?.className?.includes("_pendo-badge")) {
              const badge = mutation.addedNodes[0];
              const altText = badge?.attributes['aria-label'].value ? badge?.attributes['aria-label'].value : 'Feedback';
              const pendoBadgeImage = pendo.dom(`#${badge?.attributes?.id.value} img`);
              if (pendoBadgeImage.length) {
                pendoBadgeImage[0]?.setAttribute('alt', altText);
              }
            }
          }
        });
      });
      var config = {
        attributeFilter: ['data-layout'],
        attributes: true,
        childList: true,
        characterData: true,
        subtree: false
      };
      observer.observe(target, config);
    };
  })(apiKey);
  (function watchAndSetPendo(nextDelay, retryAttempt) {
    if (typeof pageDataTracker === 'object' && typeof pageDataTracker.getVisitorId === 'function' && pageDataTracker.getVisitorId()) {
      data.visitor.id = pageDataTracker.getVisitorId();
      console.debug(`initializing pendo`);
      pendo.initialize(data);
    } else {
      if (retryAttempt > 0) {
        return setTimeout(function () {
          watchAndSetPendo(nextDelay * 2, retryAttempt - 1);
        }, nextDelay);
      }
      pendo.initialize(data);
      console.debug(`gave up ... pendo initialized`);
    }
  })(firstDelay, maxRetries);
}
    runPendo(pendoData, {
      firstDelay: 100,
      maxRetries: 5,
      urlPrefix: 'https://cdn.pendo.io/agent/static/',
      urlSuffix: '/pendo.js',
      apiKey: 'd6c1d995-bc7e-4e53-77f1-2ea4ecbb9565',
    });
  </script>
<script type="text/x-mathjax-config">
        MathJax.Hub.Config({
          displayAlign: 'left',
          "fast-preview": {
            disabled: true
          },
          CommonHTML: { linebreaks: { automatic: true } },
          PreviewHTML: { linebreaks: { automatic: true } },
          'HTML-CSS': { linebreaks: { automatic: true } },
          SVG: {
            scale: 90,
            linebreaks: { automatic: true }
          }
        });
      </script>
<script async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=MML_SVG" type="3855cf66cf87175080abf308-text/javascript"></script>
<script async src="https://www.googletagservices.com/tag/js/gpt.js" type="3855cf66cf87175080abf308-text/javascript"></script>
<script async src="https://scholar.google.com/scholar_js/casa.js" type="3855cf66cf87175080abf308-text/javascript"></script>
<script type="3855cf66cf87175080abf308-module" src="https://static.mendeley.com/view-pdf-component/0.8.9/dist/view-pdf-element.js" crossorigin="anonymous" async></script>
<script src="/cdn-cgi/scripts/7d0fa10a/cloudflare-static/rocket-loader.min.js" data-cf-settings="3855cf66cf87175080abf308-|49" defer=""></script></body>
</html>